\documentclass{article}
\DeclareFontFamily{U}{FdSymbolC}{}
\DeclareFontShape{U}{FdSymbolC}{m}{n}{<-> s * FdSymbolC-Book}{}
\DeclareSymbolFont{fdarrows}{U}{FdSymbolC}{m}{n}
\DeclareMathSymbol{\leftfootline}{\mathrel}{fdarrows}{"AC}
\DeclareMathSymbol{\longleftfootline}{\mathrel}{fdarrows}{"C6}
\DeclareMathSymbol{\leftblackspoon}{\mathrel}{fdarrows}{"6E}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage[english]{babel}
\usepackage{amsmath}
\newcommand{\andop}{\mathbin{\mathrm{and}}}
\newcommand{\orop}{\mathbin{\mathrm{or}}}
\newcommand{\notop}{\mathop{\mathrm{not}}}
\usepackage{longtable}
\setcounter{LTchunksize}{200}
\setlength{\LTcapwidth}{1.15\textwidth}
\usepackage[font=bf,labelfont=bf,justification=justified,singlelinecheck=false]{caption}
\usepackage{subfloat}
\usepackage{booktabs}
\usepackage{makecell}
\usepackage{microtype}
\usepackage[dvipsnames]{xcolor}
\usepackage[margin=1.0in]{geometry}
\usepackage[raggedright]{titlesec}
\renewcommand*{\thepage}{S\arabic{page}}
\renewcommand*{\thesection}{S\arabic{section}}
\renewcommand*{\thetable}{S\arabic{table}}
\renewcommand*{\thefigure}{S\arabic{figure}}
\renewcommand*{\figurename}{Supplemental Material, Figure}
\newcommand*{\nodeoneaspec}{\dimexpr 0.14\textwidth-2\tabcolsep}
\newcommand*{\nodeonebspec}{\dimexpr 0.86\textwidth-2\tabcolsep}
\newcommand*{\nodetwoaspec}{\dimexpr 0.14\textwidth-2\tabcolsep}
\newcommand*{\nodetwobspec}{\dimexpr 0.13\textwidth-2\tabcolsep}
\newcommand*{\nodetwocspec}{\dimexpr 0.73\textwidth-2\tabcolsep}
\newcommand*{\linkaspec}{\dimexpr 0.14\textwidth-2\tabcolsep}
\newcommand*{\linkbspec}{\dimexpr 0.13\textwidth-2\tabcolsep}
\newcommand*{\linkcspec}{\dimexpr 0.14\textwidth-2\tabcolsep}
\newcommand*{\linkdspec}{\dimexpr 0.59\textwidth-2\tabcolsep}
\begin{document}\newpage{}\section{Description \& experimental support for the modules of COPD\_{}EMT\_{}CellCycle\_{}Apoptosis\_{}Fine. 
}
\begin{subtables}\begin{longtable}[c]{@{}llll@{}}
\caption{Neighbors module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CellDensity\_{}High}=\mathbf{CellDensity\_{}High}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{CellDensity\_High} node in our model represents an extracellular environment with high enough cell density to block cell spreading. This input node is self-sustaining in the absence of \textit{in silico} perturbation.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{\textit{CellDensity\_High} is self-sustaining in the absence of \textit{in silico} perturbation.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CellDensity\allowbreak\_{}Low}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CellDensity\_{}Low}=\mathbf{CellDensity\_{}Low}\orop\mathbf{CellDensity\_{}High}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{CellDensity\_Low} node represents an environment with cell density comparable to the edge of a monolayer, where cells can maintain strong adhesions with each other but are also able to spread and polarize horizontally.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{\textit{CellDensity\_Low} is automatically ON at very high cell density.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}Low}&\multicolumn{1}{m{\linkdspec}}{\textit{CellDensity\_Low} is self-sustaining in the absence of \textit{in silico} perturbation.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Matrix module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{COL1\allowbreak\_{}3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{COL1\_{}3}=\mathbf{COL1\_{}3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} The interstitial ECM of alveoli is composed of a relaxed mesh, dominated by type I and III collagens and elastin \cite{burgstallerInstructiveExtracellularMatrix2017}. Here we assume that these ECM components are secreted by tissue-resident fibroblasts into the connective tissue surrouding the alveolar basement membrane of alveolar type I and type II epithelial cells \cite{burgstallerInstructiveExtracellularMatrix2017}, and treat the \textit{COL1\_3} node as an environmental input. Complete absence of either the basement membrane (\textit{Basement\_Membrane} node) or \textit{COL1\_3} represents a scenario where the epithelial cells cannot form any ECM attachments (loss of anchorage; equivalent to the \textit{ECM} = OFF and \textit{Stiff\_ECM} = OFF input in our \cite{sullivanBooleanModelingMechanosensitive2023} model).}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{COL1\allowbreak\_{}3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue}\textit{COL1\_3} is self-sustaining in the absence of \textit{in silico} perturbation.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Elastin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Elastin}=\notop\left(\mathbf{MMP7}\orop\mathbf{MMP9}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} In healthy lungs, the interstitial ECM, maitained by tissue-resident fibroblasts, is a loose meshwork of collagen and elastin fibers, cross-linked and connected to the basement membrane of the alveolar epithelial cell layer \cite{burgstallerInstructiveExtracellularMatrix2017}. Elastin fibers are degraded by \textit{MMP-7/9}, both \textit{in vitro} and in lung tissue \cite{heinzInsightsDegradationHuman2011, kristensenLevelsCirculatingMMP72015,skjot-arkilMeasurementMMP9122012}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{MMP7}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Elastin fibers are degraded by \textit{MMP7}, both \textit{in vitro} and in lung tissue \cite{heinzInsightsDegradationHuman2011, kristensenLevelsCirculatingMMP72015}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{MMP9}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Elastin fibers are cleaved by \textit{MMP9}, both \textit{in vitro} and in lung tissue from COPD and IPF patients \cite{skjot-arkilMeasurementMMP9122012}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Laminin\allowbreak\_{}5a}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Laminin\_{}5a}=\notop\mathbf{MMP2}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} Laminin $\alpha$5 is a key component of the alveolar basement membrane in developing as well as adult lungs \cite{minerLamininAlphaChains1997}. During EMT, laminins are cleaved by \textit{MMP2} \cite{giannelliInductionCellMigration1997}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{MMP2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Cleavage of Laminin 5 by MMP2 was shown to aid epithelial cell migration \cite{giannelliInductionCellMigration1997}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Collagen\allowbreak\_{}4}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Collagen\_{}4}=\notop\left(\mathbf{MMP9}\orop\mathbf{MMP2}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} Collagen IV is found primarily in the basal lamina, an exclusive member of basement membranes. Collagen IV fibers lack the tigh collagen helix structure of other collagens, and link to each other head-to-head (rather than in parallel). These two features allow the collagen IV molecules to form the sheet structure of basal lamina \cite{khoshnoodiMammalianCollagenIV2008}. \textit{MMP-2/9}, secreted by mesenchynal cells, can bind and degrade Collagen IV \cite{miskoMatrixMetalloproteinaseMediated2002,monacoEnzymaticProcessingCollagen2006}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{MMP9}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{MMP9}, secreted by mesenchynal cells, can bind and degrade Collagen IV \cite{miskoMatrixMetalloproteinaseMediated2002}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{MMP2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue}  \textit{MMP2} binds and degrades Collagen IV \cite{monacoEnzymaticProcessingCollagen2006}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Basement\allowbreak\_{}Membrane}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Basement\_{}Membrane}=\mathbf{Collagen\_{}4}\andop\mathbf{Laminin\_{}5a}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} The alveolar basement membrane is a thin glycoproteins sheet that supports the basal side of the alveolar epithelium, separating it from the interstitial ECM and connective tissue. Its two critical components are Collagen IV and Laminin 5 \cite{burgstallerInstructiveExtracellularMatrix2017}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Laminin\allowbreak\_{}5a}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Laminin 5 is a critical structural component of the basement membrane  \cite{burgstallerInstructiveExtracellularMatrix2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Collagen\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Collagen IV is a critical structural component of the basement membrane  \cite{burgstallerInstructiveExtracellularMatrix2017}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Fibronectin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Fibronectin}=\mathbf{SNAI1}\andop\mathbf{NfkB}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} Basal levels of fibronectin serve to anchor the interstitial ECM to the basemement membrane, and cross-link collagen and ellastin fibers. Increased fibronectin production and secretion, however, can stiffen the ECM and lead to fibrosis \cite{burgstallerInstructiveExtracellularMatrix2017}. In our model, \textit{Fibronectin} = OFF represnets basal levels, while the ON state reflects substantial epithelial cell-derived increase is fibronectin by \textit{SNAI1}- and \textit{NF-$\kappa$B}-induced transcription during partial of full EMT \cite{stanisavljevicP65SubunitNfkB2011,leeNFkappaBActivatesFibronectin2002}. }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SNAIL1} is a direct transcriptional inducer of \textit{fibronectin} \cite{stanisavljevicP65SubunitNfkB2011, canoTranscriptionFactorSnail2000}}. }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{NF-$\kappa$B} is a direct transcriptional inducer of \textit{fibronectin} \cite{leeNFkappaBActivatesFibronectin2002}.} }\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{MMP2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{MMP2}=\left(\left(\notop\mathbf{CEBP\_{}alpha}\right)\andop\mathbf{SNAI1}\right)\andop\left(\mathbf{ZEB2}\orop\mathbf{HMGA1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{MMP-2}, also known as gelatinase A, is a matrix metalloprotease that can degrade type IV collagen and thus compromise the basement membrane \cite{miskoMatrixMetalloproteinaseMediated2002}. \textit{MMP-2} is transcriptionally upregulated by \textit{SNAI1} \cite{yokoyamaIncreasedInvasionMatrix2003} and \textit{HMGA1} \cite{hillionUpregulationMMP2HMGA12009}, though \textit{ZEB2} knockdown can significnatly reduce its levels as well \cite{liZEB2PromotesTumor2017}. In contrast, \textit{C/EBP$\alpha$} can interfere with \textit{MMP-2} mRNA induction \cite{lourencoEBPaCrucialDeterminant2020}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{CEBP\allowbreak\_{}alpha}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{C/EBP$\alpha$} expression can interfere with TGF-$\beta$-mediated \textit{MMP-2} induction \cite{lourencoEBPaCrucialDeterminant2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SNAI1} is a direct transcriptional inducer of \textit{MMP-2}, binding to   E-box element essential for its effect \cite{singhSNAI1mediatedTranscriptionalRegulation2021}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ZEB2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{MMP-2} mRNA expression is significantly downregulated following \textit{ZEB2} downregulation \cite{liZEB2PromotesTumor2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{HMGA1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{MMP2} is a direct transcriptional target of \textit{HMGA1} \cite{hillionUpregulationMMP2HMGA12009}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{MMP7}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{MMP7}=\mathbf{LEF1}\andop\mathbf{b\_{}catenin\_{}TCF4}\andop\mathbf{ERK}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{MMP-7} (matrilysin), is a potent elastase in humans, expressed by the mucosal epithelium and macrophages in the lung. It is upregulated in airway epithelial cells in response to injury \cite{gharibMatrixMetalloproteinase7Coordinates2013}, and it degrades ECM components such as elastin \cite{christopoulouMatrixMetalloproteinasesChronic2023}. Elevated \textit{MMP-7} levels are associated with impared lung function, and indeed its expression is increased in the blood of COPD patients \cite{christopoulouMatrixMetalloproteinasesChronic2023}. \textit{MMP-7} transcription is increased by \textit{ERK} as well as \textit{$beta$-catenin/TCF4/LEF1}\cite{chatterjeeEGFRmediatedMatrixMetalloproteinase72018,crawfordPEA3SubfamilyEts2001,brabletzBetacateninRegulatesExpression1999}. }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{LEF1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin/TCF-4/LEF1} complexes are direct transcriptional activators of \textit{MMP7}\cite{crawfordPEA3SubfamilyEts2001}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin/TCF-4} complexes are direct transcriptional activators of \textit{MMP7} \cite{brabletzBetacateninRegulatesExpression1999}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ERK1}, together with AP-1 (not explicitly modeled) are direct transcriptional activators of \textit{MMP7} \cite{chatterjeeEGFRmediatedMatrixMetalloproteinase72018}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{MMP9}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{MMP9}=\notop\mathbf{CEBP\_{}alpha}\andop\mathbf{SNAI1}\andop\mathbf{SNAI2}\andop\mathbf{ZEB2}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{MMP-9}, or Gelatinase B is a matrix metalloprotease with increased expression in COPD \cite{christopoulouMatrixMetalloproteinasesChronic2023}. Among its targets are type IV collagen and elastin \cite{christopoulouMatrixMetalloproteinasesChronic2023}. It's expression during EMT is induced by \textit{SNAI1/2}, \textit{ZEB2} and blocked by CEB/P-$alpha$.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SNAI1} is a direct transcriptional inducer of \textit{MMP-9}; chromatin immunoprecipitation (ChIP) shows binding of \textit{SNAI1} (together with EGR1 and SP1, not modeled) to the \textit{MMP-9} promoter \cite{wuSnailCollaboratesEGR12017c}. Moreover, \textit{SNAI1} siRNA blocks \textit{TGF-$\beta$1}-induced\textit{MMP-9} expression \cite{sunTransformingGrowthFactorbeta2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SNAI2} (\textit{Slug}) increases \textit{MMP-9} at both the protein and mRNA levels. Moreover, \textit{Slug} siRNA decreases \textit{TGF-$\beta$1}-induced \textit{MMP-9} expression \cite{josephSlugDownstreamMediator2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{MMP-9} mRNA expression is significantly upregulated by \textit{ZEB2} overexpression, while siRNA against \textit{ZEB2} strongly downregulates \textit{MMP-9} expression as well as activity \cite{liZEB2PromotesTumor2017a}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CEBP\allowbreak\_{}alpha}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{C/EBP}-$\alpha$ can induce upregulation of \textit{MMP-9} \cite{sarkerMTLCEBPASmallActivating2020}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ECM}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ECM}=\left(\mathbf{COL1\_{}3}\andop\mathbf{Elastin}\right)\orop\mathbf{Stiff\_{}ECM}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{OliveGreen} The \textit{ECM} node represents access to at least a soft extracellular matrix that supports anchorage-dependent survival signaling, though it does not, by itself, support cell spreading or stress fiber formation ($<$ 0.5 kPa) \cite{maMechanotransductionAnoikisDeath2008}. In out COPD model, this node requires the \textit{COL1\_3} input node along with \textit{Elastin}, or it is overriden to an ON state by the \textit{Stiff\_ECM} node (meaning there is ECM).}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{COL1\allowbreak\_{}3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The interstitial ECM of alveoli is composed of a relaxed mesh, dominated by type I and III collagens and elastin \cite{burgstallerInstructiveExtracellularMatrix2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Elastin}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The interstitial ECM of alveoli is composed of a relaxed mesh, dominated by type I and III collagens and elastin \cite{burgstallerInstructiveExtracellularMatrix2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Stiff\allowbreak\_{}ECM}&\multicolumn{1}{m{\linkdspec}}{\textit{ECM} node is automatically ON when cells have access to stiff \textit{ECM}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Stiff\allowbreak\_{}ECM}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Stiff\_{}ECM}=\mathbf{Basement\_{}Membrane}\orop\left(\mathbf{COL1\_{}3}\andop\left(\mathbf{Fibronectin}\orop\notop\mathbf{Elastin}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{OliveGreen} The \textit{Sfiff\_ECM} node represents access to a very stiff extracellular matrix that promotes / supports stress fiber formation sufficiently to place no limitation on a cell’s capacity to proliferate ( $>$ 100 kPa) \cite{maMechanotransductionAnoikisDeath2008}. For alveolar epithelial cells, the basememnt membrane can serve this purpose, but so can a stiffened intersticial ECM outside of the basement membrane, containing excess fibronectin or lacking elastin.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Basement\allowbreak\_{}Membrane}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The elastic or Young's modulus of adult BMs is in the 1-4 MPa range across a range of tissues,  or $\sim$1000 times greater than that of the epithelial cell layer (1-4 kPa) \cite{leclechBasementMembraneStructured2020} or bulk lung tissue (0.44-7.5 kPa) \cite{whiteLungExtracellularMatrix2015}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{COL1\allowbreak\_{}3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} In the absence of a BM, we assume that our model cell experiences a stiff ECM if in contact with the interstitial ECM (mesh dominated by type I and III collagens) \cite{burgstallerInstructiveExtracellularMatrix2017}. Thus, we require \textit{COL1\_3} = ON for all extracellular conditions with ECM contact.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Elastin}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} In the absence of a BM, we assume that our model cell experiences a stiff interstitial ECM, altered from its soft healthy mesh state by the loss of elastin \cite{guoExtracellularMatrixStiffness2022}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Fibronectin}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Accumulation of excess fibronectin increases the density and fibrillar architecture of the ECM, increasing its stiffness \cite{carraherRegulationMatrixAssembly2013, maiModulatingExtracellularMatrix2024}.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Transforming\_{}env module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TGFb\allowbreak\_{}ext}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TGFb\_{}ext}=\mathbf{TGFb\_{}ext}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{TGFb\_ext} node in our model represents an extracellular environment with saturating levels of \textit{TGF-$\beta$}. This input node is self-sustaining in the absence of \textit{in silico} perturbation.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{TGFb\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{The \textit{TGFb\_ext} input node is self-sustaining in the absence of \textit{in silico} perturbation.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SHH\allowbreak\_{}ext}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SHH\_{}ext}=\mathbf{SHH\_{}ext}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue}  The \textit{SHH\_ext} node in our model represents an extracellular environment with saturating levels of \textit{SHH}. This input node is self-sustaining in the absence of \textit{in silico} perturbation.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The \textit{SHH\_ext} input node is self-sustaining in the absence of \textit{in silico} perturbation.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Self\allowbreak\_{}Loop}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Self\_{}Loop}=\mathbf{Self\_{}Loop}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{Self\_Loop} input node qualtifies the extent to which \textit{TGF-$\beta$} secreted by a single isolated cell can drive its own autocrine signaling to saturating levels. When tuned stochastically between 0 and 1, its level represents the fraction of saturating \textit{TGF-$\beta$} signal driven by the autocrine loop. We expect the fraction that accurately characterises a given cell to be a funciton of both cell type (intrinsic ability to secrete \textit{TGF-$\beta$}) and microenvironment, as the latter can influence diffusion and bio-availability of the secreted ligand.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{Self\allowbreak\_{}Loop}&\multicolumn{1}{m{\linkdspec}}{{The \textit{Self\_Loop} input node is self-sustaining in the absence of \textit{in silico} perturbation.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{TGF\_{}beta module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TGF\_{}bRII}=\mathbf{TGFb\_{}ext}\orop\left(\mathbf{TGFb\_{}secr}\andop\left(\mathbf{CellDensity\_{}Low}\orop\mathbf{Self\_{}Loop}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{{The ON state of the \textit{TGF\_bRII} node represents a \textit{TGF-$\beta$}-bound \textit{T$\beta$RII} receptor, and thus requires external \textit{TGF-$\beta$} \cite{seoaneTGFvFamilySignaling2017}, or secreted \textit{TGF-$\beta$} along with either strong autocrine signaling (\textit{Self\_Loop}=ON) or at least medium cell density (\textit{CellDensity\_Low}=ON).}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}Low}&\multicolumn{1}{m{\linkdspec}}{{Though direct experimental evidence is hard to find, our model makes the assumption that autocrine signaling among cells that have at least medium density is strong enough to support autocrine \textit{TGF-$\beta$} signaling, even if a single isolated cell's secretion is not sufficient. The assumption is indirecty supported by evidence that autocrine signaling is required for, and can indeed support the maintenance of a mesencymal state even after external \textit{TGF-$\beta$} is no longer supplied \cite{gregoryAutocrineTGFbetaZEB2011, tianCoupledReversibleIrreversible2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{TGFb\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{Externally supplied \textit{TGF-$\beta$} binds to and activates \textit{T$\beta$RII} (\textit{TGFBR2} gene) \cite{seoaneTGFvFamilySignaling2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Self\allowbreak\_{}Loop}&\multicolumn{1}{m{\linkdspec}}{{In cells / environments where a single cell can secrete sufficient \textit{TGF-$\beta$} to saturate its own \textit{TGF-$\beta$} signaling, this input removes the requirement of neighboring cells to boost the availability of secreted \textit{TGF-$\beta$}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{TGFb\allowbreak\_{}secr}&\multicolumn{1}{m{\linkdspec}}{{Mesenhymal cells secrete \textit{TGF-$\beta$}, creating an autocrine signaling loop required to maintian their mesenchymal state \cite{gregoryAutocrineTGFbetaZEB2011, tianCoupledReversibleIrreversible2013}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TGF\_{}bRI}=\mathbf{TGF\_{}bRII}\andop\left(\mathbf{TGFb\_{}ext}\orop\left(\mathbf{TGFb\_{}secr}\andop\left(\mathbf{CellDensity\_{}Low}\orop\mathbf{Self\_{}Loop}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{{The ON state of \textit{T$\beta$RI}  represents an active, ligand-bound receptor complex of \textit{TGF-$\beta$}, \textit{T$\beta$RII}, and \textit{T$\beta$RI} \cite{seoaneTGFvFamilySignaling2017}. Thus it requires \textit{TGF-$\beta$}-bound \textit{T$\beta$RII}, where \textit{TGF-$\beta$} is either externally supplied or secreted \textit{TGF-$\beta$} (the latter requiring strong autocrine signaling or  medium cell density). }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}Low}&\multicolumn{1}{m{\linkdspec}}{{Our model assumes that autocrine signaling among cells that have at least medium density is strong enough to support autocrine \textit{TGF-$\beta$} signaling, even if a single isolated cell's secretion is not \cite{gregoryAutocrineTGFbetaZEB2011, tianCoupledReversibleIrreversible2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{TGFb\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{Externally supplied \textit{TGF-$\beta$} binds to and activates \textit{T$\beta$RII}, which in turn recruits and phosphorylates \textit{T$\beta$RI} \cite{seoaneTGFvFamilySignaling2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Self\allowbreak\_{}Loop}&\multicolumn{1}{m{\linkdspec}}{{In cells / environments where a single cell can secrete sufficient \textit{TGF-$\beta$} to saturate its own \textit{TGF-$\beta$} signaling, this input removes the requirement of neighboring cells to boost the availability of secreted \textit{TGF-$\beta$}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{When \textit{TGF-$\beta$} binds to its type II receptor (\textit{T$\beta$RII}), it recruits the type I receptor \textit{T$\beta$RI} and activates it by phosphorylation \cite{seoaneTGFvFamilySignaling2017}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{TGFb\allowbreak\_{}secr}&\multicolumn{1}{m{\linkdspec}}{{Mesenhymal cells secrete \textit{TGF-$\beta$}, creating an autocrine signaling loop required to maintian their mesenchymal state \cite{gregoryAutocrineTGFbetaZEB2011, tianCoupledReversibleIrreversible2013}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SMAD2\_{}3\_{}4}=\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\andop\notop\mathbf{SPRY2}\andop\left(\left(\mathbf{YAP}\andop\mathbf{TAZ}\andop\notop\mathbf{Rassf1a}\right)\orop\notop\mathbf{ApicalBasal\_{}Pol}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{SMAD2\_3\_4} node represents transcriptionally active \textit{Smad2/Smad3/Smad4} complexes. \textit{Smad2} and \textit{Smad3} are activated via phosphorylation by \textit{T$\beta$RI}, which releases them from the receptor complex to aid their nuclear translocation in partnership with \textit{Smad4} \cite{seoaneTGFvFamilySignaling2017}. \textit{SPRY2} reduces \textit{Smad} phosphorylation in dense monolayers \cite{tanSprouty2SuppressesEpithelialMesenchymal2016}, which also sequester \textit{TGF-$\beta$} receptors to their baso-lateral surface, hiding them from apically applied \textit{TGF-$\beta$} \cite{nallet-staubCellDensitySensing2015}. In contrast, {\color{NavyBlue}in the absence of \textit{Rassf1a}} \textit{YAP} and \textit{TAZ} bind to \textit{Smads}, aid their nuclear localization, and potentiate \textit{Smad}-induced transcription \cite{szetoYAPTAZAre2016,pefaniTGFvTargetsHippo2016}. Our model assumes that in order to respond to \textit{TGF-$\beta$}, cells either need active \textit{YAP/TAZ}, or no apical-basal polarizaiton.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}DP}}}&\multicolumn{1}{m{\linkcspec}}{SPRY2}&\multicolumn{1}{m{\linkdspec}}{{\textit{SPRY2} supresses \textit{Smad2} phosphorylation and can block \textit{TGF-$\beta$} induced EMT \cite{tanSprouty2SuppressesEpithelialMesenchymal2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP/TAZ} bind to \textit{Smad2} and aid their nuclear accumulation and \textit{Smad}-induced transcription. Fibroblast growth on soft gels with cytosolic \textit{YAP/TAZ} or treated with chemical  \textit{YAP/TAZ} inhibitors show impaired \textit{TGF-$\beta$}–induced \textit{Smad2/3}-driven transcription \cite{szetoYAPTAZAre2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{TAZ}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP/TAZ} bind to \textit{Smads} and aid their nuclear accumulation and \textit{Smad}-induced transcription \cite{szetoYAPTAZAre2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Rassf1a}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Rassf1a} degradation is required for \textit{YAP1} association with SMADs and the nuclear translocation of active \textit{SMAD2} \cite{pefaniTGFvTargetsHippo2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Epithelial cell polarization was shown to block \textit{TGF-$\beta$} signaling  upstream and independently of cytoplasmic \textit{YAP/TAZ} sequestration. In cells polarized along their apical-basal axis (represented in our model by \textit{ApicalBasal\_Pol} = ON), \textit{TGF-$\beta$} receptors I and II are sequestered to the basolateral surface of the cell, depriving apically delivered \textit{TGF-$\beta$} of access to its receptors and thus weakening the \textit{TGF-$\beta$} response \cite{nallet-staubCellDensitySensing2015}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{The actions of \textit{T$\beta$RI} on \textit{Smad2} and \textit{Smad3} require an active receptor/ligand complex, and thus \textit{TGF\_bRII} \cite{seoaneTGFvFamilySignaling2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{The type I \textit{TGF-$\beta$} receptor (\textit{T$\beta$RI}) phosphorylates receptor-bound \textit{Smad2} and \textit{Smad3} at their carboxy-terminal, which releases them from the receptor complex and triggers their nuclear translocation.  \textit{Smad4} partners with activated \textit{Smads} to help carry out their function \cite{seoaneTGFvFamilySignaling2017}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ILK\allowbreak\_{}Rictor}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ILK\_{}Rictor}=\left(\mathbf{ILK}\andop\mathbf{TGF\_{}bRI}\right)\andop\mathbf{TGF\_{}bRII}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{TGF$\beta$-1} induces expression of the \textit{mTORC2} component \textit{Rictor}, \textit{ILK} binding to \textit{Rictor}, and \textit{ILK}-dependent \textit{Rictor} phosphorylation \cite{serranoRoleIntegrinlinkedKinase2013}. The complex, in turn, is known to activate \textit{AKT1} \cite{mcdonaldRictorIntegrinlinkedKinase2008}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{ILK}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$-1} induces \textit{ILK} binding and phosphorylation of \textit{Rictor} \cite{serranoRoleIntegrinlinkedKinase2013}; we assume this requres an active \textit{ILK} kinase. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\textit{ILK/Rictor} binding is mediated by \textit{TGF$\beta$} receptors I and II, responsive to \textit{TGF$\beta$-1} \cite{serranoRoleIntegrinlinkedKinase2013}.  }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\textit{ILK/Rictor} binding is mediated by \textit{TGF$\beta$} receptors I and II, responsive to \textit{TGF$\beta$-1} \cite{serranoRoleIntegrinlinkedKinase2013}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{HMGA2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{HMGA2}=\mathbf{SMAD2\_{}3\_{}4}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{HMGA2} is a transcription factor induced by \textit{SMAD 3/4} in response to \textit{TGF-$\beta$}. Once induced, it drives transcription of the master EMT switch by inducing \textit{SNAI1/2} and \textit{Twist} \cite{thuaultTransformingGrowthFactorbeta2006}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{HMGA2} is a direct transcriptonal target of \textit{TGF-$\beta$} induced \textit{SMAD 3/4} \cite{thuaultTransformingGrowthFactorbeta2006}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{p15}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{p15}=\mathbf{SMAD2\_{}3\_{}4}\orop\left(\left(\mathbf{FoxO1}\orop\mathbf{FoxO3}\right)\andop\notop\mathbf{Myc}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}CDKI}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{p15} (\textit{Ink4B}) is a cyclin-dependent kinase inhibitor that binds \textit{cdk4 / cdk6}, displacing \textit{Cyclin D} and thus blocking G1/S progression \cite{hannonP15INK4BPotentialEffector1994}. \textit{TGF$\beta$} induces both transcription \cite{fengSmad2Smad3Smad42000} and stabilization of \textit{p15} \cite{sandhuTransformingGrowthFactor1997}, while \textit{Myc} represses its transcription \cite{stallerRepressionP15INK4bExpression2001}. In addition, \textit{FoxO1/3} can also induce \textit{p15} \cite{katayamaFOXOTranscriptionFactordependent2008}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO3} is a direct transctiptional activator of \textit{p15} \cite{katayamaFOXOTranscriptionFactordependent2008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO1}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO1} is a direct transctiptional activator of \textit{p15} \cite{katayamaFOXOTranscriptionFactordependent2008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{Smad2}, \textit{Smad3} and \textit{Smad4} induce \textit{p15}  transcription and increase its protein stability in response to \textit{TGF$\beta$} \cite{fengSmad2Smad3Smad42000}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{{\textit{Myc} is a direct transctiptional repressor of \textit{p15} \cite{stallerRepressionP15INK4bExpression2001}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{DUSP4}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{DUSP4}=\mathbf{SMAD2\_{}3\_{}4}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Ph}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{TGF$\beta$} induces the transcription of the mitogen-activated protein kinase  phosphatase \textit{MKP2}, or \textit{DUSP4}, through a \textit{SMAD3}-dependent mechanism \cite{rameshTGFvMediatedBIM2008}. \textit{DUSP4}, in turn, attenuates \textit{ERK} and allows \textit{BIM} accumulation, aiding \textit{TGF$\beta$}-mediated apoptosis \cite{rameshTGFvMediatedBIM2008,wildeySmad3PotentiatesTransforming2003}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$} induces \textit{DUSP4} transcription through \textit{SMAD3} \cite{rameshTGFvMediatedBIM2008}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Runx1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Runx1}=\mathbf{SMAD2\_{}3\_{}4}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{Runx1} transcription factor is transcriptionally induced by \textit{TGF$\beta$}, leading to elevated mRNA and protein levels \cite{wildeyRunx1CoactivatorFOXO32009}. \textit{Runx1} in turns binds to \textit{FoxO3} to induce \textit{BIM} and promote \textit{TGF$\beta$}-mediated apoptosis \cite{wildeyRunx1CoactivatorFOXO32009}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{Either \textit{Smad2} or \textit{Smad3}, plus \textit{Smad4}, are required for induction of \textit{Runx} transcription factors by \textit{TGF$\beta$}1 \cite{ebelFunctionsSmadTranscription2016}.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Hedgehog module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SHH\allowbreak\_{}secr}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SHH\_{}secr}=\left(\left(\mathbf{GLI}\andop\mathbf{NfkB}\right)\orop\mathbf{SMAD2\_{}3\_{}4}\orop\left(\mathbf{GLI\_{}H}\andop\notop\mathbf{GLI3R}\right)\right)\andop\notop\mathbf{Casp3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{Shh} secretion by mesenchymal cell can aid a self-sustaining autocrine signal linked to \textit{TGF-$\beta$} signaling. Namely, \textit{Shh} signaling itself can boost \textit{Shh} secretion via transcriptional activation by \textit{GLI2} (\textit{GLI} node; antagonized by the \textit{GLI3R} repressor) \cite{fengHierarchicalRegulatoryNetwork2022,liMiR130a5pFoxa2Axis2020,metzakopianGenomewideCharacterizationFoxa22012}, potentially aided by \textit{NF-$\kappa$B} \cite{nakashimaNuclearFactorkappaBContributes2006}. In addition, \textit{TGF-$\beta$} can stimulate \textit{Shh} transcription (presumably via \textit{SMAD2/3/4}) \cite{chungCrosstalkShhTGFv2013}. Finally, apoptotic \textit{Caspase 3} activation blocks \textit{Shh} secretion by dismantling vesicular transport between the ER and Golgi \cite{loweCaspasemediatedCleavageSyntaxin2004}. }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of the \textit{FoxA2} transcriptipn factor \cite{fengHierarchicalRegulatoryNetwork2022}, which in turn induces \textit{Shh} mRNa expression in several tissue contexts, including lung branching \cite{liMiR130a5pFoxa2Axis2020}. Moreover, \textit{FoxA2} and \textit{GLI2} bind to the same regulatory region in the \textit{Shh} promoter, suggesting that they may work together to induce \textit{Shh} \cite{metzakopianGenomewideCharacterizationFoxa22012}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of the \textit{FoxA2} transcriptipn factor \cite{fengHierarchicalRegulatoryNetwork2022}, which in turn induces \textit{Shh} mRNa expression in several tissue contexts, including lung branching \cite{liMiR130a5pFoxa2Axis2020}. Moreover, \textit{FoxA2} and \textit{GLI2} bind to the same regulatory region in the \textit{Shh} promoter, suggesting that they may work together to induce \textit{Shh} \cite{metzakopianGenomewideCharacterizationFoxa22012}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI3R}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Activator and repressor forms of the GLI transcription factors share the same DNA binding motifs. Thus, we assume that \textit{GLI2} (the GLI node here) competes with \textit{GLI3R} for access to the GLI binding site on their target genes. Here we assume that \textit{Shh} transcription, secretion, and positive autocrine feedback requires this balance to facor the transcriptional activator \textit{GLI2} \cite{chaudhryGLI3RepressorDetermines2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{NF-$\kappa$B} is a direct transcriptional inducer of the \textit{Shh} gene \cite{kasperczykCharacterizationSonicHedgehog2009}. Inhibition of of\textit{NF-$\kappa$B} can suppress constitutive expression of \textit{Shh} mRNA, whereas its stimulation by inflammatory signals boosts \textit{Shh} production and secretion \cite{nakashimaNuclearFactorkappaBContributes2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{TGF-$\beta$} stimulation induces \textit{Shh} mRNA and protein expression \cite{chungCrosstalkShhTGFv2013}; here we assume that the mechanism involves \textit{SMAD2/3/4} complexes.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} During apopotsis, caspases cleave key components vesicular transport between the ER and Golgi, resulting in a complete block in ER-mediated protein secretion \cite{loweCaspasemediatedCleavageSyntaxin2004}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PTCH1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PTCH1}=\notop\left(\mathbf{SHH\_{}ext}\orop\left(\mathbf{SHH\_{}secr}\andop\left(\notop\mathbf{HHIP}\orop\left(\mathbf{GLI\_{}H}\andop\notop\mathbf{GLI3R}\right)\right)\andop\left(\mathbf{CellDensity\_{}Low}\orop\mathbf{Self\_{}Loop}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} In the absence of Hh ligand binding, the \textit{PTCH1} receptor represses the activity of Smoothened (\textit{SMO}). When Hh ligands such as \textit{Shh} bind to \textit{PTCH1}, the receptor is trafficked to the lysosome for degradation \cite{sigafoosHedgehogSignalingPathway2021}. Here we assume that exogenous, saturating levels of \textit{Shh} (\textit{SHH\_ext} node) can repress \textit{PTCH1}, while secreted \textit{Shh} (\textit{SHH\_secr} node) can only fully repress it in the absence of \textit{HHIP} or under high \textit{Shh} expression conditions (balance between \textit{GLI} activators and repressors strongly favors the former). In addition, we require either the assumption that cells secrete sufficient \textit{Shh} to fully activate the hedgehog pathway (modeled by \textit{Self\_Loop} = ON), or that the cell is not isolated and may also receive \textit{Shh} signals from its neighborhood.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue}  Hh ligands such as \textit{Shh} bind to \textit{PTCH1}, inducing its internalization and lysosomal degradation \cite{sigafoosHedgehogSignalingPathway2021}. Here we assume that exogenous \textit{Shh} can saturate the pathway.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}secr}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue}  Hh ligands such as \textit{Shh} bind to \textit{PTCH1}, inducing its internalization and lysosomal degradation \cite{sigafoosHedgehogSignalingPathway2021}. Here we assume that secreted \textit{Shh} can only saturate the pathway (block \textit{PTCH1}) if not antagonized by \textit{HHIP}, or produced at a higher rate by \textit{GLI} \cite{fengHierarchicalRegulatoryNetwork2022,liMiR130a5pFoxa2Axis2020,metzakopianGenomewideCharacterizationFoxa22012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Here we assume that maximal \textit{GLI2}-mediated transcription and secretion of \textit{Shh} can saturate the pathway by fully blocking \textit{PTCH1} \cite{fengHierarchicalRegulatoryNetwork2022,liMiR130a5pFoxa2Axis2020,metzakopianGenomewideCharacterizationFoxa22012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{GLI3R}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Here we assume that the absence of the \textit{GLI3R} repressor is required for \textit{GLI2}-mediated transcription and secretion of sufficient \textit{Shh} to saturate the autocrine signaling pathway \cite{fengHierarchicalRegulatoryNetwork2022,liMiR130a5pFoxa2Axis2020,metzakopianGenomewideCharacterizationFoxa22012,chaudhryGLI3RepressorDetermines2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{HHIP}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{HHIP} is a membrane glycoprotein that binds to all three mammalian Hedgehog proteins with an affinity comparable to that of \textit{PTCH1} \cite{chuangVertebrateHedgehogSignalling1999}, and its loss upregulates \textit{Hh} signaling during early lung branching \cite{chuangFeedbackControlMammalian2003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Self\allowbreak\_{}Loop}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen} In cells / environments where a single cell can secrete sufficient \textit{SHH} to saturate its own \textit{SHH} signaling (even in the dampening presence of \textit{HHIP}), this input removes the requirement of neighboring cells to boost the availability of secreted \textit{SHH}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}Low}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen} Our model makes the assumption that autocrine signaling among cells that have at least medium density is strong enough to support autocrine \textit{SHH} signaling, even if a single isolated cell's secretion is not sufficient. This matches our handling of \textit{SHH} autocrine signaling, where the above assumption is indirecty supported by evidence that such autocrine signaling is required for, and can indeed support the maintenance of a mesencymal state even after external \textit{SHH} is no longer supplied \cite{gregoryAutocrineTGFbetaZEB2011, tianCoupledReversibleIrreversible2013}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SMO}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SMO}=\left(\notop\mathbf{PTCH1}\right)\andop\left(\mathbf{SHH\_{}secr}\orop\mathbf{SHH\_{}ext}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} In the absence of \textit{PTCH1}, the transmembrane signaling protein \textit{SMO} is phosphorylated. This activates \textit{SMO}, promotes its translocation into the primary cilia where they block phosphorylation and proteolytic cleavage of GLI2/3 (by cAMP-dependent \textit{PKA} and \textit{CK1}, not modeled) \cite{sigafoosHedgehogSignalingPathway2021}. We assume that external \textit{Shh} is required to keep \textit{PTCH1} from blocking \textit{SMO}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{PTCH1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} In the absence of \textit{PTCH1}, \textit{SMO} is phosphorylated and activated by \textit{CK1$\alpha$} and \textit{GRK2} (kinases not modeled explicitly) \cite{sigafoosHedgehogSignalingPathway2021}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that external or secreted \textit{Shh} is required to keep \textit{PTCH1} from blocking \textit{SMO}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}secr}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that external or secreted  \textit{Shh} is required to keep \textit{PTCH1} from blocking \textit{SMO}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SUFU}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SUFU}=\notop\mathbf{SMO}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} Suppressor of fused, or \textit{SUFU}, is a major \textit{GLI} regulator. It binds and sequesters these factorst to the primary cilia, where it aids their phosphorylation by \textit{PKA}, \textit{CK1}, and \textit{GSK3$\beta$}, marking them for proteolytic cleavage \cite{sigafoosHedgehogSignalingPathway2021}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SMO}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Once activated and localized to prolary cilia, \textit{SMO} inhibits the sequestration of \textit{GLI} transcription factors by \textit{SUFU}, sparing them from proteolytic cleavage \cite{sigafoosHedgehogSignalingPathway2021}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{GLI}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{GLI}=\left(\notop\mathbf{SUFU}\orop\notop\mathbf{CEBP\_{}alpha}\orop\mathbf{SMAD2\_{}3\_{}4}\orop\mathbf{S6K}\orop\mathbf{Myc}\right)\andop\mathbf{PAK1}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{GLI1/2} are the main activator transcription factors that mediate \textit{Hh} signaling. They are stabilized and released to the nucleus in the absence of \textit{SUFU} \cite{sigafoosHedgehogSignalingPathway2021} -- overriden by phosphorylation by \textit{S6K1} \cite{wangCrosstalkMTORS6K12012} --, and are induced in a \textit{Hh}-independent manner by the loss of \textit{C/EBP$\alpha$} \cite{basseresRespiratoryFailureDue2006,martisEBPalphaRequiredLung2006} or transcription by \textit{SMAD2/3/4} \cite{dennlerCloningHumanGLI22009,dennlerInductionSonicHedgehog2007}. Finally, \textit{PAK1} activation helps relieve \textit{SUFU}-mediated sequestration of \textit{GLI1/2}, allowing them to accumulate in an active form \cite{tangHedgehogSignalingControlled2022}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SUFU}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SUFU} binds and sequesters these factorst to the primary cilia, where it aids their phosphorylation by \textit{PKA}, \textit{CK1}, and \textit{GSK3$\beta$}, marking them for proteolytic cleavage \cite{sigafoosHedgehogSignalingPathway2021}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CEBP\allowbreak\_{}alpha}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Loss of \textit{C/EBP$\alpha$} increases \textit{GLI1}, including in alveolar cells \cite{basseresRespiratoryFailureDue2006,martisEBPalphaRequiredLung2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{S6K}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{S6K1}-mediated \textit{GLI1} phosphorylation at Ser84 releases \textit{GLI1} from its endogenous inhibitor, \textit{SUFU} \cite{wangCrosstalkMTORS6K12012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{TGF-$\beta$} induces \textit{GLI2} via \textit{SMAD2/3/4}-mediated transcription \cite{dennlerCloningHumanGLI22009,dennlerInductionSonicHedgehog2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{c-Myc} is a direct transcriptional inducer of \textit{GLI1} \cite{yoonNoncanonicalRegulationHedgehog2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Rac1}-dependent active \textit{PAK1} dampens \textit{SUFU-GLI} complex formation and thus aids \textit{GLI} activation \cite{tangHedgehogSignalingControlled2022}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{GLI\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{GLI\_{}H}=\mathbf{GLI}\andop\left(\notop\mathbf{SUFU}\orop\mathbf{SMAD2\_{}3\_{}4}\right)\andop\left(\notop\mathbf{CEBP\_{}alpha}\orop\mathbf{SHH\_{}ext}\orop\left(\mathbf{SHH\_{}secr}\andop\notop\mathbf{HHIP}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{GLI1/2} are the main activator transcription factors that mediate \textit{Hh} signaling. Here we model their activity levels with two Boolean nodes: \textit{GLI} = ON and \textit{GLI\_H} = OFF represents moderate activation, whereas \textit{GLI\_H} = ON represents high levels of \textit{GLI}-mediated transcription. For model consistency, this requires \textit{GLI} = ON. In additon, we assume that saturating \textit{GLI\_H} activity is aided by ongoing \textit{Shh} signaling (i.e., absence of \textit{SUFU}, presence of a strong \textit{Shh} signal), as well increased transcription by \textit{SMAD2/3/4} \cite{dennlerCloningHumanGLI22009,dennlerInductionSonicHedgehog2007} or loss of \textit{C/EBP$\alpha$} \cite{basseresRespiratoryFailureDue2006,martisEBPalphaRequiredLung2006}. }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI\_H} = ON requires \textit{GLI} = ON, making sure that the two nodes together represent three distinct levels of \textit{GLI} activity.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{SUFU}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SUFU} binds and sequesters these factorst to the primary cilia, where it aids their phosphorylation by \textit{PKA}, \textit{CK1}, and \textit{GSK3$\beta$}, marking them for proteolytic cleavage \cite{sigafoosHedgehogSignalingPathway2021}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CEBP\allowbreak\_{}alpha}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Loss of \textit{C/EBP$\alpha$} increases \textit{GLI1}, including in alveolar cells \cite{basseresRespiratoryFailureDue2006,martisEBPalphaRequiredLung2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{TGF-$\beta$} induces \textit{GLI2} via \textit{SMAD2/3/4}-mediated transcription \cite{dennlerCloningHumanGLI22009,dennlerInductionSonicHedgehog2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}ext}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that saturating \textit{GLI\_H} activity is aided by ongoing expgenous \textit{Shh} signaling.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SHH\allowbreak\_{}secr}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that saturating \textit{GLI\_H} activity is aided by ongoing secreted \textit{Shh}, unrestrained by \textit{HHIP} \cite{chuangVertebrateHedgehogSignalling1999}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{HHIP}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that saturating \textit{GLI\_H} activity is aided by ongoing secreted \textit{Shh}, unrestrained by \textit{HHIP} \cite{chuangVertebrateHedgehogSignalling1999}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{GLI3R}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{GLI3R}=\notop\left(\mathbf{GLI\_{}H}\orop\left(\mathbf{GLI}\andop\mathbf{GSK3}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} The \textit{GLI3} gene encodes a strong repressor, the proteolytically processed active form of which is termed \textit{GLI3R}. This truncated protein antagonizes HH signalling, and it is dominant in the absence of HH ligands. Upon \textit{SMO} activation, the activators \textit{GLI1/2} are transcribed, while \textit{GLI3R} formation is repressed \cite{varnatLossWNTTCFAddiction2010}. Thus, we assume that \textit{GLI3R} is inactive in the presence of high \textit{GLI1/2} \cite{chaudhryGLI3RepressorDetermines2017}, or moderate \textit{GLI} paired with a  \textit{GSK3} = ON cell state \cite{trnskiGSK3vGli3Play2015}. }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} As activator and repressor forms of the GLI share the same DNA binding motifs, we assume that \textit{GLI\_H} = ON represents a state where the activators robustly outcompete the repressor forms \cite{chaudhryGLI3RepressorDetermines2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that moderate \textit{GLI1/2} can outcompete \textit{GLI3R} only with the aid of \textit{Myc}-mediated repression of \textit{GLI3}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The Ser-9 phosphorylated, inactivated form of \textit{GSK3$\beta$} (corresponding to \textit{GSK3} = OFF in our model) can nevertheless phosphorylate the unprocessed form of \textit{GLI3}, aiding its conversion into the repressor form \textit{GLI3} \cite{trnskiGSK3vGli3Play2015}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{HHIP}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{HHIP}=\mathbf{GLI}\orop\mathbf{GLI\_{}H}\orop\mathbf{SMAD2\_{}3\_{}4}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{HHIP} is a membrane glycoprotein that binds to all three mammalian Hedgehog proteins with an affinity comparable to that of \textit{PTCH1} \cite{chuangVertebrateHedgehogSignalling1999}, and its loss upregulates \textit{Hh} signaling \cite{chuangFeedbackControlMammalian2003}. It is transcriptionally induced by \textit{GLI} proteins as part of a homeostatic negative feedback loop, as well as by \textit{TGF-$\beta$} via \textit{SMAD2/3/4}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{HHIP} is a \textit{GLI}-activated, Hh induced gene \cite{abergerContextdependentSignalIntegration2014}; \textit{GLI} binds to an enhancer region of the \textit{HHIP} gene \cite{vokesGenomicCharacterizationGliactivator2007,vokesGenomescaleAnalysisCisregulatory2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{HHIP} is a \textit{GLI}-activated, Hh induced gene \cite{abergerContextdependentSignalIntegration2014}; \textit{GLI} binds to an enhancer region of the \textit{HHIP} gene \cite{vokesGenomicCharacterizationGliactivator2007,vokesGenomescaleAnalysisCisregulatory2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SMAD3} binds to a distant enhancer of the \textit{HHIP} gene and upregulates it in response to \textit{TGF-$\beta$} \cite{guoIdentificationDistalEnhancer2024}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CEBP\allowbreak\_{}alpha}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CEBP\_{}alpha}=\notop\left(\mathbf{SMAD2\_{}3\_{}4}\orop\mathbf{CyclinE}\orop\left(\mathbf{CyclinA}\andop\mathbf{Cdc25A}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{C/EBP$\alpha$} is a transcriptipn factor that is required for the maturation of the respiratory epithelium in late gestation. It's loss can alter surfactant secretion (not modeled), and altered the biomechanical environment of the lung \cite{martisEBPalphaRequiredLung2006}. }}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The \textit{CEBPA} gene is a direct target of \textit{SMAD3} represssion during \textit{TGF-$\beta$}-mediated EMT \cite{lourencoEBPaCrucialDeterminant2020a}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Cdk2} (the activity of which is imlicitly modeled as \textit{CyclinE} = ON or \textit{CyclinA} and \textit{Cdc25A} = ON) negatively regulates \textit{C/EBP$\alpha$} protein levels by ubiquitin-mediated proteasome degradation \cite{thackerCDK2DestabilizesTumor2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Cdk2} (the activity of which is imlicitly modeled as \textit{CyclinE} = ON or \textit{CyclinA} and \textit{Cdc25A} = ON) negatively regulates \textit{C/EBP$\alpha$} protein levels by ubiquitin-mediated proteasome degradation \cite{thackerCDK2DestabilizesTumor2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Cdk2} (the activity of which is imlicitly modeled as \textit{CyclinE} = ON or \textit{CyclinA} and \textit{Cdc25A} = ON) negatively regulates \textit{C/EBP$\alpha$} protein levels by ubiquitin-mediated proteasome degradation \cite{thackerCDK2DestabilizesTumor2020}.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{GrowthFactor\_{}Env module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{GF}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{GF}=\mathbf{GF}\orop\mathbf{GF\_{}High}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{GF} node represents an extracellular environment with low levels of growth factors capable of sustaining survival signaling. Thus, the \textit{GF} input node is self-sustaining in the absence of in silico perturbation.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{GF}&\multicolumn{1}{m{\linkdspec}}{The \textit{GF} input node is self-sustaining in the absence of in silico perturbation.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{GF\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{The \textit{GF} node represents an extracellular environment with low levels of growth factors capable of sustaining survival signaling. Thus, this node is ON in high growth factor as well.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{GF\allowbreak\_{}High}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{GF\_{}High}=\mathbf{GF\_{}High}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{GF\textsubscript{High}} node in our model represents an extracellular environment with saturating levels of growth factors; this input node is self-sustaining in the absence of \textit{in silico} perturbation.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{GF\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{The \textit{GF\textit{\textsubscript{High}}} input node is self-sustaining in the absence of \textit{in silico} perturbation.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{GF\_{}Basal\_{}MAPK module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{RTK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{RTK}=\left(\notop\mathbf{CAD}\right)\andop\left(\mathbf{GF\_{}High}\orop\mathbf{GF}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{The ON state of the \textit{RTK} node in our model represents basal growth receptor activation (required to keep a normal cell alive). Thus it requires the absence of \textit{CAD} and at least low growth levels of growth factors in the extracellular environment \cite{Lemmon:2010bq}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{GF}&\multicolumn{1}{m{\linkdspec}}{The ON state of the \textit{RTK} node in our model represents basal growth receptor activation by low / medium growth factor availability, encoded by the \textit{GF} node (required to keep a normal cell alive).}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{GF\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{Similarly, high growth factor availability also keeps \textit{RTK} on.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{CAD}&\multicolumn{1}{m{\linkdspec}}{Caspase-activated DNase (\textit{CAD}) inhibition of receptor tyrosine kinases ensures that apoptotic cells no longer maintain even basal levels of growth signaling.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Shc}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Shc}=\left(\left(\mathbf{RTK}\andop\mathbf{GF\_{}High}\right)\orop\left(\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\right)\right)\andop\left(\mathbf{FAK}\orop\mathbf{Src}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Adap}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{Shc} is ON when \textit{RTK}s are activated by high levels of extracellular growth factors (capable of driving proliferation) \cite{ahmedInsightsShcFamily2017} {or \textit{TGF$\beta$} activates \textit{T$\beta$RI-T$\beta$RII} \cite{zhangNonSmadPathwaysTGFbeta2009}}, and its recruitment is aided by active \textit{Src} kinase \cite{schlaepferMultipleGrb2mediatedIntegrinstimulated1998} or focal adhesion kinase (\textit{FAK}) \cite{schlaepferMultipleGrb2mediatedIntegrinstimulated1998}. The two mediators of integrin signaling appears to be able to act independently, forming two parallel links between integrin signaling and full \textit{RTK} activation \cite{schlaepferMultipleGrb2mediatedIntegrinstimulated1998}.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{GF\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{{The ON state of \textit{Shc} in our model encode the change from basal \textit{Shc} recruitment to weakly stimulated \textit{RTKs} to the level of recruitment seen in high growth factor environments, capable of mediating \textit{Ras} activation.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{{\textit{Shc} proteins are adaptors that binds to phosphor-tyrosine motifs, facilitating their recruitment to activated receptors such as receptor tyrosine kinases \textit{RTKs} \cite{ahmedInsightsShcFamily2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{FAK}&\multicolumn{1}{m{\linkdspec}}{{\textit{FAK} activated at sites of integrin-ECM attachments directly phosphorylates \textit{Shc Tyr-317}, promoting its ability to assemble MAPK-inducing signaling scaffolds, including \textit{Grb2} binding \cite{schlaepferMultipleGrb2mediatedIntegrinstimulated1998}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Src}&\multicolumn{1}{m{\linkdspec}}{{\textit{c-Src} recruited to and activated by integrin-ECM attachments directly phosphorylate \textit{Shc}, promoting its ability to assemble MAPK-inducing signaling scaffolds, including \textit{Grb2} binding \cite{schlaepferMultipleGrb2mediatedIntegrinstimulated1998}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$} receptor II (\textit{T$\beta$RII}) can be phosphorylated by \textit{Src} on Y284, which provides a docking site for \textit{Grb2} and \textit{Shc}, and subsequent MAPK activation \cite{zhangNonSmadPathwaysTGFbeta2009}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\textit{T$\beta$RII} activation requires complex formation with \textit{T$\beta$RI} upon ligand binding \cite{zhangNonSmadPathwaysTGFbeta2009}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Grb2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Grb2}=\left(\mathbf{RTK}\orop\left(\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\right)\right)\andop\mathbf{Shc}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Adap}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Grb2} is recruited to \textit{RTKs} {or \textit{TGF$\beta$}} receptors upon ligand binding and subsequent recruitment of \textit{Shc} adaptors \cite{Uzman:2001iy, zhangNonSmadPathwaysTGFbeta2009}.  }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{The SH2 domain of \textit{Grb2} binds to a phosphotyrosine residue in the activated \textit{RTK}, where it functions as an adaptor protein \cite{Uzman:2001iy}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Shc}&\multicolumn{1}{m{\linkdspec}}{{\textit{Shc} proteins phosphorylated by tyrosine kinases represent binding sites for \textit{Grb2}, aiding its recruitment to active RTKs \cite{ahmedInsightsShcFamily2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\textit{T$\beta$RII} phosphorylated by \textit{Src} on Y284, provides a docking site for \textit{Grb2} and \textit{Shc} \cite{zhangNonSmadPathwaysTGFbeta2009}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\textit{T$\beta$RII} works in complex  with \textit{T$\beta$RI} and their ligand \cite{zhangNonSmadPathwaysTGFbeta2009}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SOS}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SOS}=\mathbf{Grb2}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GEF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{RTK}-bound \textit{Grb2} recruits \textit{SOS}, a guanine nucleotide–exchange protein (GEF) that converts inactive \textit{Ras} to its active GTP-bound form \cite{Uzman:2001iy}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Grb2}&\multicolumn{1}{m{\linkdspec}}{\textit{Grb2} recruits \textit{SOS} to activated \textit{RTK}s \cite{Uzman:2001iy}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Ras}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Ras}=\left(\left(\mathbf{Grb2}\andop\mathbf{SOS}\right)\andop\mathbf{Src}\right)\andop\left(\left(\mathbf{IQGAP1\_{}LeadingE}\orop\left(\notop\mathbf{Merlin}\right)\right)\orop\mathbf{N\_{}bcatenin\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GTPa}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Ras} activation requires the GEF activity of \textit{SOS} and the \textit{RTK}-linked (active) adaptor protein \textit{Grb2} \cite{Uzman:2001iy} and \textit{Src} \cite{belschesRoleCSrcTyrosine1997,rocksAcylationCycleRegulates2005}. In addition to aiding sustained \textit{Ras/Raf-1} signaling, \textit{Src}  may also physically link \textit{IQGAP1} to \textit{RTKs} such as \textit{VEGFR2} \cite{meyerIQGAP1dependentSignalingPathway2008}. \textit{IQGAP1}, in turn, serves a scaffold for \textit{MAPK} and \textit{PI3K} signaling \cite{choiAktionIQGAP1Control2017}, leading us to link its activation at the leading edge. In contrast, \textit{Merlin} blocks \textit{Ras} activation at sites that link focal adhesions and actin filaments to \textit{MAPK} signaling \cite{morrisonMerlinNeurofibromatosisType2007}. Here we assume that concentrated \textit{IQGAP1} at the leading edge can override remaining \textit{Merlin} activity in the rest of the cell. {Alternatively, high levels of \textit{$\beta$-catenin} can also sustain \textit{Ras} by protecting it from lysosomal degradation \cite{jeonAxinInhibitsExtracellular2007}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Grb2}&\multicolumn{1}{m{\linkdspec}}{\textit{RTK}-bound \textit{Grb2} is required to recruits \textit{SOS}, the GEF responsible for converting inactive \textit{Ras} to its GTP-bound active form \cite{Uzman:2001iy}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}GEF}}}&\multicolumn{1}{m{\linkcspec}}{SOS}&\multicolumn{1}{m{\linkdspec}}{\textit{SOS} is a GEF that is recruited to activate \textit{Ras} near ligand-bound, active \textit{RTKs} \cite{Uzman:2001iy}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Src}&\multicolumn{1}{m{\linkdspec}}{{Cellular Src (\textit{c-Src}) is required for mitogenesis initiated by multiple growth factor receptors, including epidermal growth factor (\textit{EGF}), platelet-derived growth factor (\textit{PDGF}), colony stimulating factor-1 (\textit{CSF-1}), and basic fibroblast growth factor (\textit{bFGF}) \cite{belschesRoleCSrcTyrosine1997}. In addition to aiding the formation of Shc/Grb2/SOS/Ras/Raf-1 cascade, \textit{Src} may also increase the rate of receptor internalization and aid sustained \textit{MAPK} signaling by internalized \textit{Ras} on endosomes and Golgi \cite{belschesRoleCSrcTyrosine1997,rocksAcylationCycleRegulates2005}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} uncouples \textit{Ras} from growth factor signals by counteracting the ERM (ezrin, radixin, moesin)–dependent activation of \textit{Ras}, which aids \textit{Grb2}, \textit{SOS}, \textit{Ras} complex formation linked to filamentous actin \cite{morrisonMerlinNeurofibromatosisType2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{IQGAP1\allowbreak\_{}LeadingE}&\multicolumn{1}{m{\linkdspec}}{{\textit{IQGAP1} acts as a scaffold for the \textit{MAPK} cascade, binding directly to \textit{B-Raf}, \textit{MEK}, and \textit{ERK} and regulating their activation \cite{mcnultyMAPKScaffoldIQGAP12011}. The \textit{IQGAP1-Leading-E} node in our model specifically links the availability of active \textit{IQGAP1} recruited to lamellipodia and enhanced \textit{MAPK} / \textit{AKT} signaling.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{High levels of $\beta$-\textit{catenin} protect \textit{Ras} from  lysosomal degradation \cite{jeonAxinInhibitsExtracellular2007}. Overall, $\beta$-\textit{catenin} overexpression / silencing can activate / block \textit{ERK} in a \textit{MEK}-dependent way, like via \textit{Ras/Raf} \cite{yunBothERKWnt2005}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{RAF}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{RAF}=\left(\left(\notop\mathbf{Casp3}\right)\andop\mathbf{Ras}\right)\andop\left(\notop\mathbf{SPRY2}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Raf} is active in response to \textit{Ras} activity in the absence of \textit{Caspase 3}. As active \textit{Raf-1} is continuously dephosphorylated and bound by \textit{14-3-3}, which translocates it to the cytoplasm from the plasma membrane (not modeled explicitly), ongoing \textit{Ras} activity \cite{Chang:2003cy} and lack of \textit{SPRY2} inhibition \cite{yusoffSprouty2InhibitsRas2002,bradySprouty2AssociationBRaf2009} are necessary to keep \textit{Raf} ON.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Ras}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Ras} phosphorylates \textit{Raf}, enhancing its kinase activity \cite{Chang:2003cy}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{SPRY2}&\multicolumn{1}{m{\linkdspec}}{{Sprouty2 (\textit{SPRY2}) blocks Raf activity and downstream MAPK signaling \cite{yusoffSprouty2InhibitsRas2002,bradySprouty2AssociationBRaf2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Raf-1} is cleaved and inhibited by \textit{Caspase 3} \cite{Widmann:1998tm}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{MEK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{MEK}=\mathbf{RAF}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Raf} phosphorylates and activates the \textit{MEK} kinase \cite{Chang:2003cy}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{RAF}&\multicolumn{1}{m{\linkdspec}}{\textit{Raf} phosphorylates and activates the \textit{MEK} kinase \cite{Chang:2003cy}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ERK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ERK}=\left(\mathbf{MEK}\andop\left(\notop\mathbf{BIK}\right)\right)\andop\left(\left(\mathbf{FocalAdhesions}\orop\left(\notop\mathbf{DUSP4}\right)\right)\orop\mathbf{N\_{}bcatenin\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{ERK} kinase is active when phoisphotylated by \textit{MEK} \cite{Chang:2003cy} and allowed to translocate to the nucleus in the absence of \textit{BIK} \cite{Mebratu:2008gn}. {Here we assume that in the presence of \textit{DUSP4} \cite{rameshTGFvMediatedBIM2008,wildeySmad3PotentiatesTransforming2003}, \textit{ERK} activity can be maintained /compensated by either strong focal adhesion assembly (\textit{FocalAdhesions} node) \cite{yinGIT1ScaffoldERK12005}, or high levels of $\beta$-\textit{catenin} \cite{yunBothERKWnt2005}. } }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{MEK}&\multicolumn{1}{m{\linkdspec}}{\textit{MEK} phosphorylates and activates the \textit{ERK} kinase \cite{Chang:2003cy}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{Focal adhesions recruit the MAPK scaffolding protein GIT1 and locally potentiate \textit{ERK1/2} activation \cite{yinGIT1ScaffoldERK12005}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}DP}}}&\multicolumn{1}{m{\linkcspec}}{DUSP4}&\multicolumn{1}{m{\linkdspec}}{{\textit{DUSP4} is a phosphatase that blocks \textit{ERK} and allows \textit{BIM} accumulation in response to \textit{TGF$\beta$}, aiding apoptosis \cite{rameshTGFvMediatedBIM2008,wildeySmad3PotentiatesTransforming2003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{$\beta$-\textit{catenin} overexpression / silencing can activate / block \textit{ERK} in a \textit{MEK}-dependent way \cite{yunBothERKWnt2005}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BIK}&\multicolumn{1}{m{\linkdspec}}{\textit{BIK} binds to active, phosphorylated \textit{ERK1/2} and suppresses its nuclear translocation \cite{Mebratu:2008gn}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{MCRIP1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{MCRIP1}=\notop\mathbf{ERK}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{MCRIP1} is a binding partner and repressor of \textit{CtBP}, a binding partner  of \textit{Zeb1} during transcription , and thus \textit{MCRIP1}  blocks \textit{Zeb1}-mediated repression of \textit{E-cadherin} unless inhibited by  \textit{ERK} phosphorylation \cite{ichikawaMCRIP1ERKSubstrate2015}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{{Upon \textit{ERK} phosphorylation \textit{MCRIP1}, an inhibitor of EMT, dissociates and releases the \textit{ZEB1} co-activator \textit{CtBP} \cite{ichikawaMCRIP1ERKSubstrate2015}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{mTORC2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{mTORC2}=\mathbf{PIP3}\orop\left(\notop\mathbf{S6K}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{Our model assumes that \textit{mTORC2} is active in quiescent cells with basal levels of \textit{PI3K} activity leading to basal \textit{PIP3} generation. Alternatively, the absence of high growth factor-stimulated \textit{mTORC1} and \textit{S6K1} can also increase \textit{mTORC2} activity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PBind}}}&\multicolumn{1}{m{\linkcspec}}{PIP3}&\multicolumn{1}{m{\linkdspec}}{PtdIns(3,4,5)P3 (\textit{PIP3}), interacts with the \textit{mTORC2} component \textit{Sin1} to release its inhibition on the \textit{mTOR} kinase domain. Thus, \textit{PIP3} is necessary for \textit{mTORC2} activation \cite{Liu:2015ey}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{S6K}&\multicolumn{1}{m{\linkdspec}}{\textit{Rictor}, a component of the \textit{mTORC2 complex}, undergoes \textit{S6K1}-mediated phosphorylation at T1135, dampening \textit{mTORC2}-dependent phosphorylation of \textit{Akt} \cite{Zhang:2006bw,Dibble:2009gh}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PI3K}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PI3K}=\left(\left(\mathbf{FAK}\orop\mathbf{Src}\orop\mathbf{GLI}\right)\andop\left(\mathbf{Ras}\orop\mathbf{RTK}\right)\right)\orop\left(\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, basal \textit{PI3K} activity can be maintained by active \textit{RTK}s \cite{Lemmon:2010bq}, active \textit{Ras} \cite{RodriguezViciana:1994kn,Gupta:2007dw}, {or active \textit{T$\beta$RI-T$\beta$RII} \cite{zhangSignalingInterplayTransforming2013}}. In addition, survival signalling via \textit{PI3K} requires anchorage-dependent signals via active \textit{FAK} \cite{khwajaMatrixAdhesionRas1997}, via \textit{Src}-mediated blocking of basal \textit{PTEN} activity (not modeled explicitly) \cite{luSrcFamilyProteintyrosine2003}, or {\color{NavyBlue} a \textit{GLI1}-mediated boost to \textit{PI3K} activity \cite{zhouGLI1ReducesDrug2021}}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{Active \textit{RTK}s recruit \textit{PI3K} to the signaling complex they nucleate, where \textit{PI3K} catalyzes the production of PtdIns(3,4,5)P3 (PIP3) \cite{Lemmon:2010bq}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Ras}&\multicolumn{1}{m{\linkdspec}}{\textit{Ras} binds the catalytic subunit of \textit{PI3K} and \textit{Ras} knockdown / over expression decreases /increases the \textit{PI3K}-dependent generation of PIP3 \cite{RodriguezViciana:1994kn,Gupta:2007dw}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{FAK}&\multicolumn{1}{m{\linkdspec}}{{Attachment to the ECM activates \textit{FAK} kinase, which promotes anchorage-dependent survival signaling via \textit{PI3K} / \textit{AKT} \cite{khwajaMatrixAdhesionRas1997}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Src}&\multicolumn{1}{m{\linkdspec}}{{\textit{Src} kinases regulate \textit{PI3K} signaling cascade by altering the function of the \textit{PTEN} tumor suppressor via inhibitory phosphorylation \cite{luSrcFamilyProteintyrosine2003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF}$\beta$ induces \textit{PI3K} activation and \textit{AKT} phosphorylatio, a pathway required for EMT \cite{bakinPhosphatidylinositol3kinaseFunction2000}. This activation requires \textit{T$\beta$RI} and \textit{T$\beta$RII} kinase activity \cite{zhangSignalingInterplayTransforming2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF}$\beta$ induced \textit{PI3K} activation requires \textit{T$\beta$RI} and \textit{T$\beta$RII} kinase activity \cite{bakinPhosphatidylinositol3kinaseFunction2000,zhangSignalingInterplayTransforming2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Overexpression of \textit{GLI1} increases the phosphorylation of \textit{PI3K} and \textit{AKT}, while stable knockdown of \textit{GLI1} attenuates it \cite{zhouGLI1ReducesDrug2021}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PIP3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PIP3}=\mathbf{PI3K\_{}H}\orop\mathbf{PI3K}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Met}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, PIP3 is ON as a result of basal or high \textit{PI3K} activity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Cat}}}&\multicolumn{1}{m{\linkcspec}}{PI3K}&\multicolumn{1}{m{\linkdspec}}{Active \textit{PI3K} recruited to the membrane catalyzes the production of membrane-bound PtdIns(3,4,5)P3 (PIP3) from PtdIns(4,5)P2 (PIP2)  \cite{Lemmon:2010bq}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Cat}}}&\multicolumn{1}{m{\linkcspec}}{PI3K\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Active \textit{PI3K}  recruited to the membrane catalyzes the production of membrane-bound PtdIns(3,4,5)P3 (PIP3) from PtdIns(4,5)P2 (PIP2)  \cite{Lemmon:2010bq}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PDK1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PDK1}=\mathbf{PI3K}\andop\mathbf{PIP3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{PDK1} enzyme activation requires active (at least basal) \textit{PI3K} and \textit{PIP3} \cite{Manning:2017di}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{PI3K}&\multicolumn{1}{m{\linkdspec}}{The \textit{PDK1} kinase is recruited to the plasma membrane by \textit{PIP3} at the sites of active \textit{PI3K} activity \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{PIP3}&\multicolumn{1}{m{\linkdspec}}{The \textit{PDK1} kinase is recruited to the plasma membrane by \textit{PIP3} at the sites of active \textit{PI3K} activity \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{AKT\allowbreak\_{}B}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{AKT\_{}B}=\left(\left(\notop\mathbf{Casp3}\right)\andop\mathbf{PIP3}\right)\andop\left(\mathbf{PDK1}\orop\mathbf{mTORC2}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{Basal \textit{AKT1} activity in our model requires the absence of \textit{Caspase 3}, the availability of at least basal levels of \textit{PIP3}, and phosphorylation by \textit{PDK1} or \textit{mTORC2}. In contrast, full mitogen-stimulated \textit{AKT1} activation requires phosphorylation by both (see \textit{AKT\_H}) \cite{Manning:2017di}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{mTORC2}&\multicolumn{1}{m{\linkdspec}}{Maximal activation of \textit{AKT1} requires phosphorylation of S473 by \textit{mTORC2} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{PIP3}&\multicolumn{1}{m{\linkdspec}}{\textit{PIP3} recruits \textit{AKT1} to the plasma membrane and \textit{PIP3} binding changes the conformation of \textit{AKT1} such that it becomes accessible for T308 phosphorylation by \textit{PDK1} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PDK1}&\multicolumn{1}{m{\linkdspec}}{Membrane-recruited \textit{PDK1} phosphorylates \textit{AKT1}  at T308, a critical step in its activation \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{AKT1} is cleaved and inhibited by \textit{Caspase 3} \cite{Widmann:1998tm}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{GF\_{}PI3K module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{p110\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{p110\_{}H}=\mathbf{YAP}\andop\left(\left(\mathbf{FoxO3}\andop\left(\notop\mathbf{Nedd4L}\right)\right)\orop\left(\mathbf{p110\_{}H}\andop\left(\mathbf{FoxO3}\orop\left(\notop\mathbf{Nedd4L}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{As \textit{YAP} is a transcriptional inducer of \textit{p110} subunits \cite{linPi3kcbLinksHippoYAP2015} and their expression is low in high cell density areas where \textit{YAP} activity is suppressed \cite{Yuan:2011dm}, here we assume that high \textit{p110} expression requires active \textit{YAP}. In order to capture the cyclic dynamics of \textit{p110} protein expression, we make the assumption that high \textit{p110} protein levels can be induced by \textit{FoxO3} in the absence of the growth factor-activated \textit{Nedd4L} ubiquitin ligase. Once present, high \textit{p110} can be maintained by \textit{FoxO3} transcription, or the absence of activated \textit{Nedd4L}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{p110\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Our model assumes that maintaining high \textit{p110} levels is easier than driving the re-accumulation of the protein following its rapid destruction.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO3} is a direct inducer \textit{p110$\alpha$} (\textit{PIK3CA}), the catalytic subunit of \textit{PI3K} \cite{Hui:2008jz}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ubiq}}}&\multicolumn{1}{m{\linkcspec}}{Nedd4L}&\multicolumn{1}{m{\linkdspec}}{\textit{p110$\alpha$} (\textit{PIK3CA}) is polyubiquitinated by the E3 ligase \textit{Nedd4L}, leading to its proteasomal degradation. Both free \textit{p110$\alpha$}  and the regulatory subunit-bound protein is subject to ubiquitination by \textit{Nedd4L} \cite{Wang:2016ej}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} is a transcriptional inducer of both catalytic \textit{p110} subunits of \textit{PI3K}, \textit{p110a} and \textit{p110b}; \textit{p110-H} albeit its effect on \textit{p110a} expression requires raising \textit{p110b} levels first. Moreover, \textit{YAP} knockdown leads to downregulation of both subunits \cite{linPi3kcbLinksHippoYAP2015}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PI3K\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PI3K\_{}H}=\left(\left(\left(\left(\notop\mathbf{PTEN\_{}c}\right)\andop\mathbf{p110\_{}H}\right)\andop\left(\mathbf{RTK}\orop\left(\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\right)\right)\right)\andop\mathbf{PI3K}\right)\andop\left(\mathbf{Ras}\orop\mathbf{GLI\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{Full, peak-level activation of \textit{PI3K} requires high levels of \textit{p110} protein, basal \textit{PI3K} activation, active \textit{Ras}, and active \textit{RTK}s {or \textit{T$\beta$RI-T$\beta$RII}}. As the ON-state of \textit{Ras} in our model represents strong \textit{Ras} activation in the presence of proliferation-inducing (high) growth factors, \textit{PI3K\_H} activation can only occur with either high \textit{Ras} {or \color{NavyBlue} a \textit{GLI1}-mediated boost to \textit{PI3K} activity \cite{zhouGLI1ReducesDrug2021}}. In addition, a reduction of cytoplasmic \textit{PTEN} levels is also required for peak \textit{PI3K} activity. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{High levels of \textit{PI3K} activation only occur at growth factor-bound \textit{RTK}s, which recruit and activate \textit{PI3K} at the plasma membrane \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Ras}&\multicolumn{1}{m{\linkdspec}}{\textit{Ras} binds the catalytic subunit of \textit{PI3K} and \textit{Ras} knockdown / over expression decreases /increases the \textit{PI3K}-dependent generation of PIP3 \cite{RodriguezViciana:1994kn,Gupta:2007dw}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{PI3K}&\multicolumn{1}{m{\linkdspec}}{In our model, high \textit{PI3K} activation is contingent on the ON-state of the basal \textit{PI3K} node.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{p110\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{High levels of \textit{PI3K} activity in response to strong growth factor stimulation only occur in cells that express high levels of \textit{p110} protein \cite{Yuan:2011dm}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}DP}}}&\multicolumn{1}{m{\linkcspec}}{PTEN\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{Cytoplasmic \textit{PTEN} regulates \textit{PI3K} signaling by dephosphorylating its lipid signaling intermediate \textit{PIP3} \cite{chalhoubPTENPI3KinasePathway2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF}$\beta$ induced \textit{PI3K} activation requires \textit{T$\beta$RI} and \textit{T$\beta$RII} kinase activity \cite{bakinPhosphatidylinositol3kinaseFunction2000,zhangSignalingInterplayTransforming2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF}$\beta$ induced \textit{PI3K} activation requires \textit{T$\beta$RI} and \textit{T$\beta$RII} kinase activity \cite{bakinPhosphatidylinositol3kinaseFunction2000,zhangSignalingInterplayTransforming2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Overexpression of \textit{GLI1} increases the phosphorylation of \textit{PI3K} and \textit{AKT}, while stable knockdown of \textit{GLI1} attenuates it \cite{zhouGLI1ReducesDrug2021}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{AKT\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{AKT\_{}H}=\left(\left(\left(\left(\left(\mathbf{AKT\_{}B}\andop\mathbf{p110\_{}H}\right)\andop\left(\mathbf{PI3K\_{}H}\orop\left(\mathbf{PI3K}\andop\mathbf{ILK\_{}Rictor}\right)\right)\right)\andop\mathbf{PIP3}\right)\andop\mathbf{PDK1}\right)\andop\left(\mathbf{mTORC2}\orop\mathbf{ILK\_{}Rictor}\right)\right)\andop\left(\mathbf{Ras}\orop\mathbf{PAK1}\orop\mathbf{GLI\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{In contact to basal \textit{AKT1}, high \textit{AKT1} activity in our model requires basal \textit{AKT1} (\textit{AKT\_B}), the ongoing presence of high \textit{p110} protein levels along with active \textit{PI3K\_H} and \textit{PIP3}. In addition this maximal \textit{AKT1} activation requires phosphorylation by both \textit{PDK1} and \textit{mTORC2}, , as well as either active \textit{Ras} \cite{Manning:2017di}, active \textit{PAK1} \cite{higuchiScaffoldingFunctionPAK2008} or {\color{NavyBlue} a \textit{GLI1}-mediated boost to \textit{PI3K/AKT} activity \cite{zhouGLI1ReducesDrug2021}}. {In response to \textit{TGF$\beta$}, high  \textit{AKT1} activity can also be induced by \textit{PI3K} and the \textit{ILK-Rictor} complex; the latter capable of replacing \textit{mTORC2} to supply S473 phosphorylation of \textit{AKT1} \cite{serranoRoleIntegrinlinkedKinase2013,mcdonaldRictorIntegrinlinkedKinase2008}}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Overexpression of \textit{GLI1} increases the phosphorylation of \textit{PI3K} and \textit{AKT}, while stable knockdown of \textit{GLI1} attenuates it \cite{zhouGLI1ReducesDrug2021}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Ras}&\multicolumn{1}{m{\linkdspec}}{\textit{Ras} binding to the catalytic subunit of \textit{PI3K} is required for its full potency in \textit{PIP3} generation \cite{RodriguezViciana:1994kn,Gupta:2007dw}. Active \textit{Ras} is thus required for inducing peak \textit{AKT\_H} activity.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{mTORC2}&\multicolumn{1}{m{\linkdspec}}{Maximal activation of \textit{AKT1} requires phosphorylation of S473 by \textit{mTORC2} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PI3K}&\multicolumn{1}{m{\linkdspec}}{{As maximal \textit{AKT1} activity also requires T308 phosphprylation by \textit{PDK1} recruited to the membrane by \textit{PI3K} \cite{Manning:2017di}, we assume that \textit{ILK/Rictor} complexes can only dirve high \textit{AKT} activity in the presence of at least basal \textit{PI3K} activation.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{PIP3}&\multicolumn{1}{m{\linkdspec}}{\textit{PIP3} recruits \textit{AKT1} to the plasma membrane and \textit{PIP3} binding changes the conformation of \textit{AKT1} such that it becomes accessible for T308 phosphorylation by \textit{PDK1} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PDK1}&\multicolumn{1}{m{\linkdspec}}{Membrane-recruited \textit{PDK1} phosphorylates \textit{AKT1}  at T308, a critical step in its activation \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{In our model, high \textit{AKT1} activation is contingent on the ON-state of basal \textit{AKT1} (\textit{AKT\_B}).}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{p110\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Ongoing high \textit{p110} availability and \textit{PI3K\_H} activity are required to induce maximal activation of \textit{AKT\_H} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PI3K\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Ongoing high \textit{p110} availability and \textit{PI3K\_H} activity are required to induce maximal activation of \textit{AKT\_H} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{{\textit{PAK1} interacts with and directly phosphorylates \textit{AKT1} \cite{maoRegulationAktPKB2008}. In addition, \textit{PAK1} provides a scaffold to facilitate \textit{Akt} stimulation by \textit{PDK1} and to aid \textit{AKT’s} membrane recruitment \cite{higuchiScaffoldingFunctionPAK2008}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ILK\allowbreak\_{}Rictor}&\multicolumn{1}{m{\linkdspec}}{{Maximal activation of \textit{AKT1} requires phosphorylation of S473. In response to \textit{TGF-$\beta$}, \textit{ILK} complexes with \textit{Rictor} to phosphoprylate \textit{AKT1} \cite{serranoRoleIntegrinlinkedKinase2013,mcdonaldRictorIntegrinlinkedKinase2008}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{FoxO3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{FoxO3}=\left(\notop\left(\left(\mathbf{AKT\_{}B}\orop\mathbf{AKT\_{}H}\right)\orop\mathbf{ERK}\right)\right)\orop\left(\left(\notop\left(\mathbf{AKT\_{}H}\andop\left(\left(\left(\mathbf{Plk1}\orop\mathbf{Plk1\_{}H}\right)\orop\mathbf{AKT\_{}B}\right)\orop\mathbf{ERK}\right)\right)\right)\andop\left(\notop\left(\left(\mathbf{Plk1}\andop\mathbf{Plk1\_{}H}\right)\andop\mathbf{ERK}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{In order to account for all the influences on \textit{FoxO3} activity, we used the following logic. In the absence of basal or high \textit{AKT1} as well as \textit{ERK}, \textit{FoxO3} remains active. In addition, \textit{FoxO3} can overcame peak (\textit{AKT\_H}) activation only if no other inhibitor is present and \textit{AKT\_B} is OFF (indicating that \textit{AKT1} levels are falling). Finally, the joint activity of \textit{ERK} and \textit{Plk1} can also block \textit{FoxO3}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{ERK downregulates \textit{FoxO3} transcriptional activity by phosphorylating it at three Serines, inducing its \textit{MDM2}-mediated ubiquitination and degradation \cite{Yang:2008ex}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{\textit{AKT1} mediates the translocation of the \textit{FoxO3} our of the nucleus through direct phosphorylation of three conserved residues. These events create a recognition site for \textit{14-3-3} family proteins, which export and sequester \textit{FoxO3} in the cytosol \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{AKT1} mediates the translocation of the \textit{FoxO3} our of the nucleus through direct phosphorylation of three conserved residues. These events create a recognition site for \textit{14-3-3} family proteins, which export and sequester \textit{FoxO3} in the cytosol \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} binds \textit{FoxO3}, induces its translocation to the cytosol, phosphorylates it and suppresses its activity through most of the the cell cycle, but most significantly during G2 and M \cite{Bucur:2014ix}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{Plk1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} binds \textit{FoxO3}, induces its translocation to the cytosol, phosphorylates it and suppresses its activity through most of the the cell cycle, but most significantly during G2 and M \cite{Bucur:2014ix}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PLCgamma}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PLCgamma}=\left(\left(\left(\left(\mathbf{RTK}\andop\mathbf{Grb2}\right)\andop\mathbf{GF\_{}High}\right)\andop\mathbf{p110\_{}H}\right)\andop\mathbf{PI3K\_{}H}\right)\andop\mathbf{PIP3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Enz}}&\multicolumn{2}{m{\nodetwocspec}}{Peak activation of \textit{PLC}$\gamma$ requires active an \textit{RTK} receptor node bound by active \textit{Grb2}, as well as high \textit{PI3K} activity (including high \textit{p110} availability and the presence of PIP3).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{GF\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{We assume that high levels of \textit{RTK} activity is required for tyrosine phosphorylation of \textit{PLC}$\gamma$.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{The SH2 domains of \textit{PLC}$\gamma$ binds to active \textit{RTK}s at tyrosine autophosphorylation sites, leading to tyrosine phosphorylation of \textit{PLC}$\gamma$ and stimulation its enzymatic activity \cite{Cockcroft:1992tp,Kim:1991wo}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Grb2}&\multicolumn{1}{m{\linkdspec}}{\textit{RTK} tyrosine autophosphorylation induces \textit{PLC}$\gamma$ binding to the \textit{Grb2} adaptor protein and likely aids the translocation of \textit{PLC}$\gamma$ to the plasma membrane \cite{Zhang:2000cea}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{PIP3}&\multicolumn{1}{m{\linkdspec}}{Membrane targeting of \textit{PLC}$\gamma$ to growth receptor stimulation is mediated by \textit{PIP3} binding of \textit{PLC}$\gamma$ \cite{Falasca:1998ht,Rameh:1998us}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{p110\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Membrane targeting of \textit{PLC}$\gamma$ to growth receptor stimulation requires \textit{PI3K} activity and \textit{PIP3} generation near growth receptors \cite{Falasca:1998ht}. Thus, peak  \textit{PLC}$\gamma$ activity in our model requires high \textit{p110} protein expression \cite{Rameh:1998us}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PI3K\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{In addition to high \textit{p110} protein levels, high \textit{PI3K} activation is also required to fully activate \textit{PLC}$\gamma$ \cite{Falasca:1998ht,Rameh:1998us}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{IP3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{IP3}=\mathbf{PLCgamma}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Met}}&\multicolumn{2}{m{\nodetwocspec}}{Membrane-bound, active \textit{PLC}$\gamma$ is responsible for converting phosphatidylinositol(4,5)P2 (\textit{PIP2}) to the second messenger inositol(1,4,5)P3 (\textit{IP3}) responsible for triggering a sudden \textit{Ca$^{2+}$} influx from the endoplasmic reticulum, along with \textit{DAG} (diacylglycerol, another second messenger) \cite{Gresset:2012hm}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Cat}}}&\multicolumn{1}{m{\linkcspec}}{PLCgamma}&\multicolumn{1}{m{\linkdspec}}{Membrane-bound, active \textit{PLC}$\gamma$ is responsible for converting phosphatidylinositol(4,5)P2 (\textit{PIP2}) to the second messenger inositol(1,4,5)P3 (\textit{IP3}) responsible for triggering a sudden \textit{Ca$^{2+}$} influx from the endoplasmic reticulum, along with \textit{DAG} (diacylglycerol, another second messenger) \cite{Gresset:2012hm}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Ca2p}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Ca2p}=\mathbf{IP3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Met}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{IP3} travels from the cell membrane to the endoplasmic reticulum where it opens \textit{IP3}-sensitive \textit{Ca$^{2+}$} channels, releasing a sudden \textit{Ca$^{2+}$} efflux from the ER into the cytosol \cite{Michell:1981th}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{IP3}&\multicolumn{1}{m{\linkdspec}}{\textit{IP3} travels from the cell membrane to the endoplasmic reticulum where it opens \textit{IP3}-sensitive \textit{Ca$^{2+}$} channels, releasing a sudden \textit{Ca$^{2+}$} efflux from the ER into the cytosol \cite{Michell:1981th}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Nedd4L}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Nedd4L}=\mathbf{Ca2p}\andop\mathbf{IP3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}UbL}}&\multicolumn{2}{m{\nodetwocspec}}{Activation of \textit{Nedd4L} requires both \textit{Ca$^{2+}$} and \textit{IP3} binding \cite{Escobedo:2014dg}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{IP3}&\multicolumn{1}{m{\linkdspec}}{In order to transition to its active form, the E3 ubiquitin ligase \textit{Nedd4L} binds \textit{Ca$^{2+}$} and inositol 1,4,5-trisphosphate (\textit{IP3}) \cite{Escobedo:2014dg}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Ca2p}&\multicolumn{1}{m{\linkdspec}}{In order to transition to its active form, the E3 ubiquitin ligase \textit{Nedd4L} binds \textit{Ca$^{2+}$} and inositol 1,4,5-trisphosphate (\textit{IP3}) \cite{Escobedo:2014dg}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{GF\_{}mTOR module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TSC2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TSC2}=\left(\notop\mathbf{AKT\_{}H}\right)\orop\left(\notop\left(\mathbf{AKT\_{}B}\orop\mathbf{ERK}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Blocking \textit{TSC2} requires ongoing mitogen stimulation through \textit{AKT} and/or \textit{ERK}. In our model, \textit{TSC2} inhibition requires high (peak) \textit{AKT} activity, supported by either \textit{ERK} or basal \textit{AKT} (assuring that complete loss of \textit{AKT} activity is not impending) \cite{saxtonMTORSignalingGrowth2017}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{\textit{ERK} phosphorylates \textit{TSC2} directly, causing dissociation of the complex and inhibition of its activity \cite{maPhosphorylationFunctionalInactivation2005}. In addition, the \textit{ERK} target \textit{p90RSK} can also inactivate \textit{TSC2} \cite{rouxTumorpromotingPhorbolEsters2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{\textit{TSC2} is phosphorylated by  \textit{AKT1}, inhibiting it by dissociating  \textit{TSC2} from lysosomal membranes \cite{inokiTSC2PhosphorylatedInhibited2002}, where it stimulates GTP hydrolysis of the small GTPase  \textit{Rheb}, this inactivating it \cite{saxtonMTORSignalingGrowth2017}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{TSC2} is phosphorylated by  \textit{AKT1}, inhibiting it by dissociating  \textit{TSC2} from lysosomal membranes \cite{inokiTSC2PhosphorylatedInhibited2002}, where it stimulates GTP hydrolysis of the small GTPase  \textit{Rheb}, this inactivating it \cite{saxtonMTORSignalingGrowth2017}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PRAS40}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PRAS40}=\left(\notop\mathbf{AKT\_{}H}\right)\andop\left(\left(\notop\mathbf{mTORC1}\right)\orop\left(\notop\mathbf{AKT\_{}B}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{PRAS40} is inhibited by peak \textit{AKT1} activity aided by either basal \textit{AKT1} (meaning \textit{AKT\_H} is on its way down), or ongoing \textit{mTORC1} activation. Both \textit{AKT1} and \textit{mTORC1} phosphorylate \textit{PRAS40}, leading to its dissociation from \textit{mTORC1} \cite{wizaRolePRAS40Akt2012}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{\textit{PRAS40} is phosphorylated by \textit{AKT}, triggering its dissociation from \textit{mTORC1} \cite{vanderhaarInsulinSignallingMTOR2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{PRAS40} is an inhibitory component of the \textit{mTORC1} complex. It is phosphorylated by \textit{AKT}, triggering its dissociation from \textit{mTORC1} and loss of \textit{mTORC1} inhibition \cite{vanderhaarInsulinSignallingMTOR2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{mTORC1}&\multicolumn{1}{m{\linkdspec}}{\textit{PRAS40} is a substrate of the \textit{mTORC1} kinase; its phosphorylation aids its dissociation from \textit{mTORC1} and its sequestration by 14-3-3 proteins \cite{fonsecaPRAS40TargetMammalian2007}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{DAG}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{DAG}=\mathbf{PLCgamma}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Met}}&\multicolumn{2}{m{\nodetwocspec}}{Membrane-bound, active \textit{PLC}$\gamma$ is responsible for converting phosphatidylinositol(4,5)P2 (\textit{PIP2}) to the second messenger diacylglycerol (\textit{DAG}), along with \textit{IP3} \cite{Gresset:2012hm}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Cat}}}&\multicolumn{1}{m{\linkcspec}}{PLCgamma}&\multicolumn{1}{m{\linkdspec}}{Membrane-bound, active \textit{PLC}$\gamma$ is responsible for converting phosphatidylinositol(4,5)P2 (\textit{PIP2}) to the second messenger diacylglycerol (\textit{DAG}), along with \textit{IP3} \cite{Gresset:2012hm}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Rheb}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Rheb}=\left(\notop\mathbf{TSC2}\right)\andop\mathbf{DAG}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GTPa}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{PKC} (and \textit{DAG})-dependent activation of \textit{mTORC1} recruits \textit{mTORC1} to the site of \textit{Rheb} activity (to prenuclear lysosomes), while \textit{AKT} and \textit{ERK}-mediated \textit{TSC2} inhibition guarantees that \textit{Rheb} remains potent \cite{inokiRhebGTPaseDirect2003,liuCoordinatedActivationClassical2017,demetriadesLysosomalRecruitmentTSC22016}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}GAP}}}&\multicolumn{1}{m{\linkcspec}}{TSC2}&\multicolumn{1}{m{\linkdspec}}{\textit{TSC2}, a key component of the heterotrimeric \textit{TSC} complex, is a GTPase activating protein (GAP) that induces ATP hydrolysis and deactivation of the small GTPase \textit{Rheb} \cite{inokiRhebGTPaseDirect2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{DAG}&\multicolumn{1}{m{\linkdspec}}{The second messenger \textit{DAG} activates both classical and novel \textit{PKC}s. One of its targets, \textit{PKC}$\eta$, is responsible for the translocation and accumulation of \textit{mTORC1} to perinuclear lysosomes, where the majority of \textit{Rheb} is anchored. Thus, \textit{DAG} brings \textit{Rheb} in proximity with its target, \textit{mTORC1} \cite{liuCoordinatedActivationClassical2017}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{mTORC1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{mTORC1}=\notop\mathbf{Casp3}\andop\left(\notop\mathbf{PRAS40}\orop\mathbf{ERK}\right)\andop\left(\left(\left(\mathbf{Rheb}\andop\notop\mathbf{Merlin}\right)\orop\mathbf{E2F1}\right)\orop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\andop\mathbf{GSK3}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{mTORC1} is activated by mitogenic signals via \textit{Rheb} in the absence of \textit{PRAS40}, {\color{NavyBlue} counteracted by active \textit{ERK} \cite{makarFormateAssayBody1975c}}. This, however, also requires inactivation of \textit{Merlin} independently of \textit{RTK}-induced signals. In addition, \textit{E2F1} can promote \textit{mTORC1} activity. Finally, the mitotic \textit{Cyclin B} and its \textit{Cdk1} kinase can also activate \textit{mTORC1}, aided by \textit{GSK3}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{PRAS40}&\multicolumn{1}{m{\linkdspec}}{\textit{PRAS40} is an inhibitory component of the \textit{mTORC1} complex, removed by phosphorylation by \textit{AKT} or \textit{mTORC1} itself \cite{saxtonMTORSignalingGrowth2017}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ERK1/2} phosphorylate the \textit{Raptor} component of mTORC1 to activate it downstream of \textit{Ras} \cite{makarFormateAssayBody1975c}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Rheb}&\multicolumn{1}{m{\linkdspec}}{The \textit{Rheb} small GTPase binds \textit{mTORC1} directly and activates the complex \cite{longRhebBindsRegulates2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} suppresses \textit{mTORC1} activity via an unknown mechanism that appears to be independent of \textit{PI3K/AKT} or of \textit{TSC2} inhibition \cite{jamesNF2MerlinNovel2009}, and it suppression appears to be critical for integrin-mediated \textit{mTORC1} activation \cite{lopez-lagoLossTumorSuppressor2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1} induces \textit{mTORC1} activity by inducing \textit{mTORC1} translocation to late endosomes. This effect does not require \textit{AKT} and is not blocked by high levels of \textit{TSC2} \cite{realE2F1RegulatesCellular2011}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{During mitosis, \textit{mTORC1} is activated by the G2/M-specific phosphorylation of \textit{Raptor}, a component of \textit{mTORC1}, by \textit{CyclinB / Cdk1} complexes, aided by \textit{GSK3} \cite{ramirez-valleMitoticRaptorPromotes2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{During mitosis, \textit{mTORC1} is activated by the G2/M-specific phosphorylation of \textit{Raptor}, a component of \textit{mTORC1}, by \textit{CyclinB / Cdk1} complexes, aided by \textit{GSK3} \cite{ramirez-valleMitoticRaptorPromotes2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{During mitosis, \textit{mTORC1} is activated by the G2/M-specific phosphorylation of \textit{Raptor}, a component of \textit{mTORC1}, by \textit{CyclinB / Cdk1} complexes, aided by \textit{GSK3} \cite{ramirez-valleMitoticRaptorPromotes2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Raptor}, a key component of the \textit{mTORC1} complex, is cleaved and inhibited by \textit{Caspase 3} \cite{martinCaspasemediatedCleavageRaptor2016}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{S6K}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{S6K}=\left(\notop\mathbf{Casp3}\right)\andop\mathbf{mTORC1}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{S6K} is activated by \textit{mTORC1} in the absence of \textit{Caspase 3}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{mTORC1}&\multicolumn{1}{m{\linkdspec}}{\textit{mTORC1} phosphorylates and activates 40S ribosomal S6 kinases (\textit{S6K}s) \cite{maMolecularMechanismsMTORmediated2009}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{S6K} is cleaved and inhibited by \textit{Caspase 3} \cite{dharProteolyticCleavageP702009}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{h\allowbreak\_{}4EBP1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{h\_{}4EBP1}=\left(\notop\mathbf{mTORC1}\right)\orop\mathbf{SMAD2\_{}3\_{}4}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{4EBP1}, or eukaryotic translation initiation factor 4E-binding protein 1, binds and sequesters the initiation factor \textit{eIF4F} and thus limits translation. \textit{mTORC1} phosphorylates and deactivates \textit{4EBP} to upregulate protein biosynthesis \cite{saxtonMTORSignalingGrowth2017}. While in our previous models this protein was implicit along the  \textit{mTORC1} $\rightarrow$ \textit{eIF4F} link \cite{sizekBooleanModelGrowth2019,gubermanBooleanModelAnchorage2020}, we include it here because \textit{TGF$\beta$} signaling induces \textit{4EBP1} via \textit{SMAD4} to block proliferative metabolism \cite{azar4EBP1TargetSmad42009}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{mTORC1}&\multicolumn{1}{m{\linkdspec}}{\textit{mTORC1} phosphorylates \textit{4EBP}, triggering its dissociation from \textit{eIF4F} \cite{saxtonMTORSignalingGrowth2017}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{4EBP1} is a transcriptional target of \textit{SMAD4}, and its increase is critical for \textit{TGF-$\beta$} mediated inhibition of proliferation \cite{azar4EBP1TargetSmad42009}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{eIF4E}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{eIF4E}=\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\mathbf{h\_{}4EBP1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{eIF4E} is activated by \textit{mTORC1}-mediated repression {of \textit{4EBP1}, which normally binds to it and blocks its activity} \cite{saxtonMTORSignalingGrowth2017}. \textit{eIF4E} is cleaved and deactivated by \textit{Caspase 3} \cite{bushellCaspase3NecessarySufficient1999}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{h\allowbreak\_{}4EBP1}&\multicolumn{1}{m{\linkdspec}}{{\textit{mTORC1}-phosphorylated \textit{4EBP} dissociates from \textit{eIF4F}, allowing it to initiate translation \cite{saxtonMTORSignalingGrowth2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{eIF4E} is cleaved and inhibited by \textit{Caspase 3} \cite{bushellCaspase3NecessarySufficient1999}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{GF\_{}connect module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{GSK3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{GSK3}=\left(\notop\mathbf{AKT\_{}H}\right)\andop\left(\notop\left(\mathbf{S6K}\andop\mathbf{ERK}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{GSK3} activity can be completely blocked by peak \textit{AKT} activation ( \textit{AKT\_H}), or by the joint action of  \textit{S6K} and  \textit{ERK}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{\textit{ERK} binds and phosphorylates \textit{GSK3}$\beta$ at Thr-43, which primes it for subsequent phosphorylation by the \textit{ERK} target \textit{p90RSK} at Ser-9, which inactivates \textit{GSK3}$\beta$ \cite{dingErkAssociatesPrimes2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{AKT} blocks \textit{GSK3} kinase activity via an inhibitory phosphorylation on the amino terminus, which blocks the substrate accessibility of \textit{GSK3} \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{S6K}&\multicolumn{1}{m{\linkdspec}}{\textit{GSK3} is a direct phosphorylation target of \textit{S6K1}, resulting in its inhibition \cite{Zhang:2006bw}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{FoxO1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{FoxO1}=\left(\notop\mathbf{Plk1}\right)\andop\left(\notop\mathbf{AKT\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{FoxO1} is transcriptionally active in the absence of peak \textit{AKT1} activation and \textit{Plk1} activity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{AKT1} mediates the translocation of \textit{FoxO1} our of the nucleus through direct phosphorylation of three conserved residues. These events create a recognition site for \textit{14-3-3} family proteins, which export and sequester \textit{FoxO1} in the cytosol \cite{Manning:2017di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} interacts with and phosphorylates \textit{FoxO1}, mainly at the G2/M phase of the cell cycle. \textit{Plk1}-mediated phosphorylation leads to the impairment of \textit{FoxO1}'s transcriptional activity in an \textit{Akt}-independent manner. \textit{Plk1}-induced \textit{FoxO1} phosphorylation causes its nuclear exclusion  \cite{Yuan:2014bi}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{p21\allowbreak\_{}mRNA}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{p21\_{}mRNA}=\left(\left(\mathbf{FoxO1}\andop\mathbf{FoxO3}\right)\orop\left(\mathbf{SMAD2\_{}3\_{}4}\orop\notop\mathbf{Myc}\right)\andop\left(\mathbf{FoxO1}\orop\mathbf{FoxO3}\right)\right)\andop\notop\left(\mathbf{ZEB1\_{}H}\orop\mathbf{b\_{}catenin\_{}TCF4}\orop\mathbf{N\_{}bcatenin\_{}H}\orop\mathbf{HMGA1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}mRNA}}&\multicolumn{2}{m{\nodetwocspec}}{Our model requires both FoxOs to induce \textit{p21\textsuperscript{Cip1}} if \textit{Myc} is active and one of the two if \textit{Myc} is OFF {or \textit{FoxO} factors are aided by \textit{Smad} activity \cite{gomisFoxOSmadSynexpressionGroup2006}}. This is based on data showing that both \textit{FoxO3} and \textit{FoxO1} bind and induce the \textit{p21\textsuperscript{Cip1}} promoter and that loss of \textit{Myc} repression alone is not sufficient to induce \textit{p21\textsuperscript{Cip1}} \cite{seoaneIntegrationSmadForkhead2004}. {Finally, high levels of \textit{ZEB1}, {\color{NavyBlue} \textit{HMGA1}, or \textit{$\beta$-catenin/TCF4} can all} repress \textit{p21\textsuperscript{Cip1}} mRNA expression \cite{huDeltaEF1PromotesBreast2010,xiHmga1DifferentiallyExpressed2012,kameiNegativeRegulationP212003}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{p21\textsuperscript{Cip1}} is a direct transcriptional target of \textit{FoxO3} \cite{seoaneIntegrationSmadForkhead2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO1}&\multicolumn{1}{m{\linkdspec}}{\textit{p21\textsuperscript{Cip1}} is a direct transcriptional target of \textit{FoxO1} \cite{seoaneIntegrationSmadForkhead2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{Smad3–Smad4} complexes associate with \textit{FoxO} transcription factors to induce \textit{p21\textsuperscript{CIP1}} \cite{gomisFoxOSmadSynexpressionGroup2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} (old name $\delta$EF1) is a direct transcriptional repressor of the p21 promoter \cite{huDeltaEF1PromotesBreast2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{HMGA1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{HMGA1} maintains cell proliferation through direct inhibition of \textit{p21\textsuperscript{Cip1}} \cite{xiHmga1DifferentiallyExpressed2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin/TCF4} are direct transcriptional repressors of \textit{p21\textsuperscript{Cip1}} \cite{kameiNegativeRegulationP212003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin/TCF4} are direct transcriptional repressors of \textit{p21\textsuperscript{Cip1}} \cite{kameiNegativeRegulationP212003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{\textit{Myc} is a direct transcriptional repressor of the \textit{p21\textsuperscript{Cip1}} promoter (it is recruited by the DNA-binding \textit{Miz-1}) \cite{stallerRepressionP15INK4bExpression2001,seoaneMycSuppressionP212002}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{IKKa}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{IKKa}=\mathbf{AKT\_{}H}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{IKK$\alpha$} is a subunit of the \textit{IKK} protein complex composed of two catalytic subunits, \textit{IKK$\alpha$} and \textit{IKK$\beta$}, and the regulatory protein \textit{NEMO}. Activation of the transcription factor \textit{NF-$\kappa$B} is mediated by the IKK complex, which phosphorylates and degrades the inhibitory \textit{I$\kappa$B} proteins \cite{mitchellSignalingNfkBSystem2016}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{IKK$\alpha$} is phosphorylated by \textit{AKT} at T23, and as subsequent \textit{NF-$\kappa$B} activation is induced when high \textit{AKT} activity is observed, our model requires \textit{AKT\_H = ON} for this to occur \cite{baiAktmediatedRegulationNfkB2009}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{NfkB}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{NfkB}=\mathbf{IKKa}\orop\mathbf{PAK1}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{NF-$\kappa$B} is a transcription factor primarily known as a master regulator of inflammatory signaling and the immune system. Its role in cancer is partly due to its ability to aid EMT. It is activated via the destruction of its inhibitory binding partner \textit{I$\kappa$B}, which is phosphorylated by the \textit{IKK} complex subsequently destroted \cite{mitchellSignalingNfkBSystem2016}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{PAK1} binds to with \textit{NF-$\kappa$B}-inducing kinase \textit{NIK}, which induces degradation of \textit{I$\kappa$B} and thus activates \textit{NF-$\kappa$B} \cite{foryst-ludwigP21activatedKinaseActivates2000}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{IKKa}&\multicolumn{1}{m{\linkdspec}}{{\textit{IKK$\alpha$}, part of the \textit{IKK} complex, phosphorylates and degrades the inhibitory \textit{I$\kappa$B} proteins \cite{mitchellSignalingNfkBSystem2016}; an action that can be independent of the \textit{IKK$\beta$} subunit \cite{israelIKKComplexCentral2010}.} }\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{c\allowbreak\_{}Myb}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{c\_{}Myb}=\mathbf{NfkB}\orop\mathbf{E2F1}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{c-Myb} is a transcription factor that can induce the epithelial micro-RNA \textit{miR-200} \cite{pieraccioliActivationMiR200CMyb2013}. \textit{c-Myb} is induced by \textit{AKT}-mediated activation of \textit{NF-$\kappa$B} and/or \textit{E2F1} \cite{lauderCMybTranscriptionActivated2001}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{\textit{NF-$\kappa$B} is a direct transcriptional inducer of \textit{c-Myb} \cite{lauderCMybTranscriptionActivated2001}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{{\textit{E2F1} is a direct transcriptional inducer of \textit{c-Myb} \cite{lauderCMybTranscriptionActivated2001}.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Adhesion module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Integrin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Integrin}=\mathbf{ECM}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{Integrins are a superfamily of heterodimeric cell adhesion receptors that bind to extracellular matrix ligands, cell-surface ligands, and soluble ligands. Upon ligand binding, integrins transduce biomechanical information to the cell interior \cite{takadaIntegrins2007}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{ECM}&\multicolumn{1}{m{\linkdspec}}{{\textit{Integrin} activation and signaling requires \textit{integrin-ECM} attachment \cite{schwartzIntegrinsExtracellularMatrix2010}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ILK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ILK}=\mathbf{Integrin}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{{Integrin-linked kinase \textit{ILK} interacts with the cytoplasmic domain of $\beta$1 integrins, and acrs as a scaffold connecting integrins to the actin cytoskeleton and other signalling pathways \cite{hanniganIntegrinlinkedKinaseCancer2005}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Integrin}&\multicolumn{1}{m{\linkdspec}}{{Integrin-mediated adhesion to the ECM recruits and activates  \textit{ILK} \cite{hanniganIntegrinlinkedKinaseCancer2005}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{FAK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{FAK}=\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\left(\mathbf{Cdk1}\andop\mathbf{CyclinB}\right)\right)\right)\andop\mathbf{Integrin}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{FAK} is activated at integrin-ECM attachment sites in the absence of \textit{Caspase 3}-mediated cleavage and \textit{Cyclin B/Cdk1} activity \cite{mitraFocalAdhesionKinase2005,gervaisCaspasesCleaveFocal1998,yamakitaDissociationFAKP1301999}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Integrin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Integrin} activation leads to recruitment and phosphorylation of the Focal Adhesion Kinase (\textit{FAK}) \cite{mitraFocalAdhesionKinase2005}, one of its key signaling mediators.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{{During mitosis, cells detach most of their focal adhesions and round up. This process is \textit{Cdk1/Cyclin B} dependent \cite{jonesCellAdhesionRegulated2018}, and it leads to the dissociation of focal adhesion complex components including focal adhesion kinase (\textit{FAK}), \textit{paxillin}, and \textit{CAS}. These proteins all change their phosphorylation status by losing active tyrosine and gaining inhibitory serine/threonine phosphorylation \cite{yamakitaDissociationFAKP1301999}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{{During mitosis, cells detach most of their focal adhesions and round up. This process is \textit{Cdk1/Cyclin B} dependent \cite{jonesCellAdhesionRegulated2018}, and it leads to the dissociation of focal adhesion complex components including focal adhesion kinase (\textit{FAK}), \textit{paxillin}, and \textit{CAS}. These proteins all change their phosphorylation status by losing active tyrosine and gaining inhibitory serine/threonine phosphorylation \cite{yamakitaDissociationFAKP1301999}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{Caspase 3 cleaves and deactivates \textit{FAK} during apoptisis by separating its tyrosine kinase from its focal adhesion targeting domain. These fragments further suppress phosphorylation of intact \textit{FAK} \cite{gervaisCaspasesCleaveFocal1998}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Src}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Src}=\left(\left(\left(\mathbf{Integrin}\orop\left(\mathbf{Nectin3}\andop\mathbf{J\_{}Ecadherin}\right)\right)\andop\left(\mathbf{RTK}\orop\mathbf{FAK}\right)\right)\orop\left(\mathbf{Cdk1}\andop\mathbf{CyclinB}\right)\right)\orop\left(\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Src} is activated by \textit{FAK} or \textit{RTKs} at sites of integrin-ECM attachments \cite{mitraIntegrinregulatedFAKSrcSignaling2006,bromannInterplaySrcFamily2004}  {\color{NavyBlue} aided by \textit{Nectins} \cite{fukuyamaInvolvementCSrcCrkC3GRap1Signaling2005} and junctional \textit{E cadherin} at cell cell contacts \cite{mclachlanECadherinAdhesionActivates2007}}, or by \textit{TGF$\beta$} signaling \cite{ungefrorenDifferentialRolesSrc2011}. In addition, \textit{Cyclin B / Cdk1} phosphorylate \textit{Src} during mitosis \cite{stoverCdc2mediatedModulationPp60csrc1994}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{{\textit{RTKs} cooperate with \textit{integrins} to recruit and activate \textit{Src} kinases, which in turn help potentiate \textit{RTK} signaling. Thus, in our model \textit{Src} may be activated by basal \textit{RTK} activity, and is, in turn, required for peak \textit{RTK} activation \cite{bromannInterplaySrcFamily2004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{Integrin}&\multicolumn{1}{m{\linkdspec}}{{\textit{FAK} phosphorylation at Y397 at sites of \textit{integrin-ECM} adhesion creates a high-affinity binding site for \textit{Src}, which leads to the assembly of a \textit{FAK–Src} signaling complex \cite{mitraIntegrinregulatedFAKSrcSignaling2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{FAK}&\multicolumn{1}{m{\linkdspec}}{{\textit{FAK} phosphorylation at Y397 at sites of \textit{integrin-ECM} adhesion creates a high-affinity binding site for \textit{Src}, which leads to the assembly of a \textit{FAK–Src} signaling complex \cite{mitraIntegrinregulatedFAKSrcSignaling2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{Nectin3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Nectin} was found to locally recruit and activate \textit{Src}, leading to downstream phosphorylation of targets \textit{Cdc42} and \textit{Rac1}; downstream signaling was unable to be induced without \textit{Nectins} \cite{fukuyamaInvolvementCSrcCrkC3GRap1Signaling2005}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{E\_cadherin} was found to upregulate \textit{Src} signaling through localized phosphotryosine signaling in response to homophilic ligation; \textit{Src} activity was also found to be increased around cell-cell contact sites \cite{mclachlanECadherinAdhesionActivates2007}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$} activates \textit{Src} via active \textit{T$\beta$RI}/\textit{T$\beta$RII} complexes \cite{ungefrorenDifferentialRolesSrc2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$} activates \textit{Src} via active \textit{T$\beta$RI}/\textit{T$\beta$RII} complexes \cite{ungefrorenDifferentialRolesSrc2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{{Mitotic \textit{Cyclin B/Cdk1} complexes phosphorylate and activate \textit{c-Src} during mitosis \cite{stoverCdc2mediatedModulationPp60csrc1994}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{{Mitotic \textit{Cyclin B/Cdk1} complexes phosphorylate and activate \textit{c-Src} during mitosis \cite{stoverCdc2mediatedModulationPp60csrc1994}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Nectin3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Nectin3}=\mathbf{CellDensity\_{}Low}\orop\mathbf{CellDensity\_{}High}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}CAM}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Nectins} form weak adhesions between adjacent cells by binding to \textit{Nectins} on other cells and promoting local membrane ruffling that is required for adherens junction formation \cite{takaiNectinsNectinlikeMolecules2003}. As downstream effects of \textit{Nectin3} - \textit{Nectin}  binding between two cells do not require tight junction formation and high cell density, \textit{Nectin3} activation in our model only requires the presence of some neighbors.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{Nectins avtivate by forming weak adhesions between adjacent cells \cite{takaiNectinsNectinlikeMolecules2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}Low}&\multicolumn{1}{m{\linkdspec}}{Nectins avtivate by forming weak adhesions between adjacent cells \cite{takaiNectinsNectinlikeMolecules2003}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Necl5}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Necl5}=\mathbf{FocalAdhesions}\orop\left(\notop\left(\mathbf{Nectin3}\andop\mathbf{CellDensity\_{}High}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{Necl-5} activity is controlled by co-localization with focal adhesions where it binds \textit{Spry2} and aids receptor tyrosine kinase signaling \cite{ogitaCellAdhesionMolecules2010}. We modeled this by turning the \textit{Necl-5} node ON when the \textit{Focal Adhesions} node is ON (indicating strong attachments that pull on the ECM and can form stress fibers), or in the absence of cell-cell adhesions at all sites of cell ECM-adhesion (i.e, fully surrounded with no free edge).}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{{At high cell density, adherens junctions that surround the cell suppress integrin-mediated activation and recruitment of \textit{Necl-5} to the cell surface across the entire cell, which releases its block on \textit{Spry2} \cite{ogitaCellAdhesionMolecules2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Unbind}}}&\multicolumn{1}{m{\linkcspec}}{Nectin3}&\multicolumn{1}{m{\linkdspec}}{{\textit{Necl-5} interacts with \textit{Nectin3} on neighboring cells (\textit{Nectin 3} in our model is a proxy for this, as it is activated by cell-cell contacts), which promotes downstream reorganization of the cytoskeleton to aid adherens junction formation, which, in turn releases \textit{Necl-5} from these adhesions \cite{ogitaCellAdhesionMolecules2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{\textit{Necl-5} is recruited to focal adhesions at the leading edge of cells by direct interactions with integrins \cite{minamiNecl5PoliovirusReceptor2007}. This localization is important for its downstream effects.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SPRY2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SPRY2}=\left(\left(\notop\mathbf{Necl5}\right)\andop\mathbf{RTK}\right)\andop\mathbf{Src}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{SPRY2} (\textit{Sprouty 2}) is a negative regulator of growth factor-induced signaling, especially \textit{Ras/MAPK} \cite{kajitaRegulationPlateletderivedGrowth2007}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{RTK}&\multicolumn{1}{m{\linkdspec}}{{\textit{Sprouty} proteins are activated by ligand-bound RTKs to modulate / inhibit downstream MAPK signaling  \cite{masonSproutyProteinsMultifaceted2006}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Src}&\multicolumn{1}{m{\linkdspec}}{{Growth factor-induced tyrosine phosphorylation of \textit{Spry2} is mediated by a \textit{Src}-like kinase \cite{masonSproutyProteinsMultifaceted2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{Necl5}&\multicolumn{1}{m{\linkdspec}}{{\textit{Necl-5} localized to integrin clusters binds to and blocks the activity of \textit{SPRY2} \cite{kajitaRegulationPlateletderivedGrowth2007}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{J\_{}Ecadherin}=\left(\notop\mathbf{Casp3}\right)\andop\notop\mathbf{MMP7}\andop\left(\mathbf{Nectin3}\andop\left(\mathbf{Ecadherin\_{}mRNA\_{}H}\orop\mathbf{Ecadherin\_{}mRNA}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}CAM}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{E-cadherin} proteins are adhesion molecules required for adherens junction formation. The presence of juncitonal \textit{E-cadherin} (J\_Ecadherin = ON) in our model requires {\textit{E-cadherin} mRNA expression}, \textit{Nectin3}-mediated sensing of contact with neighboring cells (binding to nectins) \cite{ogitaCellAdhesionMolecules2010} and inactive {\color{NavyBlue}\textit{MMP-7} \cite{chatterjeeEGFRmediatedMatrixMetalloproteinase72018a}} and/or Caspase 3 \cite{steinhusenCleavageSheddingEcadherin2001}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{MMP7}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Active \textit{MMP-7} promotes EMT by cleaving \textit{E-cadherin} protein \cite{chatterjeeEGFRmediatedMatrixMetalloproteinase72018a}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Nectin3}&\multicolumn{1}{m{\linkdspec}}{{Cadherins are recruited to cell-cell adhesions formed by \textit{Nectin3-nectin} interactions between neighboring cells, where they bind to cadherins on adjacent cells to form AJs \cite{ogitaCellAdhesionMolecules2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TL}}}&\multicolumn{1}{m{\linkcspec}}{Ecadherin\allowbreak\_{}mRNA\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{E-cadherin} mRNA expression is required for maintenance of \textit{E-cadherin} protein.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TL}}}&\multicolumn{1}{m{\linkcspec}}{Ecadherin\allowbreak\_{}mRNA}&\multicolumn{1}{m{\linkdspec}}{{\textit{E-cadherin} mRNA expression is required for maintenance of \textit{E-cadherin} protein.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{Caspase 3 cleaves junctional E-cadherin, dissociating it from the cell surface and blocking its ability to form adherens junctions \cite{steinhusenCleavageSheddingEcadherin2001}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{J\allowbreak\_{}bcatenin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{J\_{}bcatenin}=\left(\notop\mathbf{Casp3}\right)\andop\mathbf{J\_{}Ecadherin}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Junctional \textit{E-cadherin} proteins binds to and recruits/sequesters \textit{$\beta$-catenin} to adherens junctions. This negatively regulates {$\beta$-catenin} mediated transcription, but aids adherens junction formation \cite{tianECadherinCateninComplex2011}. Here we assume that \textit{$\beta$-catenin}  is localized to cell-cell junctions (\textit{J\_bcatenin} = ON) as long as there are \textit{E-cadherin}-mediated attachments to neighboring cells (even if a cell is not fully surrounded) and \textit{Caspase 3} is inactive \cite{steinhusenApoptosisinducedCleavageVCatenin2000}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{1}{m{\linkdspec}}{{Junctional \textit{E-cadherins} recruits/sequester \textit{$\beta$-catenin} to adherens junctions \cite{tianECadherinCateninComplex2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{Caspase 3} cleaves \textit{$\beta$-catenin} into several fragments that lose their transcriptional activity and become localized to the cytoplasm \cite{steinhusenApoptosisinducedCleavageVCatenin2000}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{J\allowbreak\_{}acatenin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{J\_{}acatenin}=\mathbf{J\_{}bcatenin}\andop\mathbf{J\_{}Ecadherin}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Adap}}&\multicolumn{2}{m{\nodetwocspec}}{Junctional \textit{$\beta$-catenin} binds to and recruitcs \textit{$\alpha$-catenin} to adherens junctions. \textit{$\alpha$-catenin} links \textit{$\beta$-catenin} to the actin cytoskeleton to stabilize adherens junctions \cite{tianECadherinCateninComplex2011}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Junctional \textit{E-cadherin}-bound \textit{$\beta$-catenin} binds to and recruitcs \textit{$\alpha$-catenin} to adherens junctions \cite{tianECadherinCateninComplex2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}bcatenin}&\multicolumn{1}{m{\linkdspec}}{Junctional \textit{$\beta$-catenin} binds to and recruitcs \textit{$\alpha$-catenin} to adherens junctions \cite{tianECadherinCateninComplex2011}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{CIP module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{FocalAdhesions}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{FocalAdhesions}=\mathbf{Integrin}\andop\mathbf{FAK}\andop\mathbf{ECM}\andop\left(\mathbf{Stiff\_{}ECM}\orop\mathbf{Basement\_{}Membrane}\orop\left(\mathbf{YAP}\andop\left(\mathbf{Rac1}\orop\mathbf{Rac1\_{}H}\right)\andop\mathbf{IQGAP1\_{}LeadingE}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Focal adhesions} form at sites of \textit{Integrin-ECM} attachment and clustering \cite{geigerEnvironmentalSensingFocal2009}. In order to take into account the effect of stiff ECM \cite{plotnikovForceFluctuationsFocal2012} as well as positive feedback between focal adhesion formation and horizontal cell polarization that creates an active leading edge, here we assume that force-generating Focal Adhesion formation requires ECM-Integrin attachments and \textit{FAK} \cite{kleinschmidtFocalAdhesionKinase2017}, and either strong traction force generation supported by a stiff ECM, {\color{NavyBlue}adhesion to a basement membrame,} or the existence of a leading edge with active \textit{Rac1} \cite{pasaperaRac1DependentPFocal2015} and \textit{IQGAP1} \cite{kozlovaIQGAP1RegulatesHyaluronanmediated2012,kohnoIQGAP1LinksPDGF2013,foroutannejadNovelRoleIQGAP12014} supported by \textit{YAP}-mediated upregulation of adhesion and focal adhesion-associated proteins \cite{nardoneYAPRegulatesCell2017}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Stiff\allowbreak\_{}ECM}&\multicolumn{1}{m{\linkdspec}}{{Adhesion to stiff ECM engages the \textit{FAK/phosphopaxillin/vinculin} pathway, which generate a fluctuating “tugging” action on the ECM and probe ECM rigidity to aid \textit{focal adhesion} formation and migration towards regions of stiffer ECM (durotaxis) \cite{plotnikovForceFluctuationsFocal2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Basement\allowbreak\_{}Membrane}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Adhesion to an intact basement membrane can also engage the \textit{FAK/phosphopaxillin/vinculin} pathway, as it supports its “tugging” action to aid \textit{focal adhesion} formation and migration towards regions of stiffer ECM (durotaxis) \cite{plotnikovForceFluctuationsFocal2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{ECM}&\multicolumn{1}{m{\linkdspec}}{{\textit{Focal adhesions} form at sites of \textit{Integrin-ECM} attachment and clustering \cite{geigerEnvironmentalSensingFocal2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Integrin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Focal adhesions} form at sites of \textit{Integrin-ECM} attachment and clustering \cite{geigerEnvironmentalSensingFocal2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{FAK}&\multicolumn{1}{m{\linkdspec}}{{\textit{FAK} activation at sites of cell-ECM attachments (driven by force generation) can increase paxillin phosphorylation and strengthen cytoskeletal linkage and vinculin recruitment to such adhesions, resulting in \textit{focal adhesion} maturation \cite{kleinschmidtFocalAdhesionKinase2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} is a transcriptional inducer of \textit{f integrins} and \textit{FA} docking proteins, and promotes \textit{focal adhesion} formation by increasing cell spreading and \textit{RhoA GTPase} activity \cite{nardoneYAPRegulatesCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{IQGAP1\allowbreak\_{}LeadingE}&\multicolumn{1}{m{\linkdspec}}{{In migrating cells \textit{IQGAP1} localizes to lamellipodia at the leading edge, recruited by active \textit{RTKs} synergistically activated here by \textit{integrin-RTK} crosstalk. Here, \textit{IQGAP1} supports \textit{Rac1} activation and \textit{focal adhesion} formation; it required for migration in response to growth signals such as \textit{PDGF}, \textit{VEGF}, ect \cite{kozlovaIQGAP1RegulatesHyaluronanmediated2012,kohnoIQGAP1LinksPDGF2013,foroutannejadNovelRoleIQGAP12014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Rac1}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{Rac1} promotes the association of nonmuscle myosin II (MIIA) with \textit{focal adhesions} at the leading edge during cell migration, aiding the assembly of mini- filaments in \textit{focal adhesions}. These promote further assembly of \textit{focal adhesions} and modulation of the traction forces cells exert on the ECM \cite{pasaperaRac1DependentPFocal2015}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Rac1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} High \textit{Rac1} activity promotes the association of nonmuscle myosin II (MIIA) with \textit{focal adhesions} at the leading edge during cell migration, aiding the assembly of mini- filaments in \textit{focal adhesions}. These promote further assembly of \textit{focal adhesions} and modulation of the traction forces cells exert on the ECM \cite{pasaperaRac1DependentPFocal2015}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Stress\allowbreak\_{}Fibers}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Stress\_{}Fibers}=\left(\left(\notop\mathbf{CellDensity\_{}High}\right)\andop\mathbf{Stiff\_{}ECM}\right)\andop\mathbf{FocalAdhesions}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{In order to model the independent effects of cell density and matrix stiffness on stress fiber formation, we assume that they require both the absence of high cell density, and the presence of focal adhesions attached to a stiff ECM.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{{\textit{High cell density} blocks stress fiber formation by forbidding cells access to a large enough area to spread and exert force on the \textit{ECM} \cite{aragonaMechanicalCheckpointControls2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Stiff\allowbreak\_{}ECM}&\multicolumn{1}{m{\linkdspec}}{{In the absence of a sufficiently stiff \textit{ECM}, \textit{Focal Adhesions} are small and stress fibers are less abundant or fail to form, as cells cannot generate sufficient traction \cite{leeActomyosinStressFiber2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{\textit{Stress Fibers} are anchored to the \textit{ECM} via strong, stable \textit{Focal Adhesions} \cite{leeActomyosinStressFiber2016}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{YAP}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{YAP}=\left(\mathbf{FocalAdhesions}\andop\mathbf{Stress\_{}Fibers}\right)\andop\left(\notop\left(\left(\left(\left(\mathbf{ApicalBasal\_{}Pol}\andop\mathbf{J\_{}acatenin}\right)\andop\mathbf{AMOT}\right)\andop\mathbf{Merlin}\right)\andop\mathbf{Lats1\_{}2}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{YAP is a mechanosensitive transcriptional regulator of proliferation and migration, and its activation is controlled by both the cell’s ability to spread on an ECM (\textit{FocalAdhesions} and \textit{Stress\_Fibers}), and the lack of apical-basal polarity with mature adherens junctions that can sequester \textit{YAP} in the cytoplasm by binding and inhibitory phosphorylation. Experimental evidence indicates that in cells that maintain apical-basal polarity, several junctional proteins (\textit{$\alpha$-catenin}, \textit{AMOT}, \textit{Merlin}) and inhibitory kinases (\textit{Lats1} and \textit{Lats2}  work together to sequester and block \textit{YAP} \cite{karamanCellJunctionsHippo2018,hoaCharacterisationLATS2Kinase2016}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}acatenin}&\multicolumn{1}{m{\linkdspec}}{{Junctional \textit{alpha-catenin} binds \textit{YAP} and sequesters it in the cytoplasm \cite{silvisAcateninTumorSuppressor2011} \cite{schlegelmilchYap1ActsDownstream2011} \cite{sarpalRoleACateninIts2019}. This also concentrates \textit{YAP} close proximity to junction-localized Hippo pathway components such as \textit{Lats1/2}, \textit{Merlin} and \textit{Amot.}}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} activation is abolished by the absence of \textit{stress fibers} anchored to \textit{focal adhesions}, even in the absence of inhibitory Hippo signaling \cite{aragonaMechanicalCheckpointControls2013,dupontRoleYAPTAZ2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Stress\allowbreak\_{}Fibers}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} activation is abolished by the absence of \textit{stress fibers} anchored to \textit{focal adhesions}, even in the absence of inhibitory Hippo signaling \cite{aragonaMechanicalCheckpointControls2013,dupontRoleYAPTAZ2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Full inhibition of \textit{YAP} by Hippo signaling linked to adherens and tight junction formation requires the establishment of \textit{apical-basal polarity}, as cells at the edges of monolayers or in decreased cell density areas have active (nuclear) \textit{YAP} in spite of strong remaining attachments to neighboring cells \cite{dupontRoleYAPTAZ2011,parkSwitchlikeEnhancementEpithelialmesenchymal2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{Lats1\allowbreak\_{}2}&\multicolumn{1}{m{\linkdspec}}{{The \textit{Lats1} and \textit{Lats2} tumor suppressor kinases bind to and phosphorylate \textit{YAP} in vitro and in vivo \cite{haoTumorSuppressorLATS12008,hoaCharacterisationLATS2Kinase2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{AMOT}&\multicolumn{1}{m{\linkdspec}}{{\textit{AMOT} localizes to tight junctions, where it suppresses \textit{YAP} activity by direct binding and recruitment of \textit{YAP} inhibitory kinase \textit{LATS2} \cite{karamanCellJunctionsHippo2018}. By binding both and \textit{YAP/TAZ}, \textit{AMOT} works as a scaffold that connects \textit{LATS1/2} to both its activator \textit{MST1} and its target \textit{YAP/TAZ} \cite{mana-capelliAngiomotinsStimulateLATS2018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} localizes to adherens junctions where it activates the \textit{Hippo pathway} by binding to and recruiting \textit{LATS1/2} kinases and \textit{YAP/TAZ} to \textit{adherens junctions} \cite{yinSpatialOrganizationHippo2013}. In the absence of \textit{Merlin}, Hippo pathway components fail to block \textit{YAP} activity \cite{zhangMerlinNF2Tumor2010}. \textit{Merlin} - \textit{YAP} binding requires active (phosphorylated) \textit{AMOT} \cite{moleirinhoRegulationLocalizationFunction2017}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TRIO}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TRIO}=\mathbf{YAP}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GEF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{TRIO} is a \textit{Rac1}-activating GTP-exchange factor induced by \textit{YAP} \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019,shahYAPControlsCell2019}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} is a transcriptional inducer of \textit{TRIO} \cite{shahYAPControlsCell2019}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{WT1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{WT1}=\mathbf{YAP}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{The Wilms Tumor 1 (\textit{WT1}) transcription factor is a repressor of \textit{E-cadherin} expression \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019}. Its nuclear localization is controlled by \textit{YAP} binding \cite{shahYAPControlsCell2019}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} binds to and controls nuclear localization of the Wilms Tumor 1 (\textit{WT1}) transcription factor \cite{shahYAPControlsCell2019}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TAZ}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TAZ}=\mathbf{Stress\_{}Fibers}\andop\left(\notop\left(\left(\left(\left(\mathbf{ApicalBasal\_{}Pol}\andop\mathbf{J\_{}acatenin}\right)\andop\mathbf{AMOT}\right)\andop\mathbf{Merlin}\right)\andop\mathbf{Lats1\_{}2}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{TAZ} is a mechanosensitive transcriptional regulator of cell migration, and its activation is controlled by both the cell’s ability to spread on stiff ECM (\textit{Stress\_Fibers} = ON) and the lack of apical-basal polarity with mature adherens junctions that can sequestered \textit{TAZ} in the cytoplasm by binding and inhibitory phosphorylation \cite{dupontRoleYAPTAZ2011,parkSwitchlikeEnhancementEpithelialmesenchymal2019}. In cells that maintain apical-basal polarity, junctional proteins (\textit{$\alpha$-catenin}, \textit{AMOT}, \textit{Merlin}) and inhibitory kinases (\textit{Lats1} and \textit{Lats2}) work together to sequester and block \textit{TAZ} \cite{silvisAcateninTumorSuppressor2011,schlegelmilchYap1ActsDownstream2011,sarpalRoleACateninIts2019,dasguptaControlCellularResponses2019,hoaCharacterisationLATS2Kinase2016}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}acatenin}&\multicolumn{1}{m{\linkdspec}}{{Junctional \textit{alpha-catenin} binds \textit{YAP/TAZ} and sequesters them in the cytoplasm \cite{silvisAcateninTumorSuppressor2011,schlegelmilchYap1ActsDownstream2011,sarpalRoleACateninIts2019,dasguptaControlCellularResponses2019}. This also concentrates \textit{YAP/TAZ} close proximity to junction-localized Hippo pathway components such as \textit{Lats1/2}, \textit{Merlin} and \textit{Amot}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Stress\allowbreak\_{}Fibers}&\multicolumn{1}{m{\linkdspec}}{{\textit{TAZ} activation is abolished by the absence of \textit{stress fibers} anchored to \textit{focal adhesions}, even in the absence of inhibitory Hippo signaling \cite{aragonaMechanicalCheckpointControls2013,dupontRoleYAPTAZ2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Full inhibition of \textit{TAZ} by Hippo signaling linked to adherens and tight junction formation requires the establishment of apical-basal polarity, as cells at the edges of monolayers or in decreased cell density areas have active (nuclear) \textit{TAZ} in spite of strong remaining attachments to neighboring cells \cite{dupontRoleYAPTAZ2011,parkSwitchlikeEnhancementEpithelialmesenchymal2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Lats1\allowbreak\_{}2}&\multicolumn{1}{m{\linkdspec}}{{The \textit{Lats1} and \textit{Lats2} tumor suppressor kinases bind to and phosphorylate \textit{YAP/TAZ} in vitro and in vivo \cite{haoTumorSuppressorLATS12008,hoaCharacterisationLATS2Kinase2016,dasguptaControlCellularResponses2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{AMOT}&\multicolumn{1}{m{\linkdspec}}{{\textit{AMOT} localizes to tight junctions, where it suppresses \textit{YAP/TAZ} activity by direct binding and recruitment of \textit{YAP} inhibitory kinase \textit{LATS2} \cite{karamanCellJunctionsHippo2018,dasguptaControlCellularResponses2019}. By binding both and \textit{YAP/TAZ}, \textit{AMOT} works as a scaffold that connects \textit{LATS1/2} to both its activator \textit{MST1} and its target \textit{YAP/TAZ} \cite{mana-capelliAngiomotinsStimulateLATS2018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} localizes to adherens junctions where it activates the Hippo pathway by binding to and recruiting \textit{LATS1/2} kinases and \textit{YAP/TAZ} to adherens junctions \cite{yinSpatialOrganizationHippo2013}. In the absence of \textit{Merlin}, Hippo pathway components fail to block \textit{YAP/TAZ} activity \cite{zhangMerlinNF2Tumor2010,dasguptaControlCellularResponses2019}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Ecadherin\allowbreak\_{}mRNA\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Ecadherin\_{}mRNA\_{}H}=\mathbf{Ecadherin\_{}mRNA}\andop\left(\notop\left(\mathbf{YAP}\andop\mathbf{WT1}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}mRNA}}&\multicolumn{2}{m{\nodetwocspec}}{Experiments show that \textit{YAP} and \textit{WT1} suppress but do not abolish \textit{E-cadherin} protein expression in areas of lowered cell density \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019}. To model this, we introduced a \textit{Ecadherin\_mRNA\_H} node that is blocked by \textit{YAP/WT1} repressor complexes (requiring their joint nuclear localization). \textit{Ecadherin\_mRNA\_H}, in turn, must be ON to allow cells to establish a ring of adherens junctions sufficient for apical-basal polarity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{YAP} binds \textit{WT1} and localizes it to the nucleus, where they form a complex a the \textit{E-cadherin} promoter and reduce its transcription \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{WT1}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{YAP} binds \textit{WT1} and localizes it to the nucleus, where they form a complex a the \textit{E-cadherin} promoter and reduce its transcription \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Ecadherin\allowbreak\_{}mRNA}&\multicolumn{1}{m{\linkdspec}}{{High \textit{E-cadherin} mRNA expression requires basal levels of \textit{E-cadherin}. Our previously published epithelial model assumed this to be true \cite{gubermanBooleanModelAnchorage2020}, whereas in the current model this is only the case when \textit{E-cadherin} expression is not fully inhibited by EMT-promoting repressors (modeled as acting on the \textit{Ecadherin\_mRNA} node).}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ApicalBasal\_{}Pol}=\left(\mathbf{ECM}\andop\left(\left(\left(\left(\mathbf{CellDensity\_{}High}\andop\mathbf{Nectin3}\right)\andop\mathbf{J\_{}Ecadherin}\right)\andop\mathbf{J\_{}bcatenin}\right)\andop\mathbf{J\_{}acatenin}\right)\right)\andop\left(\mathbf{Ecadherin\_{}mRNA\_{}H}\orop\left(\notop\mathbf{Horizontal\_{}Pol}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{In addition to a need for high cell density and cell-cell adhesion proteins that help assemble adherens junctions (\textit{Nectin3}, \textit{J\_Ecadherin}, \textit{J\_bcatenin}, \textit{J\_acatenin}), we assumed that either high (unimpeded) \textit{E-cadherin} mRNA expression or lack of a horizontally polarized cell morphology are required for the establishment of apical-basal polarity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{{Establishment of \textit{apical-basal polarity} requires a ring of adherens and tight junctions that can only form in high cell density \cite{mccaffreySignalingPathwaysCell2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{ECM}&\multicolumn{1}{m{\linkdspec}}{{Establishment of \textit{apical-basal polarity} requires an underlying surface such as \textit{ECM} to define a basal side.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Nectin3}&\multicolumn{1}{m{\linkdspec}}{{A key driver of \textit{apical-basal polarization}, \textit{Par-3}, is recruited to newly formed cell-cell adhesions by \textit{Nectin-3} binding \cite{mccaffreySignalingPathwaysCell2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{1}{m{\linkdspec}}{{Formation of adherens junctions is a prerequisite for tight junction assembly, which is, in turn, required for \textit{apical-basal polarity} \cite{mccaffreySignalingPathwaysCell2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}bcatenin}&\multicolumn{1}{m{\linkdspec}}{{Formation of adherens junctions is a prerequisite for tight junction assembly, which is, in turn, required for \textit{apical-basal polarity} \cite{mccaffreySignalingPathwaysCell2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}acatenin}&\multicolumn{1}{m{\linkdspec}}{{Formation of adherens junctions is a prerequisite for tight junction assembly, which is, in turn, required for \textit{apical-basal polarity} \cite{mccaffreySignalingPathwaysCell2012}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Ecadherin\allowbreak\_{}mRNA\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{Formation of adherens junctions in high concentration around the cell is required for \textit{apical-basal polarity} \cite{mccaffreySignalingPathwaysCell2012}, and thus aided by high \textit{E-cadherin} protein expression.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Horizontal polarization and apical-basal (vertical) polarization are mutually exclusive; cells must first lose the asymmetry between their leading and trailing edge in order to establish a ring of adherens and tight junctions.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{N\allowbreak\_{}bcatenin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{N\_{}bcatenin}=\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\left(\mathbf{ApicalBasal\_{}Pol}\andop\mathbf{GSK3}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{When released from cell-cell junctions, \textit{$\beta$-catenin} localizes to the nucleus and induces genes that promote proliferation and EMT \cite{bienzVCateninPivotCell2005}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{{The cytoplasmic pool of \textit{$\beta$-catenin} not tied to junctions is highly unstable due to multiple phosphorylations promoting its proteasome-mediated degradation. This phosphorylations is maintained by the "\textit{$\beta$-catenin} Destruction Complex" composed of the tumor suppressor \textit{APC}, the scaffolding protein \textit{Axin}, and the serine/threonine kinases \textit{GSK3}$\beta$ and \textit{CK1} (casein kinase 1, which primes \textit{GSK3}). When \textit{GSK3} is inhibited, unphosphorylated \textit{$\beta$-catenin} accumulates, translocates to the nucleus, and promotes transcription \cite{bienzVCateninPivotCell2005}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Junctional \textit{E-cadherins} recruits/sequesters \textit{$\beta$-catenin} to adherens junctions and block \textit{$\beta$-catenin} nuclear localization \cite{tianECadherinCateninComplex2011}. Here we assume that cells able to form a ring of adherens junctions and establish apical-basal polarity lack nuclear \textit{$\beta$-catenin}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{Active Caspase 3 cleaves \textit{$\beta$-catenin} into several fragments that lose their transcriptional activity and become localized to the cytoplasm \cite{steinhusenApoptosisinducedCleavageVCatenin2000}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Mst1\allowbreak\_{}2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Mst1\_{}2}=\mathbf{ApicalBasal\_{}Pol}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{In cells that establish apical-basal cell polarity, \textit{Mst1} and \textit{2} activate the Hippo pathway by phosphorylating \textit{Lats1/2} kinases to promote \textit{YAP/TAZ} inhibition \cite{chanSte20likeKinaseMst22005,praskovaMOBKL1AMOBKL1BPhosphorylation2008,kwanDLG5ConnectsCell2016}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{Activation of the Hippo pathway by \textit{Mst1/2} requires apical-basal cell polarity \cite{kwanDLG5ConnectsCell2016}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Lats1\allowbreak\_{}2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Lats1\_{}2}=\mathbf{Mst1\_{}2}\andop\mathbf{Merlin}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Mst1/2} activate \textit{Lats1/2} kinases by phosphorylation, aided by their mutual binding to \textit{Merlin}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Mst1\allowbreak\_{}2}&\multicolumn{1}{m{\linkdspec}}{{\textit{Mst1/2} phosphorylate and activate \textit{Lats 1 / 2} kinases to promote \textit{YAP/TAZ} inhibition \cite{chanSte20likeKinaseMst22005,praskovaMOBKL1AMOBKL1BPhosphorylation2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} binds to \textit{Lats1/2} and recruits it to the plasma membrane near adherens and tight junctions, where \textit{Mst1/2} can phosphorylate it \cite{yinSpatialOrganizationHippo2013}.} }\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{AMOT}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{AMOT}=\mathbf{Lats1\_{}2}\andop\mathbf{Merlin}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Adap}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, \textit{AMOT} = ON represents phosphorylated and tight junction localized \textit{AMOT}, which requires \textit{Merlin} binding and \textit{Lats1/2} mediated phosphorylation \cite{hirateRoleAngiomotinPhosphorylation2014,yiTightJunctionassociatedMerlinangiomotin2011}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Lats1\allowbreak\_{}2}&\multicolumn{1}{m{\linkdspec}}{{\textit{Lats1/2} kinases phosphorylate the N-terminal regions of \textit{Amot}, disrupting its interaction with F-actin \cite{daiPhosphorylationAngiomotinLats12013}. As tight junction localized \textit{AMOT} aids Hippo signaling whereas F-actin bound \textit{AMOT} hinders it \cite{hirateRoleAngiomotinPhosphorylation2014}, \textit{AMOT} = ON in our model represents the phosphorylated, TJ-bound protein.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} binds to \textit{AMOT} proteins and sequesters them to tight junctions \cite{daiPhosphorylationAngiomotinLats12013}, shifting its activity from cytoplasmic (where it binds \textit{F-actin} and blocks the \textit{Rac1} inhibitor \textit{Rish1}) to junctional (where it forms a scaffold for Hippo signaling) \cite{yiTightJunctionassociatedMerlinangiomotin2011}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{miR\allowbreak\_{}29c}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{miR\_{}29c}=\mathbf{YAP}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}miR}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{YAP}  induces the expression of microRNA \textit{miR-29c} to target \textit{PTEN} for degradation \cite{tumanengYAPMediatesCrosstalk2012}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{\textit{YAP} is a direct inducer of \textit{miR-29c} \cite{tumanengYAPMediatesCrosstalk2012}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PTEN\allowbreak\_{}c}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PTEN\_{}c}=\left(\notop\mathbf{miR\_{}29c}\right)\andop\left(\left(\mathbf{S6K}\andop\left(\notop\left(\mathbf{ERK}\andop\mathbf{GSK3}\right)\right)\right)\orop\left(\left(\notop\mathbf{ERK}\right)\andop\left(\notop\mathbf{GSK3}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Ph}}&\multicolumn{2}{m{\nodetwocspec}}{Cellular \textit{PTEN} is a tumor supressor phosphatase that reverses PIP2 $\rightarrow$ PIP3 conversion carried out by \textit{PI3K}, and thus supresses \textit{PI3K / AKT} signaling \cite{chenPTENTumorSuppressor2018}. The combinatorial effect of the distinct \textit{PTEN} regulators included below is not well documented. We  chose to model cytoplasmic \textit{PTEN} availability as ON in the absence of \textit{miR-29c} \cite{tumanengYAPMediatesCrosstalk2012}. In addition, we combined the positive and negative effects of \textit{S6K}, \textit{ERK} and \textit{GSK3} by assuming that \textit{PTEN} is blocked by the joint action of \textit{ERK} and \textit{GSK3} in the presence of \textit{S6K}, and by either inhibitor in its absence.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{\textit{ERK} activation suppresses \textit{PTEN mRNA} and protein expression \cite{chowRASERKModulates2007,beckBMPSuppressesPTEN2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{S6K}&\multicolumn{1}{m{\linkdspec}}{\textit{S6K} phosphorylates \textit{PTEN}, which leads to \textit{PTEN} deubiquitination and export from the nucleus to the cytoplasm \cite{wuPTENPhosphorylationNuclear2014}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{\textit{GSK3} phosphorylates \textit{PTEN} on Thr366 which leads to destabilization of the \textit{PTEN} protein \cite{maccarioPTENDestabilizedPhosphorylation2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}29c}&\multicolumn{1}{m{\linkdspec}}{\textit{YAP} induces transcription of \textit{miR-29}, which in turn binds to \textit{PTEN mRNA} to block its translation \cite{tumanengYAPMediatesCrosstalk2012}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Migration\_{}SW module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Merlin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Merlin}=\left(\left(\mathbf{J\_{}bcatenin}\andop\mathbf{J\_{}acatenin}\right)\andop\left(\notop\left(\mathbf{PAK1}\andop\mathbf{ILK}\right)\right)\right)\andop\left(\notop\mathbf{AKT\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Merlin} is functionally localized and able to mediate Hippo signaling when recruited by junctional $\beta$- and $\alpha$-\textit{catenin}, and not phosphorylated by high \textit{AKT1} {or the cooperative action of \textit{PAK1} and \textit{ILK}}. Since the \textit{AKT\_B} node in our model represents basal \textit{AKT} activity which co-occurs with \textit{Merlin}-mediated contact inhibition, we choose the peak \textit{AKT\_H} node to mark the level of \textit{AKT} signaling required to block \textit{Merlin}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{Akt} directly binds to and phosphorylates \textit{Merlin} at Thr230 and Ser315, blocking its ability to bind its regular partners and promoting its degradation \cite{tangAktPhosphorylationRegulates2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ILK}&\multicolumn{1}{m{\linkdspec}}{{\textit{ILK} suppresses the Hippo pathway via phosphorilation and inhibition of \textit{MYPT1-PP1}, leading to inactivation of \textit{Merlin} \cite{serranoInactivationHippoTumour2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}bcatenin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} binds to \textit{$\beta$-catenin} at adherens junctions, and disruption of \textit{$\beta$-catenin} expression recoiples cell density sensing from Hippo signaling downstream \textit{Merlin} \cite{kimEcadherinMediatesContact2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}acatenin}&\multicolumn{1}{m{\linkdspec}}{{\textit{Merlin} binds to \textit{$\alpha$-catenin} to localize to adherens junctions, where it plays a role in their maturation, links AJ formation and the \textit{Par3} polarity complex, and orchestrates Hippo signaling \cite{kimEcadherinMediatesContact2011,gladdenNF2TumorSuppressor2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{{\textit{Pak1} can directly phosphorylate \textit{Merlin} at Ser518 \cite{xiaoP21activatedKinaseLinks2002}. This phosphorylation prevents it from binding to /textit{AMOT} and \textit{Lats1/2} \cite{liAngiomotinBindinginducedActivation2015}, and thus carrying out its contact inhibitory function.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{IQGAP1\allowbreak\_{}LeadingE}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{IQGAP1\_{}LeadingE}=\left(\left(\notop\mathbf{CellDensity\_{}High}\right)\andop\mathbf{FocalAdhesions}\right)\andop\left(\left(\mathbf{Horizontal\_{}Pol}\andop\left(\mathbf{Rac1}\orop\mathbf{Rac1\_{}H}\right)\right)\orop\mathbf{Grb2}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Adap}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, \textit{IQGAP1\_LeadingE} = ON represents \textit{IQGAP1} localized near the leading edge of a horizontally polarized cell, where it links focal adhesion-mediated and \textit{RTK}-mediated signaling \cite{kohnoIQGAP1LinksPDGF2013}. Its recruitment is sustained by well-established horizontal polarity, or induced by \textit{Rac1} \cite{fukataRolesRhofamilyGTPases2003} or \textit{RTK-Grb2} binding \cite{mcnultyMAPKScaffoldIQGAP12011,bensenorIQGAP1RegulatesCell2007,blagoevProteomicsStrategyElucidate2003}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CellDensity\allowbreak\_{}High}&\multicolumn{1}{m{\linkdspec}}{{\textit{IQGAP1} localization to the leading edge is blocked by high cell density that leaves no free edge.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}BLoc}}}&\multicolumn{1}{m{\linkcspec}}{Grb2}&\multicolumn{1}{m{\linkdspec}}{{\textit{Grb2} bound to active \textit{RTK} binds and recruits \textit{IQGAP1}, aiding its enrichment at the leading edge where the concentration of active \textit{RTKs} is generally higher \cite{mcnultyMAPKScaffoldIQGAP12011,bensenorIQGAP1RegulatesCell2007,blagoevProteomicsStrategyElucidate2003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{\textit{IQGAP1} interacts with focal adhesion proteins and tyrosine kinase receptors to link focal adhesion and \textit{RTK} signaling. It is thus enriched at the leading edge by increased active FA formation \cite{kohnoIQGAP1LinksPDGF2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{\textit{IQGAP1} localization to the leading edge is reinforced by horizontal polarization and the stabilization of a leading edge \cite{kohnoIQGAP1LinksPDGF2013,watanabeInteractionIQGAP1Links2004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Rac1}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{Rac1} forms a complex with \textit{IQGAP1} and \textit{CLIP-170} and recruits both to the base of the leading edge, where their aid cytoskeletal reorganization, horizontal polarization and directed migration \cite{fukataRolesRhofamilyGTPases2003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Rac1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Active \textit{Rac1} forms a complex with \textit{IQGAP1} and \textit{CLIP-170} and recruits both to the base of the leading edge, where their aid cytoskeletal reorganization, horizontal polarization and directed migration \cite{fukataRolesRhofamilyGTPases2003}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Rassf1a}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Rassf1a}=\notop\left(\mathbf{ZEB1}\orop\left(\mathbf{TGF\_{}bRI}\andop\mathbf{TGF\_{}bRII}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} The cytoskeleton-associated protein \textit{Rassf1a} serves as a scaffold to bring \textit{Smurf1} (an E3 ubiquitin-protein ligase) to \textit{RhoA}, marking it for degradation \cite{leeRASSF1ADirectlyAntagonizes2016}. It is targeted for degradation by \textit{TGF-$\beta$} signaling \cite{pefaniTGFvTargetsHippo2016} and repressed by \textit{ZEB1}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRII}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} In response to \textit{TGF-$\beta$}, \textit{Rassf1a} is recruited to \textit{TGF-$\beta$ receptor I} and targeted for degradation by the co-recruited E3 ubiquitin ligase \textit{ITCH}  \cite{pefaniTGFvTargetsHippo2016}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{TGF\allowbreak\_{}bRI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Activaton of \textit{TGF-$\beta$ receptor I} (which recruits \textit{Rassf1a}) requires a complex with the ligand as well as \textit{TGF-$\beta$ receptor II} \cite{huangTGFbetaControlCell2005}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ZEB1} is a direct transcriptional repressor of \textit{Rassf1a} \cite{guoOpposingRolesZEB12022}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{RhoA}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{RhoA}=\mathbf{AKT\_{}H}\orop\notop\mathbf{Rassf1a}\orop\left(\mathbf{RhoA}\andop\mathbf{Horizontal\_{}Pol}\andop\left(\mathbf{Myc}\orop\notop\mathbf{miR\_{}34}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GTPa}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{RhoA} is a member of the Rho family of small GTPases; it promotes reorganization of the actin cytoskeleton. It's primarily role in mesenhymal of crawling cell migration is to promote actin stress fibers formation and actomyosin contractility at the rear of a horizontally polarized cell, where it aids detachment \cite{schaksActinDynamicsCell2019}. As our model focuses exlusively on  mesenchymal migration, the active state of the \textit{RhoA} node corresponds to its activity at the rear of the cell, and thus contributes to horizontal polarity. Thus, \textit{RhoA} overexpression cannot be adequarly modeled by simply locking \textit{RhoA} ON; changing its relationship to the \textit{Horizontal\_Pol} node would also be required. At the back of the cell, \textit{RhoA} is activated by mitogen-drive \textit{AKT} signaling \cite{tripathiReceptorTyrosineKinase2017} or the absence of its inhibitor \textit{Rassf1a} \cite{leeRASSF1ADirectlyAntagonizes2016}. Alternatively, \textit{RhoA} can maintain its own activation at the back of the cell \cite{chenActivatedRhoABinds2010,medinaActivatedRhoAPositive2013} as long as its Myc-mediated transcription is not couteracted by \textit{miR-34} \cite{yamamuraMicroRNA34aSuppressesMalignant2012,chanDecipheringTranscriptionalComplex2010}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{AKT} phosphorylation of the \textit{DLC1} tumor suppressor, a Rho GTPase-activating protein (RhoGAP), increases \textit{RhoA-GTP} and thus boosts \textit{RhoA} acticity \cite{tripathiReceptorTyrosineKinase2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Rassf1a}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The cytoskeleton-associated protein \textit{Rassf1a} serves as a scaffold to bring \textit{Smurf1} (an E3 ubiquitin-protein ligase) to \textit{RhoA}, marking it for degradation \cite{leeRASSF1ADirectlyAntagonizes2016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{RhoA}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Active \textit{RhoA} binds to the PH domain of its GEF, locally increaseing GEF concentration leading and further activating of \textit{RhoA} in a direct positive feedback \cite{chenActivatedRhoABinds2010,medinaActivatedRhoAPositive2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} We assume that the positive feedback auto-regulation of \textit{RhoA} requires its concentration to the back of the cell, modeled via the ON state of the \textit{Horizontal\_Pol} node.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Myc}, together with \textit{Skp2}, \textit{Miz1} and \textit{p300}, is a direct transcriptional inducer of \textit{RhoA} \cite{chanDecipheringTranscriptionalComplex2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}34}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{miR-34} targets \textit{c-Myc} mRNA for degradation, and was shown to lower its ability to induce \textit{RhoA} transcription \cite{yamamuraMicroRNA34aSuppressesMalignant2012}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Horizontal\_{}Pol}=\notop\mathbf{ApicalBasal\_{}Pol}\andop\mathbf{ECM}\andop\mathbf{IQGAP1\_{}LeadingE}\andop\mathbf{RhoA}\andop\mathbf{FocalAdhesions}\andop\mathbf{TAZ}\andop\mathbf{FAK}\andop\mathbf{RhoA}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{For cells to establish and maintain horizontal polarization, our model requires the lack of apical-basal polarization, the presence of an ECM, as well symmetric IQGAP1 and  {\color{NavyBlue} \textit{RhoA}} localization \cite{watanabeRolesIQGAP1Cell2005,schaksActinDynamicsCell2019}, focal adhesions and spreading (FAK \cite{watanabeRolesIQGAP1Cell2005,francoControlInitialEndothelial2011} and TAZ \cite{nardoneYAPRegulatesCell2017}).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ECM}&\multicolumn{1}{m{\linkdspec}}{{Horizontal polarization requires a leading edge with lamellipodia and a trailing edge linked to stress fibers; both of which require adhesions to an ECM.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{FAK}&\multicolumn{1}{m{\linkdspec}}{{\textit{FAK} activation at nascent adhesions at the leading edge is required for ongoing cell spreading, which is a prerequisite of ongoing \textit{FA} maturation and maintenance of an active leading edge \cite{watanabeRolesIQGAP1Cell2005,francoControlInitialEndothelial2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{Horizontal polarization requires a leading edge with lamellipodia and a trailing edge linked to stress fibers, both of which depend on \textit{Focal Adhesions} linked to the actin cytoskeleton.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{TAZ}&\multicolumn{1}{m{\linkdspec}}{{\textit{TAZ}-null cells lose their ability to spread on the ECM, indicating that \textit{TAZ} transcriptional activity is required for the establishment of horizontal polarization \cite{nardoneYAPRegulatesCell2017}. (This is in contrast to \textit{YAP}-null cells, which cannot even form \textit{focal adhesions}.)}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ApicalBasal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Apical-basal and horizontal polarization are mutually exclusive; cells must first lose their textit{apical-basal polarity} before they are able to establish horizontal polarization.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{IQGAP1\allowbreak\_{}LeadingE}&\multicolumn{1}{m{\linkdspec}}{{Localization of \textit{IQGAP1} to the leading edge is required for the establishment of horizontal polarization. Together with the adenomatous polyposis coli (\textit{APC}) protein that is also recruited to the leading edge by active \textit{Rac1} and \textit{Cdc42}, \textit{IQGAP1} links the actin cytoskeleton to microtubule dynamics to establish cell polarization \cite{watanabeRolesIQGAP1Cell2005}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{RhoA}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Localization of \textit{RhoA} to the back of the cell and it's effect on cell contractility are required for horizontal polarization \cite{schaksActinDynamicsCell2019}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Rac1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Rac1}=\notop\mathbf{Casp3}\andop\mathbf{FocalAdhesions}\andop\mathbf{Necl5}\andop\left(\mathbf{Horizontal\_{}Pol}\orop\mathbf{SMO}\right)\andop\mathbf{TRIO}\andop\mathbf{AKT\_{}B}\andop\left(\notop\left(\mathbf{Merlin}\andop\mathbf{Nectin3}\andop\mathbf{J\_{}Ecadherin}\right)\orop\mathbf{Stiff\_{}ECM}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GTPa}}&\multicolumn{2}{m{\nodetwocspec}}{In our model \textit{Rac1} activation on stiff ECM requires the absence of \textit{caspase 3}, Focal Adhesion formation, \textit{Necl-5} leading edge localization,  \textit{TRIO} expression, {\color{NavyBlue} at least basal \textit{AKT} activity}, and horizontal polarization {\color{NavyBlue} or \textit{SMO} activation}. In addition, on soft ECM \textit{Rac1} activity is inhibited by the cooperative action of \textit{Merlin}, \textit{Nectin3} and \textit{E-cadherin} at adherens and tight junctions \cite{kittRapidSuppressionActivated2011,yiTightJunctionassociatedMerlinangiomotin2011}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Stiff\allowbreak\_{}ECM}&\multicolumn{1}{m{\linkdspec}}{{\textit{Rac1} activated at \textit{focal adhesions} by \textit{FAK} is enhanced by force generation supported by stiff ECM, leading to increased intracellular stiffness \cite{baeFAKCasRacLamellipodinSignalingModule2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SMO}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Shh} signaling stimulates \textit{Rac1} (and RhoA) via \textit{SMO} through a Gi protein \cite{polizioHeterotrimericGiProteins2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{AKT} phosphorylates the \textit{Rac1} GEF \textit{Tiam1}, which boosts its interaction with \textit{14-3-3} scaffold proteins and increase \textit{Tiam1} stability. This results in increased \textit{Rac1} activation and enhanced migration \cite{zhuEGFRPI3KAKT2015}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Nectin3}&\multicolumn{1}{m{\linkdspec}}{{During initial cell-cell contact and adherens junciton initiation, cadherins and nectins cooperate to briefly induce, but then rapidly suppress Rac1 \cite{kittRapidSuppressionActivated2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Necl5}&\multicolumn{1}{m{\linkdspec}}{{\textit{Necl-5} associates with integrins at the leading edge, where it promotes the activation of \textit{Rac1} and \textit{Cdc42}. \textit{Necl-5} is required for serum-and \textit{PDGF}-induced cell motility cell motility, an effect that does not require Nectin-3 binding on neighboring cells \cite{ikedaNectinlikeMolecule5Tage42004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{1}{m{\linkdspec}}{{During initial cell-cell contact and adherens junciton initiation, cadherins and nectins cooperate to briefly induce, but then rapidly suppress Rac1 \cite{kittRapidSuppressionActivated2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{\textit{Rac1} activated at \textit{focal adhesions} by \textit{FAK} is enhanced by force generation supported by stiff ECM, leading to increased intracellular stiffness \cite{baeFAKCasRacLamellipodinSignalingModule2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}GEF}}}&\multicolumn{1}{m{\linkcspec}}{TRIO}&\multicolumn{1}{m{\linkdspec}}{{\textit{TRIO} is a GEF that controls \textit{Rac1} activation during migration \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019} as well as proliferation \cite{cannetIdentificationMitoticRacGEF2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{A protein complex that includes \textit{Merlin} sequesters \textit{Angiomotin} to tight junctions, releasing it from binding the \textit{Rac1}-inhibitor \textit{Rich1} \cite{yiTightJunctionassociatedMerlinangiomotin2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Growth of the microtubule network at leading-edge lamellipodia activates \textit{Rac1} to drive local actin polymerization and further lamellipodial protrusions, thus supporting the maintenance of horizontal polarization \cite{waterman-storerMicrotubuleGrowthActivates1999}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{Caspase 3 cleaves \textit{Rac1} at two sites, resulting in the inactivation of its GTPase activity and \textit{PAK1} binding \cite{zhangCaspase3MediatedInactivation2003}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Rac1\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Rac1\_{}H}=\mathbf{Rac1}\andop\notop\mathbf{Casp3}\andop\mathbf{FocalAdhesions}\andop\mathbf{Necl5}\andop\mathbf{Horizontal\_{}Pol}\andop\mathbf{TRIO}\andop\mathbf{AKT\_{}H}\andop\mathbf{SMO}\andop\left(\notop\left(\mathbf{miR\_{}200\_{}c}\orop\mathbf{miR\_{}200\_{}b}\orop\mathbf{miR\_{}34}\right)\andop\notop\left(\mathbf{Merlin}\andop\mathbf{Nectin3}\andop\mathbf{J\_{}Ecadherin}\right)\andop\mathbf{Stiff\_{}ECM}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GTPa}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} Full (high) \textit{Rac1\_H} activation requires requires the absence of \textit{caspase 3}, a stiff ECM, Focal Adhesion formation, \textit{Necl-5} leading edge localization,  \textit{TRIO} expression, {\color{NavyBlue} at least basal \textit{AKT} activity}, horizontal polarization {\color{NavyBlue} or \textit{SMO} activation}, the absence of \textit{miR-200} \cite{tangMicroRNA200bC3pRegulate2020} or \textit{miR-34} \cite{ahnZEB1DrivesPrometastatic2012} microRNAs, as well as the absence of inhibition by the cooperative action of \textit{Merlin}, \textit{Nectin3} and \textit{E-cadherin}  \cite{kittRapidSuppressionActivated2011,yiTightJunctionassociatedMerlinangiomotin2011}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Rac1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} As the \textit{Rac1} node represents moderate levels of \textit{Rac1} activity, it's ON state is required for \textit{Rac1\_H} activation.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Stiff\allowbreak\_{}ECM}&\multicolumn{1}{m{\linkdspec}}{{\textit{Rac1} activated at \textit{focal adhesions} by \textit{FAK} is enhanced by force generation supported by stiff ECM, leading to increased intracellular stiffness \cite{baeFAKCasRacLamellipodinSignalingModule2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SMO}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Shh} signaling stimulates \textit{Rac1} (and RhoA) via \textit{SMO} through a Gi protein \cite{polizioHeterotrimericGiProteins2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{AKT} phosphorylates the \textit{Rac1} GEF \textit{Tiam1}, which boosts its interaction with \textit{14-3-3} scaffold proteins and increase \textit{Tiam1} stability. This results in increased \textit{Rac1} activation and enhanced migration \cite{zhuEGFRPI3KAKT2015}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Nectin3}&\multicolumn{1}{m{\linkdspec}}{{During initial cell-cell contact and adherens junciton initiation, cadherins and nectins cooperate to briefly induce, but then rapidly suppress Rac1 \cite{kittRapidSuppressionActivated2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Necl5}&\multicolumn{1}{m{\linkdspec}}{{\textit{Necl-5} associates with integrins at the leading edge, where it promotes the activation of \textit{Rac1} and \textit{Cdc42}. \textit{Necl-5} is required for serum-and \textit{PDGF}-induced cell motility cell motility, an effect that does not require Nectin-3 binding on neighboring cells \cite{ikedaNectinlikeMolecule5Tage42004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}Ecadherin}&\multicolumn{1}{m{\linkdspec}}{{During initial cell-cell contact and adherens junciton initiation, cadherins and nectins cooperate to briefly induce, but then rapidly suppress Rac1 \cite{kittRapidSuppressionActivated2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{\textit{Rac1} activated at \textit{focal adhesions} by \textit{FAK} is enhanced by force generation supported by stiff ECM, leading to increased intracellular stiffness \cite{baeFAKCasRacLamellipodinSignalingModule2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}GEF}}}&\multicolumn{1}{m{\linkcspec}}{TRIO}&\multicolumn{1}{m{\linkdspec}}{{\textit{TRIO} is a GEF that controls \textit{Rac1} activation during migration \cite{parkSwitchlikeEnhancementEpithelialmesenchymal2019} as well as proliferation \cite{cannetIdentificationMitoticRacGEF2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Merlin}&\multicolumn{1}{m{\linkdspec}}{{A protein complex that includes \textit{Merlin} sequesters \textit{Angiomotin} to tight junctions, releasing it from binding the \textit{Rac1}-inhibitor \textit{Rich1} \cite{yiTightJunctionassociatedMerlinangiomotin2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Growth of the microtubule network at leading-edge lamellipodia activates \textit{Rac1} to drive local actin polymerization and further lamellipodial protrusions, thus supporting the maintenance of horizontal polarization \cite{waterman-storerMicrotubuleGrowthActivates1999}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}34}&\multicolumn{1}{m{\linkdspec}}{{Though \textit{miR-34} does not appear to directly target \textit{Rac1} mRNA \cite{caoImpairedNeutrophilMigration2015}, its overexpression blocks GTP-bound (active) \textit{Rac1} \cite{ahnZEB1DrivesPrometastatic2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}b}&\multicolumn{1}{m{\linkdspec}}{{\textit{miR-200b/c-3p} represses \textit{Rac1} mRNA by targeting its  3' UTR \cite{tangMicroRNA200bC3pRegulate2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{\textit{miR-200b/c-3p} represses \textit{Rac1} mRNA by targeting its  3' UTR \cite{tangMicroRNA200bC3pRegulate2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{Caspase 3 cleaves \textit{Rac1} at two sites, resulting in the inactivation of its GTPase activity and \textit{PAK1} binding \cite{zhangCaspase3MediatedInactivation2003}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{PAK1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{PAK1}=\mathbf{Rac1}\orop\mathbf{Rac1\_{}H}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{p21}-Activated kinase 1 \textit{PAK1}, a serine-threonine kinase interacts with the Rho GTPases \textit{RAC1} and \textit{CDC42} to drive migration, survival, cell cycle, EMT, stress response and inflammation \cite{yaoP21ActivatedKinaseEmerging2020}. It is activated by \textit{Rac1}, which binds to \textit{PAK1} and stimulates its kinase activity \cite{knausStructuralRequirementsPAK1998}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Rac1}&\multicolumn{1}{m{\linkdspec}}{\textit{Rac1} binds to \textit{PAK1} and stimulates its kinase activity \cite{knausStructuralRequirementsPAK1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Rac1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue}\textit{Rac1} binds to \textit{PAK1} and stimulates its kinase activity \cite{knausStructuralRequirementsPAK1998}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Migration}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Migration}=\mathbf{Horizontal\_{}Pol}\andop\mathbf{Stress\_{}Fibers}\andop\mathbf{FocalAdhesions}\andop\mathbf{PAK1}\andop\mathbf{Rac1}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} In the current model, the \textit{Migration} node represents migration by a polarized cell, whereas \textit{Fast\_Migration} tracks rapid mesenchymal migration typical of a cell that underwent EMT. \textit{Migration} requires horizontal polarization, stress fiber maintenance, focal adhesion formation and the activity of {\color{NavyBlue}\textit{Rac1}} and \textit{PAK1} kinases \cite{depascalisSingleCollectiveCell2017,bokochBiologyP21ActivatedKinases2003}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{Mesenchymal style migration requires force generation via stress fibers anchored by \textit{focal adhesions} \cite{depascalisSingleCollectiveCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Stress\allowbreak\_{}Fibers}&\multicolumn{1}{m{\linkdspec}}{{Mesenchymal style migration requires force generation via stress fibers anchored by \textit{focal adhesions} \cite{depascalisSingleCollectiveCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Directed mesenchymal style migration requires \textit{horizontal cell polarization} with a well defined leading and trailing edge \cite{depascalisSingleCollectiveCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Rac1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Rac1} GTPase signals through \textit{WRC} (WAVE regulatory complex) to drive \textit{Arp2/3}-dependent lamellipodia formation at the leading edge of a migrating cell \cite{schaksActinDynamicsCell2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{Active \textit{PAK1} coordinates a series of cytoskeletal changes at the leading edge that are required for migration and invasion. These include: a) inhibition of myosin light chain kinase in order to decrease contractility if the leading lamellipodium and loss of established actin stress fibers and very strong \textit{focal adhesions} at the leading edge; b) retraction of protrusions and the cell body at the sides and trailing edge with no active \textit{PAK1}; c) suppressing actin filament turnover and promoting leading edge stabilization; d) promote membrane ruffle formation \cite{depascalisSingleCollectiveCell2017,bokochBiologyP21ActivatedKinases2003}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Fast\allowbreak\_{}Migration}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Fast\_{}Migration}=\mathbf{Migration}\andop\mathbf{Horizontal\_{}Pol}\andop\mathbf{Stress\_{}Fibers}\andop\mathbf{FocalAdhesions}\andop\mathbf{PAK1}\andop\mathbf{Rac1\_{}H}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{OliveGreen} The \textit{Fast\_Migration} node represents sustained, rapid mesenchymal migration of a polarized cell. This requires horizontal polarization, stress fiber maintenance, focal adhesion formation, \textit{PAK1} activity, and high levels of active \textit{Rac1} \cite{depascalisSingleCollectiveCell2017,bokochBiologyP21ActivatedKinases2003}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Migration}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} As the \textit{Migration} node represents moderate levels of movement, it's ON state is required for \textit{Fast\_Migration}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{FocalAdhesions}&\multicolumn{1}{m{\linkdspec}}{{Mesenchymal style migration requires force generation via stress fibers anchored by \textit{focal adhesions} \cite{depascalisSingleCollectiveCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Stress\allowbreak\_{}Fibers}&\multicolumn{1}{m{\linkdspec}}{{Mesenchymal style migration requires force generation via stress fibers anchored by \textit{focal adhesions} \cite{depascalisSingleCollectiveCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Horizontal\allowbreak\_{}Pol}&\multicolumn{1}{m{\linkdspec}}{{Directed mesenchymal style migration requires \textit{horizontal cell polarization} with a well defined leading and trailing edge \cite{depascalisSingleCollectiveCell2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Rac1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Rac1} GTPase signals through \textit{WRC} (WAVE regulatory complex) to drive \textit{Arp2/3}-dependent lamellipodia formation at the leading edge of a migrating cell \cite{schaksActinDynamicsCell2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{{Active \textit{PAK1} coordinates a series of cytoskeletal changes at the leading edge that are required for migration and invasion. These include: a) inhibition of myosin light chain kinase in order to decrease contractility if the leading lamellipodium and loss of established actin stress fibers and very strong \textit{focal adhesions} at the leading edge; b) retraction of protrusions and the cell body at the sides and trailing edge with no active \textit{PAK1}; c) suppressing actin filament turnover and promoting leading edge stabilization; d) promote membrane ruffle formation \cite{depascalisSingleCollectiveCell2017,bokochBiologyP21ActivatedKinases2003}.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{EMT module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SNAI1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SNAI1}=\left(\mathbf{PAK1}\andop\left(\left(\left(\notop\mathbf{GSK3}\right)\orop\left(\mathbf{NfkB}\andop\mathbf{GLI}\right)\right)\orop\left(\left(\notop\mathbf{miR\_{}34}\right)\andop\left(\mathbf{ZEB1\_{}H}\orop\mathbf{ZEB1}\right)\right)\right)\right)\orop\left(\left(\mathbf{NfkB}\andop\mathbf{GLI}\andop\left(\notop\left(\mathbf{miR\_{}34}\andop\mathbf{GSK3}\right)\right)\right)\andop\left(\mathbf{ZEB1\_{}H}\orop\mathbf{ZEB1}\right)\right)\orop\left(\mathbf{ZEB1\_{}H}\andop\mathbf{ZEB1}\andop\left(\left(\notop\left(\mathbf{miR\_{}34}\orop\mathbf{GSK3}\right)\right)\orop\left(\mathbf{NfkB}\orop\mathbf{PAK1}\orop\mathbf{HMGA2}\orop\mathbf{HMGA1}\right)\right)\right)\orop\left(\mathbf{SMAD2\_{}3\_{}4}\andop\notop\left(\mathbf{miR\_{}34}\orop\mathbf{GSK3}\right)\right)\orop\mathbf{HMGA2}\orop\mathbf{HMGA1}\orop\mathbf{GLI\_{}H}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{SNAI1}, also known as \textit{Snail}, is a master inducer of EMT, responsible for starting the transcriptional cascade that locks in the EMT regulatory switch \cite{tianCoupledReversibleIrreversible2013}. In epithelial cells \textit{SNAI1} is repressed by RNA interference by \textit{miR-34} \cite{siemensMiR34SNAILForm2011} and protein degradation by \textit{GSK3} \cite{zhengGlycogenSynthaseKinase32013}. Transcriptional \textit{NF-$\kappa$B} \cite{minNfkBEpithelialMesenchymal2008}  {\color{NavyBlue} and \textit{GLI} induction} along with loss of these repressive mechanism, and nuclear localization by \textit{PAK1} \cite{yangPak1PhosphorylationSnail2005} can activate \textit{SNAI1}, starting EMT. In committed mesenhymal or hybrid E/M cells \textit{ZEB1} lowers the barrier to maitnaining high levels of \textit{SNAI1}.} {Downstream of \textit{TGF}$\beta$, {\color{NavyBlue} \textit{SMAD2/3/4},} \textit{HMGA2} or {\color{NavyBlue} \textit{HMGA1}} further aid its transcriptiopnal upregulation of \cite{thuaultHMGA2SmadsCoregulate2008}.} {\color{NavyBlue} Finally, high \textit{GLI} activity downstream of \textit{SHH} signaling also activates \textit{SNAI1} \cite{liSnailInductionEarly2006}.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Snail/Snai1} is rapidly induced by \textit{GLI1} \cite{liSnailInductionEarly2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Snail/Snai1} is rapidly induced by \textit{GLI1} \cite{liSnailInductionEarly2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{{\textit{GSK3} both degrades and prevents the transcription of \textit{SNAI1} \cite{zhengGlycogenSynthaseKinase32013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{PAK1}&\multicolumn{1}{m{\linkdspec}}{{\textit{PAK1} phosphorylation of \textit{SNAI1} activates and localizes \textit{SNAI1} the nucleus \cite{yangPak1PhosphorylationSnail2005}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{The transcription factor \textit{NF-$\kappa$B} promotes the expression of \textit{SNAI1} \cite{minNfkBEpithelialMesenchymal2008}, and \textit{NF-$\kappa$B} inhibition can  lower \textit{SNAI1} expression \cite{minNfkBEpithelialMesenchymal2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Active, nuclear \textit{Smad2/3/4} directly bind and activate the \textit{SNAI1} promoter, though likely requires cooperation from other factors that recruit \textit{HDACs} \cite{xuHistoneDeacetylaseInhibitors2018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{HMGA2}&\multicolumn{1}{m{\linkdspec}}{{\textit{HMGA2} binds to the \textit{SNAI1} promoter to induce its expression, and cooperates with \textit{Smads} by binding to them and increasing their affinity to the  \textit{SNAI1} promoter \cite{thuaultHMGA2SmadsCoregulate2008}. Thus, \textit{TGF}$\beta$ activates the EMT transciptional program via \textit{HMGA2} \cite{thuaultTransformingGrowthFactorbeta2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{HMGA1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} siRNA against \textit{HMGA1} strongly reduces \textit{SNAI1} protein expression, though it is unclear whether this is the result of direct transcriptional repression similar to \textit{HMGA2} \cite{zhongAssociationS100A13HMGA12016}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}34}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1} mRNA is a direct target of \textit{miR-34} suppression \cite{siemensMiR34SNAILForm2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is an indirect transcriptional inducer of \textit{SNAI1} \cite{zhangATMmediatedStabilizationZEB12014}, likely by acting as a competitive inhibitor of microRNAs such as \textit{miR-34} \cite{kimCompetingEndogenousRNA2021}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is an indirect transcriptional inducer of \textit{SNAI1} \cite{zhangATMmediatedStabilizationZEB12014}, likely by acting as a competitive inhibitor of microRNAs such as \textit{miR-34} \cite{kimCompetingEndogenousRNA2021}. }}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{LEF1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{LEF1}=\left(\mathbf{ZEB1}\andop\notop\mathbf{miR\_{}200\_{}c}\right)\orop\mathbf{ZEB1\_{}H}\orop\mathbf{NfkB}\orop\mathbf{SMAD2\_{}3\_{}4}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{Lymphoid enhancer-binding factor 1, or \textit{LEF1} is a high-mobility group transcription factor and mediator of \textit{Wnt/$\beta$-catenin} signaling. In addition to promotong proliferation, \textit{LEF1} helps induce EMT by activating the transcription \textit{SNAI2} and \textit{ZEB1} \cite{santiagoWntSignalingPathway2017}. It is induced by  \textit{NF-$\kappa$B} \cite{yunNfkBRegulatesLef12007} {or \textit{Smad2/4} \cite{nawshadTGFbeta3SignalingActivates2003}}, its transcriptional potency is boosted by \textit{ZEB1} \cite{rosmaninhoZeb1PotentiatesGenome2018}, and it is targeted for degradation by \textit{miR-200}.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{\textit{LEF1} is a direct transcriptipnal target of \textit{NF-$\kappa$B} \cite{yunNfkBRegulatesLef12007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{Phosphorilated \textit{Smad2} and \textit{Smad4} can induce \textit{LEF1} gene expression \cite{nawshadTGFbeta3SignalingActivates2003}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{\textit{miR-200s} are indirect repressors of \textit{LEF1}, limiting basal \textit{Pitx2} and \textit{$\beta$-catenin} complexes from inducing \textit{LEF1} transcription \cite{sharpPituitaryHomeoboxPitx22014}. Here we assume that medium \textit{ZEB1} availability can aid \textit{LEF1}-mediated transcription if \textit{miR-200c} is repressed, while high levels of \textit{ZEB1} can override \textit{miR-200c}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} can bind to and significantly boost \textit{LEF1}-mediated transcription \cite{rosmaninhoZeb1PotentiatesGenome2018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} can bind to and significantly boost \textit{LEF1}-mediated transcription \cite{rosmaninhoZeb1PotentiatesGenome2018}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Twist}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Twist}=\left(\notop\mathbf{Casp3}\right)\andop\left(\left(\mathbf{SNAI1}\andop\left(\mathbf{NfkB}\orop\left(\notop\mathbf{miR\_{}34}\right)\right)\right)\orop\mathbf{HMGA2}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{ \textit{Twist} is a master trancriptional regulator of the EMT program, induced by \textit{NF-$\kappa$B} \cite{phamUpregulationTwist1NfkB2007} {or \textit{HMGA2} \cite{tanRegulationTranscriptionFactor2012}}, stabilized and aided by \textit{SNAI1}, and cleaved during apoptosis by \textit{Caspase 3} \cite{demontisTwistSubstrateCaspase2006}.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{Transcription of \textit{Twist} is induced by \textit{NF-$\kappa$B} \cite{phamUpregulationTwist1NfkB2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{HMGA2}&\multicolumn{1}{m{\linkdspec}}{{\textit{HMGA2} is a direct transcriptional inducer of \textit{Twist}, and responsible for its increase in response to \textit{TGF$\beta$} \cite{tanRegulationTranscriptionFactor2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1} is required a rapid inrease in \textit{Twist} protein levels, and aids its subsequent transcription in response to \textit{TGF$\beta$} \cite{daveFunctionalCooperationSnail12011}. In addition, \textit{SNAI1} potentiates \textit{Twist}-mediated enhancer activation \cite{remboldConservedRoleSnail2014}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}34}&\multicolumn{1}{m{\linkdspec}}{{\textit{Twist}'s 3'UTR is a direct target of \textit{miR-34} \cite{imaniMicroRNA34aTargetsEpithelial2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{\textit{Twist} is a direct proteolytic target of \textit{Caspase 3} \cite{demontisTwistSubstrateCaspase2006}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SNAI2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SNAI2}=\left(\mathbf{Twist}\andop\left(\left(\mathbf{SNAI2}\orop\mathbf{N\_{}bcatenin}\right)\orop\left(\mathbf{N\_{}bcatenin\_{}H}\andop\mathbf{LEF1}\right)\right)\right)\orop\mathbf{SMAD2\_{}3\_{}4}\orop\mathbf{HMGA2}\orop\mathbf{HMGA1}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{SNAI2}, also known as \textit{Slug}, is a prototypical EMT transcription factor that regulates tissue development and tumorigenesis \cite{zhouMolecularRegulationSnai22019}. It is induced / maintained in mesenhymal cells by \textit{Twist} \cite{casasSnail2EssentialMediator2011}, \textit{$\beta$-catenin} \cite{saegusaRequirementAktBetacatenin2009,zhouMolecularRegulationSnai22019}, \textit{LEF1} \cite{lambertiniSLUGNewTarget2010} as well as positive auto-regulation \cite{kumarAutoregulationSlugMediates2015}. In response to \textit{TGF$\beta$} it is induced by \textit{Smad3} \cite{moritaDualRolesMyocardinrelated2007}, \textit{HMGA2} \cite{thuaultTransformingGrowthFactorbeta2006} {\color{NavyBlue} or \textit{HMGA1}\cite{gandhiBimodalFunctionChromatin2020}}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin}&\multicolumn{1}{m{\linkdspec}}{{The \textit{Wnt/$\beta$-catenin} pathway promotes \textit{SNAI2} transcription through nuclear \textit{$\beta$-catenin} \cite{saegusaRequirementAktBetacatenin2009,zhouMolecularRegulationSnai22019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{SMAD3} bind the cis-element in the SNAI2 promoter and recruits myocardin-related transcription factors to activate its transcription \cite{moritaDualRolesMyocardinrelated2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{HMGA2}&\multicolumn{1}{m{\linkdspec}}{{\textit{HMGA2} is a direct transcriptional inducer of \textit{SNAI2} \cite{thuaultTransformingGrowthFactorbeta2006}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{LEF1}&\multicolumn{1}{m{\linkdspec}}{{\textit{LEF1} is a direct transcriptional inducer of \textit{SNAI2} \cite{lambertiniSLUGNewTarget2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI2}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI2} is able to bind to its own promoter and induce transcription of its own mRNA \cite{kumarAutoregulationSlugMediates2015}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Twist}&\multicolumn{1}{m{\linkdspec}}{{\textit{Twist} is a direct transcriptional inducer of \textit{SNAI2} \cite{casasSnail2EssentialMediator2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{HMGA1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Loss of \textit{HMGA1} strongly reduces \textit{SNAI2} levels, though it is unclear whether this is the result of direct transcriptional repression \cite{gandhiBimodalFunctionChromatin2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{The \textit{Wnt/$\beta$-catenin} pathway promotes \textit{SNAI2} transcription through nuclear \textit{$\beta$-catenin} \cite{saegusaRequirementAktBetacatenin2009,zhouMolecularRegulationSnai22019}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ZEB1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ZEB1}=\mathbf{SNAI2}\orop\left(\mathbf{Twist}\andop\mathbf{SMAD2\_{}3\_{}4}\right)\orop\left(\mathbf{b\_{}catenin\_{}TCF4}\andop\notop\left(\mathbf{miR\_{}200\_{}b}\orop\mathbf{miR\_{}200\_{}c}\right)\right)\orop\mathbf{GLI}\orop\notop\mathbf{GLI3R}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{Zinc finger E-box binding homeobox 1  or \textit{ZEB1} is one of the core regulators of the EMT transcriptional switch \cite{drapelaZEB1CriticalRegulator2020}. It is induced by \textit{SNAI2} \cite{welsTranscriptionalActivationZEB12011}, nuclear \textit{$\beta$-catenin/TCF4} \cite{sanchez-tilloVcateninTCF4Complex2011}, in the absence of the epithelial \textit{miR-200} microRNAs  \cite{liMiR200CanRepress2014,titleGeneticDissectionMiR200Zeb12018}, { \color{NavyBlue} \textit{GLI2}, or the absence of the \textit{GLI} repressor  \textit{GLIR}}. As \textit{ZEB1} has two distinct activation levels in hybrid E/M cells vs fully mesenchynmal ones \cite{jollyImplicationsHybridEpithelial2015, luMicroRNAbasedRegulationEpithelialhybridmesenchymal2013,zhangTGFvinducedEpithelialtomesenchymalTransition2014}, we modeled \textit{ZEB1} activity with two nodes; this one represents at least medium \textit{ZEB1} activity (characteristic of hybrid E/M cells and compatible with ongoing \textit{miR-200} expression), while the \textit{ZEB1\_H} node representing maximal \textit{ZEB1} activation seen in fully mesenchymal cells.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of \textit{ZEB1} \cite{perrotGLI2CooperatesZEB12013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI3R}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} As activator and repressor forms of the GLI transcription factors share the same DNA binding motifs, we assume that \textit{GLI} competes with \textit{GLI3R} for access to the GLI binding site on their target genes such as \textit{ZEB1} \cite{chaudhryGLI3RepressorDetermines2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} The type I \textit{TGF-$\beta$} receptor (\textit{T$\beta$RI}) phosphorylates receptor-bound \textit{Smad2} and \textit{Smad3}, which triggers their nuclear translocation. Together with their activating partner \textit{Smad4}, they aid (and are required for) \textit{Twist}-induced transcription of \textit{Zeb1}  \cite{seoaneTGFvFamilySignaling2017}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI2}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI2} promotes \textit{ZEB1} transcription \cite{welsTranscriptionalActivationZEB12011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Twist}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Twist} works together with \textit{Smad2/3/4} active complexes to induce \textit{Zeb1} expression \cite{seoaneTGFvFamilySignaling2017}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}b}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen}\textit{miR\_200} inhibits mRNA expression of \textit{ZEB1} by targeting its mRNA for destruction \cite{parkMiR200FamilyDetermines2008,liMiR200CanRepress2014,titleGeneticDissectionMiR200Zeb12018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen}\textit{miR\_200} inhibits mRNA expression of \textit{ZEB1} by targeting its mRNA for destruction \cite{parkMiR200FamilyDetermines2008,liMiR200CanRepress2014,titleGeneticDissectionMiR200Zeb12018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{1}{m{\linkdspec}}{{Nuclear \textit{$\beta$-catenin/TCF4} are direct transcriptional inducers of the \textit{ZEB1} promoter \cite{sanchez-tilloVcateninTCF4Complex2011}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{N\_{}bcatenin\_{}H}=\mathbf{N\_{}bcatenin}\andop\left(\notop\mathbf{miR\_{}34}\right)\andop\left(\notop\mathbf{J\_{}acatenin}\orop\mathbf{SMAD2\_{}3\_{}4}\right)\andop\notop\left(\left(\mathbf{miR\_{}200\_{}c}\orop\mathbf{miR\_{}200\_{}b}\right)\andop\mathbf{GSK3}\right)\andop\notop\left(\left(\mathbf{CyclinE}\orop\mathbf{CyclinA}\right)\andop\mathbf{GSK3}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{N\_bcatenin\_H} node represents maximal nuclear \textit{$\beta$-catenin} accumulation. This requires \textit{N\_bcatenin} = ON, a lack of \textit{miR-34}, either complete absence of junctions (\textit{J\_acatenin} = OFF) {or activation by \textit{Smads}}, and the lack of joint repression \textit{GSK3} and either \textit{miR-200} or \textit{CyclinE/A}-bound \textit{Cdk2}.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{{When \textit{GSK3} is inhibited, unphosphorylated \textit{$\beta$-catenin} accumulates, translocates to the nucleus, and promotes transcription \cite{bienzVCateninPivotCell2005}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{J\allowbreak\_{}acatenin}&\multicolumn{1}{m{\linkdspec}}{{Junctional \textit{E-cadherins} recruits/sequesters \textit{$\beta$-catenin} to adherens junctions and block \textit{$\beta$-catenin} nuclear localization \cite{tianECadherinCateninComplex2011}. We assume that cells unable to form any adherens junctions have high nuclear \textit{$\beta$-catenin}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin}&\multicolumn{1}{m{\linkdspec}}{{Reaching the \textit{N\_bcatenin\_H} = ON state requires \textit{N\_bcatenin} = ON first.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$} stimulation leads to increased binding of \textit{SMAD3/4} with \textit{$\beta$-catenin}, which in turn increases its nuclear accumulation and \textit{$\beta$-catenin}-mediated transcription \cite{nishitaInteractionWntTGFbeta2000,tianInteractionTransformingGrowth2002}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}34}&\multicolumn{1}{m{\linkdspec}}{{\textit{$\beta$-catenin}'s 3' UTR is a direct \textit{miR-34} target \cite{kimP53MicroRNA34Are2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}b}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen}\textit{$\beta$-catenin}'s 3' UTR is a direct \textit{miR-200} target \cite{kangMicroRNA200aSuppressesWnt2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen}\textit{$\beta$-catenin}'s 3' UTR is a direct \textit{miR-200} target \cite{kangMicroRNA200aSuppressesWnt2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{{\textit{cyclin E/Cdk2} phosphorylate \textit{$\beta$-catenin} on Ser33, Ser37, Thr41, and Ser45, promoting its rapid proteasomal degradation \cite{parkModulationBetacateninPhosphorylation2004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{{\textit{cyclin A/Cdk2} phosphorylate \textit{$\beta$-catenin} , promoting its rapid degradation \cite{parkModulationBetacateninPhosphorylation2004}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ZEB1\_{}H}=\mathbf{ZEB1}\andop\mathbf{LEF1}\andop\left(\mathbf{SNAI2}\orop\mathbf{GLI}\orop\notop\mathbf{miR\_{}200\_{}c}\right)\andop\left(\mathbf{N\_{}bcatenin\_{}H}\orop\left(\mathbf{GLI\_{}H}\andop\notop\mathbf{GLI3R}\right)\orop\mathbf{b\_{}catenin\_{}TCF4}\orop\mathbf{HMGA1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{ZEB1\_H} node represents maximal \textit{ZEB1} activation seen in fully mesenchymal cells \cite{jollyImplicationsHybridEpithelial2015, luMicroRNAbasedRegulationEpithelialhybridmesenchymal2013,zhangTGFvinducedEpithelialtomesenchymalTransition2014}. In our model this requires medium \textit{ZEB1}, \textit{LEF1} \cite{kobayashiTranscriptionFactorLEF12013},  either \textit{SNAI2} \cite{welsTranscriptionalActivationZEB12011}, {\color{NavyBlue}\textit{GLI} \cite{perrotGLI2CooperatesZEB12013}} or the absence of \textit{miR-200c}  \cite{liMiR200CanRepress2014,titleGeneticDissectionMiR200Zeb12018}, and one of the following: high nuclear \textit{$\beta$-catenin} \cite{sanchez-tilloVcateninTCF4Complex2011}, {\color{NavyBlue}high \textit{GLI} without the inhibitory \textit{GLI3R} \cite{perrotGLI2CooperatesZEB12013,chaudhryGLI3RepressorDetermines2017}, \textit{$\beta$-catenin/TCF4} complex activation, or \textit{HMGA1} \cite{aiHMGA1AggravatesOxidative2023}}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of \textit{ZEB1} \cite{perrotGLI2CooperatesZEB12013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of \textit{ZEB1} \cite{perrotGLI2CooperatesZEB12013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI3R}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} As activator and repressor forms of the GLI transcription factors share the same DNA binding motifs, we assume that \textit{GLI} competes with \textit{GLI3R} for access to the GLI binding site on their target genes such as \textit{ZEB1} \cite{chaudhryGLI3RepressorDetermines2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{LEF1}&\multicolumn{1}{m{\linkdspec}}{{\textit{LEF1} overexpression lead to a substantial increase in \textit{ZEB1}, indicating that elevated levels of \textit{LEF1} can help push \textit{ZEB1} into its mesenchymal-specific high expression range (\textit{LEF1\_H} node) \cite{kobayashiTranscriptionFactorLEF12013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI2}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI2} promotes \textit{ZEB1} transcription \cite{welsTranscriptionalActivationZEB12011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{\color{OliveGreen}\textit{miR-200} reduces mRNA expression of \textit{ZEB1} by targeting its mRNA for destruction \cite{parkMiR200FamilyDetermines2008,liMiR200CanRepress2014,titleGeneticDissectionMiR200Zeb12018}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{As the \textit{ZEB1} node represents moderate levels of this trascription factor,  \textit{ZEB1\_H} = ON requires \textit{ZEB1} = ON.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{Nuclear \textit{$\beta$-catenin/TCF4} are direct transcriptional inducers of the \textit{ZEB1} promoter \cite{sanchez-tilloVcateninTCF4Complex2011}. We assume that high levels of nuclear \textit{$\beta$-catenin} are required for turning on the \textit{ZEB1\_H} node.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Nuclear \textit{$\beta$-catenin/TCF4} are direct transcriptional inducers of the \textit{ZEB1} promoter \cite{sanchez-tilloVcateninTCF4Complex2011}. We assume that high levels of nuclear \textit{$\beta$-catenin} are required for turning on the \textit{ZEB1\_H} node.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{HMGA1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{HMGA1} is a transcriptional inducer of \textit{JMJD3}, which in turn directly demethylates the \textit{ZEB1} promoter to increasde its transcription \cite{aiHMGA1AggravatesOxidative2023}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{b\_{}catenin\_{}TCF4}=\mathbf{N\_{}bcatenin\_{}H}\andop\mathbf{SNAI1}\andop\mathbf{SNAI2}\andop\notop\mathbf{miR\_{}200\_{}c}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{{The \textit{b\_catenin\_TCF4} node represents saturating levels of active nuclear \textit{$\beta$-catenin/TCF4} transcriptional activity. In addition to the influences required to accumulate high nuclear \textit{$\beta$-catenin}, this node's ON state also requires \textit{SNAI1/2} expression (both factors promote the formation of active \textit{$\beta$-catenin/TCF4} transcriptional complexes) \cite{mediciSnailSlugPromote2008} {\color{NavyBlue} and the absence of inhibitory \textit{miR-200c} \cite{kangMicroRNA200aSuppressesWnt2011}}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1/2} promote the formation of active \textit{$\beta$-catenin/TCF4} transcriptional complexes \cite{mediciSnailSlugPromote2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SNAI2}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1/2} promote the formation of active \textit{$\beta$-catenin/TCF4} transcriptional complexes \cite{mediciSnailSlugPromote2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{Reaching saturating levels of active nuclear \textit{$\beta$-catenin/TCF4} complex formation requires maximal nuclear \textit{$\beta$-catenin} accumularion (\textit{N\_bcatenin\_H} = ON).}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin}'s 3' UTR is a direct \textit{miR-200} target \cite{kangMicroRNA200aSuppressesWnt2011}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ZEB2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ZEB2}=\left(\notop\mathbf{miR\_{}200\_{}c}\right)\andop\left(\mathbf{NfkB}\orop\mathbf{SMAD2\_{}3\_{}4}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} Zinc finger E-box binding homeobox 2  or \textit{ZEB2} is a core regulator of the EMT transcriptional switch. \textit{ZEB2} is transcriptionally induced by \textit{NF-$\kappa$B} and/or \textit{SMAD2/3/4} \cite{katohIntegrativeGenomicAnalyses2009}, and blocked by \textit{miR\_200c}-mediated mRNA degradation \cite{parkMiR200FamilyDetermines2008}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{NF-$\kappa$B} is a direct transcriptional inducer of \textit{ZEB2} downstream of textit{TNF-$\alpha$} or \textit{IL1} signaling \cite{katohIntegrativeGenomicAnalyses2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{SMAD2/3/4} complexes are direct transcriptional inducers of \textit{ZEB2} downstream of textit{TGF-$\beta$}  signaling \cite{katohIntegrativeGenomicAnalyses2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}RNAi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue}\textit{miR\_200c} inhibits mRNA expression of \textit{ZEB2} by targeting its mRNA for destruction \cite{parkMiR200FamilyDetermines2008}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{HMGA1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{HMGA1}=\mathbf{ZEB2}\orop\left(\mathbf{b\_{}catenin\_{}TCF4}\andop\mathbf{N\_{}bcatenin\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{NavyBlue} \textit{HMGA1} is a high mobility group (HMG) proteininvolved in transcriptional regulation via chromatin remodeling. It is directly induced by \textit{$\beta$-catenin/TCF-4}-mediated transcription \cite{bushWntVcateninTcell2013}, and indirectly stabilized by \textit{ZEB2} repression of the the \textit{miR-637} mRNA (which targets its mRNA for degradation) \cite{zengOncogenicZEB2MiR6372021}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ZEB2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ZEB2} is a direct transcriptional repressor of the \textit{miR-637} promoter, a microRNA that targets \textit{HMGA1} mRNA for destruction \cite{zengOncogenicZEB2MiR6372021}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{N\allowbreak\_{}bcatenin\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin/TCF-4} complexes are direct transcriptional inducers of \textit{HMGA1} \cite{bushWntVcateninTcell2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{$\beta$-catenin/TCF-4} complexes are direct transcriptional inducers of \textit{HMGA1} \cite{bushWntVcateninTcell2013}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{TGFb\allowbreak\_{}secr}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{TGFb\_{}secr}=\left(\left(\mathbf{b\_{}catenin\_{}TCF4}\orop\left(\mathbf{GLI\_{}H}\andop\notop\mathbf{GLI3R}\right)\right)\andop\notop\left(\mathbf{miR\_{}200\_{}b}\orop\mathbf{miR\_{}200\_{}c}\right)\right)\andop\left(\notop\mathbf{Casp3}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Secr}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{TGFb\_secr} node represents the production and secretion of \textit{TGF$\beta$} by the modeled cell. \textit{TGF$\beta$} transcription in mesenchymal cells is driven by nuclear \textit{$\beta$-catenin/TCF4}- {\color{NavyBlue}or -\textit{GLI2}-}mediated transcription, while \textit{TGF$\beta$} mRNA is targeted for degradation by \textit{miR-200} mircoRNAs \cite{gregoryAutocrineTGFbetaZEB2011}. {\color{NavyBlue}Finally, apoptotic \textit{Caspase} activation blocks the secretory pathway responsible for \textit{TGF$\beta$} release \cite{loweCaspasemediatedCleavageSyntaxin2004}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of \textit{TGF-$beta$1} transcription \cite{furlerGLI2RegulatesTGFv12012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI3R}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} As activator and repressor forms of the GLI transcription factors share the same DNA binding motifs, we assume that \textit{GLI} competes with \textit{GLI3R} for access to the GLI binding site on their target genes such as \textit{ZEB1} \cite{chaudhryGLI3RepressorDetermines2017}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}b}&\multicolumn{1}{m{\linkdspec}}{{color{OliveGreen}Overexpression of \textit{miR-200} significantly reduces mRNA levels of \textit{TGF$\beta$}, with the strongest effect on \textit{TGF$\beta$3} \cite{gregoryAutocrineTGFbetaZEB2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{color{OliveGreen}Overexpression of \textit{miR-200} significantly reduces mRNA levels of \textit{TGF$\beta$}, with the strongest effect on \textit{TGF$\beta$3} \cite{gregoryAutocrineTGFbetaZEB2011}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{b\allowbreak\_{}catenin\allowbreak\_{}TCF4}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF$\beta$3} is directly regulated by activated \textit{$\beta$-catenin} \cite{yooVcateninActivatesTGFvinduced2020}, while \textit{SNAI1/2} promote the formation of \textit{$\beta$-catenin/TCF4} complexes, which  bind to the \textit{TGF$\beta$3} promoter induce its transcription \cite{mediciSnailSlugPromote2008}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} During apopotsis, caspases cleave key components vesicular transport between the ER and Golgi, resulting in a complete block in ER-mediated protein secretion \cite{loweCaspasemediatedCleavageSyntaxin2004}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{miR\allowbreak\_{}34}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{miR\_{}34}=\left(\notop\mathbf{SNAI1}\right)\orop\left(\notop\left(\mathbf{ZEB1}\orop\mathbf{ZEB1\_{}H}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}miR}}&\multicolumn{2}{m{\nodetwocspec}}{{\textit{miR-34} is an microRNA expressed in epithelial cells and central to blocking the accumulation of EMT-initiating transcription factors such as \textit{SNAI1} and \textit{$\beta$-catenin}  \cite{zhangMicroRNA34FamilyPotential2019}. \textit{SNAI1}, together with \textit{ZEB1}, feed back to repress \textit{miR-34} expression in mesenhymal and hybrid E/M cells \cite{nieRolesMicroRNA34aEpithelial2019}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1} is a direct inhibitor of \textit{miR-34} transcription  \cite{nieRolesMicroRNA34aEpithelial2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct inhibitor of \textit{miR-34} transcription \cite{nieRolesMicroRNA34aEpithelial2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct inhibitor of \textit{miR-34} transcription \cite{nieRolesMicroRNA34aEpithelial2019}.}}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{miR\allowbreak\_{}200\allowbreak\_{}b}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{miR\_{}200\_{}b}=\mathbf{p21}\andop\notop\left(\left(\mathbf{Twist}\andop\mathbf{SNAI1}\andop\left(\mathbf{ZEB1}\orop\notop\mathbf{c\_{}Myb}\right)\right)\orop\mathbf{ZEB1\_{}H}\orop\mathbf{ZEB2}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}miR}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{OliveGreen} The \textit{miR\_200\_b} node represents the miR-200b microRNA expressed in epithelial cells and central to blocking EMT transcription factors \cite{mongrooRoleMiR200Family2010}. Its levels are increased by \textit{p21} \cite{liP21ZEB1ComplexInhibits2014}, it is directly induced by \textit{c-Myb} \cite{pieraccioliActivationMiR200CMyb2013} and repressed by \textit{ZEB1/2} \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016}. This repression is indirectly supported by \textit{SNAI1} \cite{gillSnailMicroRNA200Family2011,moesNovelNetworkIntegrating2012} and \textit{Twist} \cite{daveFunctionalCooperationSnail12011a}.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{c\allowbreak\_{}Myb}&\multicolumn{1}{m{\linkdspec}}{{The proto-oncogene \textit{c-Myb} induces the expression of the \textit{miR-200} family, unless the locus is silenced by DNA methylation \cite{pieraccioliActivationMiR200CMyb2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1} induction reduces the expression of \textit{miR-200} \cite{gillSnailMicroRNA200Family2011,moesNovelNetworkIntegrating2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Twist}&\multicolumn{1}{m{\linkdspec}}{{\textit{Twist} upregulation is required to maintain high levels of \textit{ZEB1} (\textit{Twist} is a direct enhancer of \textit{ZEB1}) \cite{daveFunctionalCooperationSnail12011a}. Thus, we assume that \textit{ZEB1} and \textit{SNAI1} cannot fully repress the \textit{miR-200} cluster in the absence of \textit{Twist}. In addition, overexpression of \textit{Twist} resulted in DNA methylation of the  \textit{miR-200} locus, though this effect is likely the indirect result of \textit{ZEB1}-mediated repression \cite{nevesRoleDNAMethylation2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct transcriptional repressor of \textit{miR-200} expression \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct transcriptional repressor of \textit{miR-200} expression \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Overexpression of \textit{ZEB2} suppressed the activity of the \textit{miR-200b-200a-429} promoter \cite{funatoTBX1TargetsMiR200ZEB22022}; \textit{ZEB2} is a direct transcriptional repressor of the \textit{miR-200} cluster \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016,brackenDoubleNegativeFeedbackLoop2008}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{p21}&\multicolumn{1}{m{\linkdspec}}{{\textit{p21} knockdown downregulates several EMT-blockign miRNAs, including \textit{miR-200a}, \textit{miR-200b}, \textit{miR-200c} and the \textit{miR-183-96-182} cluster. This inhibits EMT, migration and invasion \cite{liP21ZEB1ComplexInhibits2014}. While it is unclear if \textit{p21} is a direct repressor of the \textit{miR-200} cluster, it was shown to bind \textit{ZEB1} and inhibit its transcriptipnal effets, relieving its ability to repress the \textit{miR-183-96-182} cluster \cite{liP21ZEB1ComplexInhibits2014}. As this cluster, not explicitly accounted for in our model, further inhibits \textit{ZEB1} expression \cite{liP21ZEB1ComplexInhibits2014}, it is possible that \textit{p21} increases \textit{miR-200} levels indirectly via this cluster, directly by blocking \textit{ZEB1}-mediated repression, or both. } }\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{miR\_{}200\_{}c}=\mathbf{p21}\orop\notop\left(\mathbf{Twist}\andop\mathbf{SNAI1}\andop\left(\mathbf{ZEB1\_{}H}\orop\mathbf{ZEB2}\orop\left(\mathbf{ZEB1}\andop\notop\left(\mathbf{miR\_{}200\_{}c}\orop\mathbf{c\_{}Myb}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}miR}}&\multicolumn{2}{m{\nodetwocspec}}{{\color{OliveGreen} The \textit{miR\_200\_c} node represents miR-200c microRNA expressed in epithelial cells and central to blocking EMT transcription factors \cite{mongrooRoleMiR200Family2010}. Its levels are increaased by \textit{p21} \cite{liP21ZEB1ComplexInhibits2014}, it is directly induced by \textit{c-Myb} \cite{pieraccioliActivationMiR200CMyb2013} and repressed by \textit{ZEB1/2} \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016}. This repression is indirectly supported by \textit{SNAI1} \cite{gillSnailMicroRNA200Family2011,moesNovelNetworkIntegrating2012} and \textit{Twist} \cite{daveFunctionalCooperationSnail12011a}. Here we assume that in addition to the absence of \textit{p21} and the aid of \textit{SNAI1} and \textit{Twist}, {\color{NavyBlue} either \textit{ZEB2} or} high levels of \textit{ZEB1} (\textit{ZEB\_H} = ON) are required to silence the active \textit{miR-200c} promoter. In contrast, medium \textit{ZEB1} can maintain repression as long as \textit{miR-200c} is silenced and its inducer \textit{c-Myb} is off.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{c\allowbreak\_{}Myb}&\multicolumn{1}{m{\linkdspec}}{{The proto-oncogene \textit{c-Myb} induces the expression of the \textit{miR-200} family, unless the locus is silenced by DNA methylation \cite{pieraccioliActivationMiR200CMyb2013}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1} induction reduces the expression of \textit{miR-200} \cite{gillSnailMicroRNA200Family2011,moesNovelNetworkIntegrating2012}. } }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Twist}&\multicolumn{1}{m{\linkdspec}}{{\textit{Twist} upregulation is required to maintain high levels of \textit{ZEB1} (\textit{Twist} is a direct enhancer of \textit{ZEB1}) \cite{daveFunctionalCooperationSnail12011a}. Thus, we assume that \textit{ZEB1} and \textit{SNAI1} cannot fully repress the \textit{miR-200} cluster in the absence of \textit{Twist}. In addition, overexpression of \textit{Twist} resulted in DNA methylation of the  \textit{miR-200} locus, though this effect is likely the indirect result of \textit{ZEB1}-mediated repression \cite{nevesRoleDNAMethylation2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Epi}}}&\multicolumn{1}{m{\linkcspec}}{miR\allowbreak\_{}200\allowbreak\_{}c}&\multicolumn{1}{m{\linkdspec}}{{As the \textit{miR-200} promoter is subject of DNA methylaiton and epigenetic silencing during EMT \cite{nevesRoleDNAMethylation2010}, we assume that it is more difficult to turn off than to maintain its silenced state.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct transcriptional repressor of \textit{miR-200} expression \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{ \textit{ZEB1} is a direct transcriptional repressor of \textit{miR-200} expression \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ZEB2} is a direct transcriptional repressor of \textit{miR-200} expression \cite{burkReciprocalRepressionZEB12008,haraguchiDynamicsPlasticityEpithelial2016,brackenDoubleNegativeFeedbackLoop2008}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{p21}&\multicolumn{1}{m{\linkdspec}}{{\textit{p21} knockdown downregulates several EMT-blockign miRNAs, including \textit{miR-200a}, \textit{miR-200b}, \textit{miR-200c} and the \textit{miR-183-96-182} cluster. This inhibits EMT, migration and invasion \cite{liP21ZEB1ComplexInhibits2014}. While it is unclear if \textit{p21} is a direct repressor of the \textit{miR-200} cluster, it was shown to bind \textit{ZEB1} and inhibit its transcriptipnal effets, relieving its ability to repress the \textit{miR-183-96-182} cluster \cite{liP21ZEB1ComplexInhibits2014}. As this cluster, not explicitly accounted for in our model, further inhibits \textit{ZEB1} expression \cite{liP21ZEB1ComplexInhibits2014}, it is possible that \textit{p21} increases \textit{miR-200} levels indirectly via this cluster, directly by blocking \textit{ZEB1}-mediated repression, or both. } }\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Ecadherin\allowbreak\_{}mRNA}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Ecadherin\_{}mRNA}=\notop\left(\left(\left(\mathbf{ZEB1\_{}H}\andop\mathbf{ZEB2}\right)\orop\left(\notop\mathbf{MCRIP1}\andop\left(\mathbf{ZEB1\_{}H}\orop\mathbf{ZEB2}\right)\right)\right)\andop\mathbf{ZEB1}\andop\mathbf{SNAI1}\andop\mathbf{SNAI2}\andop\mathbf{Twist}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}mRNA}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{Ecadherin\_mRNA} represents basal \textit{E-cadherin} expression, required to make at least some adherens juncitons with neighbors. This basal expression is only blocked during full EMT by the joint action of \textit{ZEB1} \cite{shirakiharaDifferentialRegulationEpithelial2007}, \textit{SNAI1} \cite{bolosTranscriptionFactorSlug2003,parkMiR200FamilyDetermines2008,nieRolesMicroRNA34aEpithelial2019}, \textit{SNAI2} \cite{bolosTranscriptionFactorSlug2003,peinadoTranscriptionalRegulationCadherins2004,parkMiR200FamilyDetermines2008}, \textit{Twist} \cite{vesunaTwistTranscriptionalRepressor2008,nieRolesMicroRNA34aEpithelial2019} and {\color{NavyBlue} \textit{ZEB2}}, protected from repression by \textit{MCRIP1} \cite{wengMCRIP1PromotesExpression2019}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{MCRIP1}&\multicolumn{1}{m{\linkdspec}}{{\textit{MCRIP1} promotes transcription of \textit{E-cadherin} by competitively binding to \textit{ZEB1} co-repressor \textit{CtBP}, thus inhibiting \textit{ZEB1}-mediated transcriptional repression \cite{wengMCRIP1PromotesExpression2019}. During EMT, \textit{MCRIP1} is inhibited by \textit{ERK} phosphorylation \cite{wengMCRIP1PromotesExpression2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI1}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI1} is a direct transcriptional inhibitor of \textit{E-cadherin} \cite{bolosTranscriptionFactorSlug2003,parkMiR200FamilyDetermines2008,nieRolesMicroRNA34aEpithelial2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SNAI2}&\multicolumn{1}{m{\linkdspec}}{{\textit{SNAI2} is a direct transcriptional inhibitor of \textit{E-cadherin} transcription \cite{bolosTranscriptionFactorSlug2003,peinadoTranscriptionalRegulationCadherins2004,parkMiR200FamilyDetermines2008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Twist}&\multicolumn{1}{m{\linkdspec}}{{\textit{Twist} is a direct transcriptional inhibitor of \textit{E-cadherin} transcription \cite{vesunaTwistTranscriptionalRepressor2008,nieRolesMicroRNA34aEpithelial2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct transcriptional inhibitor of \textit{E-cadherin} \cite{shirakiharaDifferentialRegulationEpithelial2007}, either in partnership with its co-repressor \textit{CtBP} \cite{grooteclaesEvidenceFunctionCtBP2000}, or by binding to the SWI/SNF chromatin-remodeling protein \textit{BRG1} \cite{sanchez-tilloZEB1RepressesEcadherin2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{ZEB1} is a direct transcriptional inhibitor of \textit{E-cadherin} \cite{shirakiharaDifferentialRegulationEpithelial2007}, either in partnership with its co-repressor \textit{CtBP} \cite{grooteclaesEvidenceFunctionCtBP2000}, or by binding to the SWI/SNF chromatin-remodeling protein \textit{BRG1} \cite{sanchez-tilloZEB1RepressesEcadherin2010}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB2}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ZEB2} is a direct transcriptional inhibitor of \textit{E-cadherin} \cite{comijnTwohandedBoxBinding2001}.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Restriction\_{}SW module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{p21}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{p21}=\left(\mathbf{p21\_{}mRNA}\andop\left(\notop\mathbf{Casp3}\right)\right)\andop\left(\notop\mathbf{CyclinE}\orop\notop\mathbf{Myc}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{In this model, the \textit{p21} node corresponds to nuclear p21 in cells with relatively high basal \textit{p21} activity. \textit{p21} activity and or localization can be lowered by loss of \textit{FoxO} mediated transcription (see \textit{p21\textsubscript{mRNA}} node) and via feedback from \textit{Cyclin E/Cdk2} \cite{overtonBasalP21Controls2014} {\color{NavyBlue} aided by ongoing \textit{Myc}-mediated repression \cite{gartelMycRepressesP212001}}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TL}}}&\multicolumn{1}{m{\linkcspec}}{p21\allowbreak\_{}mRNA}&\multicolumn{1}{m{\linkdspec}}{\textit{p21} protein activity requires the presence of \textit{p21} transcription.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} Induced \textit{Myc} expression lead to decreased endogenous \textit{p21} through \textit{Myc} binding to its proximal promoter, preventing transcription. \textit{Myc} is also suspected to sequester the \textit{p21} activator \textit{Sp1} \cite{gartelMycRepressesP212001}}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{\textit{p21}  and \textit{Cyclin E/Cdk2} form a positive (double-negative) feedback loop in which \textit{Cyclin E/Cdk2} activates the \textit{SCF/Skp2} complex responsible for the degradation of \textit{Cyclin E/Cdk2}-bound, phosphorylated \textit{p21} \cite{luUbiquitylationProteasomalDegradation2010}. \textit{p21}, in turn, not only blocks \textit{Cyclin E/Cdk2} activity, but it also inhibits \textit{Cyclin D1}. Thus, \textit{p21} interferes with the mitogen signal that turns on \textit{Cyclin E} in the first place. In quiescent cells with high basal \textit{p21} levels, this positive feedback renders cell cycle entry stochastic \cite{overtonBasalP21Controls2014}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves and deactivates \textit{p21} \cite{gervaisCleavageCDKInhibitor1998}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{pRB}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{pRB}=\left(\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\mathbf{CyclinD1}\right)\right)\andop\left(\notop\mathbf{CyclinA}\right)\right)\andop\left(\mathbf{p27Kip1}\orop\left(\notop\mathbf{CyclinE}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{pRB} is active in the absence of \textit{Caspase 3}, \textit{Cyclin D1}, \textit{Cyclin A}, and \textit{Cyclin E}. In addition, \textit{pRB} maintains its activity when active \textit{p27\textsuperscript{Kip1}} counteracts the effects of \textit{Cyclin E} \cite{coqueretNewRolesP212003,harbourCdkPhosphorylationTriggers1999,NovakMammalianCellCycle2004,tayaRBKinasesRBbinding1997}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{p27Kip1}&\multicolumn{1}{m{\linkdspec}}{Active \textit{p27\textsuperscript{Kip1}} can counteract the inhibitory effects of active \textit{CyclinE/Cdk2} complexes \cite{coqueretNewRolesP212003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinD1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin D1/Cdk4,6} complexes bind and phosphorylate \textit{RB}, inhibiting its activity \cite{katoDirectBindingCyclin1993,ewenFunctionalInteractionsRetinoblastoma1993,NovakMammalianCellCycle2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin E/Cdk2} complexes bind and phosphorylate \textit{RB}, inhibiting its activity \cite{hindsRegulationRetinoblastomaProtein1992,NovakMammalianCellCycle2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A/Cdk1,2} complexes phosphorylate and deactivate \textit{RB} \cite{harbourCdkPhosphorylationTriggers1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves \textit{RB}, generating fragments that do not associate with \textit{E2F1}, rendering \textit{RB} inactive \cite{fattmanSequentialTwostepCleavage2001}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{p27Kip1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{p27Kip1}=\left(\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\mathbf{CyclinD1}\right)\right)\andop\left(\notop\left(\mathbf{Cdk1}\andop\mathbf{CyclinB}\right)\right)\right)\andop\left(\left(\left(\left(\notop\left(\left(\mathbf{CyclinA}\andop\mathbf{Necl5}\right)\andop\mathbf{CyclinE}\right)\right)\andop\left(\mathbf{FoxO3}\andop\mathbf{FoxO1}\right)\right)\orop\left(\left(\left(\notop\mathbf{CyclinA}\right)\orop\left(\notop\left(\mathbf{Necl5}\orop\mathbf{CyclinE}\right)\right)\right)\andop\left(\mathbf{FoxO3}\orop\mathbf{FoxO1}\right)\right)\right)\orop\left(\left(\notop\mathbf{CyclinA}\right)\andop\left(\notop\left(\mathbf{Necl5}\andop\mathbf{CyclinE}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Active \textit{p27}$^{Kip1}$ is cleaved by \textit{Caspase 3} and inhibited (sequestered) by \textit{Cyclin D1/Cdk4,6} \cite{coqueretNewRolesP212003} or \textit{Cyclin B/Cdk1} \cite{ishidaPhosphorylationP27Kip1Serine2002}. In addition, maintenance of \textit{p27}$^{Kip1}$ requires one or both \textit{FoxO} factors when sequestered by \textit{Cyclin E/Cdk2} (one \textit{FoxO} factor) or \textit{Cyclin A/Cdk2} (both \textit{FoxO} factors), but it cannot keep pace with the simultaneous activity of \textit{Cyclin E/Cdk2} and \textit{Cyclin A/Cdk2} \cite{faureDynamicalAnalysisGeneric2006}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO} factors are direct inducers of \textit{p27\textsuperscript{Kip1}} expression \cite{medemaAFXlikeForkheadTranscription2000}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO1}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO} factors are direct inducers of \textit{p27\textsuperscript{Kip1}} expression \cite{medemaAFXlikeForkheadTranscription2000}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Necl5}&\multicolumn{1}{m{\linkdspec}}{{\textit{Necl5} downregulates the transcription of \textit{p27Kip1} in response to growth factor stimulation \cite{kakunagaEnhancementSerumPlateletderived2004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinD1}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Cyclin D/Cdk4,6} complexes competitively bind to \textit{p27Kip1} and progressively inhibit its ability to keep \textit{Cyclin-E/Cdk2} inactive, thereby inducing cdk2 activity and cell-cycle progression \cite{coqueretNewRolesP212003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Cyclin-E/Cdk2}	phosphorylate \textit{p27Kip1} at threonine 187 (Thr187) \cite{Muller:1997ed}, which marks it for degradation by the \textit{SCF}$^\text{SKP2}$ complex at the onset of S-phase \cite{sheaffCyclinECDK2Regulator1997}. (\textit{Cyclin-E/Cdk2} complexes remain active in the presence of \textit{p27Kip1} and promote its degradation when \textit{Cyclin-A} is also active.)}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A/Cdk2} complexes bind and inactivate \textit{p27}$^{Kip1}$  by sequestration, phosphorylate it, and promote its degradation \cite{montagnoliUbiquitinationP27Regulated1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} complexes phosphorylate \textit{p27}$^{Kip1}$ \cite{montagnoliUbiquitinationP27Regulated1999}, and although they do not promote its degradation, phosphorylated \textit{p27}$^{Kip1}$ is exported from the nuclear compartment and looses its ability to inhibit \textit{Cdk} activity \cite{ishidaPhosphorylationP27Kip1Serine2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}PLoc}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} complexes phosphorylate \textit{p27}$^{Kip1}$ \cite{montagnoliUbiquitinationP27Regulated1999}, and although they do not promote its degradation, phosphorylated \textit{p27}$^{Kip1}$ is exported from the nuclear compartment and looses its ability to inhibit \textit{Cdk} activity \cite{ishidaPhosphorylationP27Kip1Serine2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves \textit{p27Kip1} \cite{eyminCaspaseinducedProteolysisCyclindependent1999}; the cleaved fragments can no longer associate with \textit{Cdk2 / Cyclin} complexes \cite{levkauCleavageP21Cip1Waf11998}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Myc}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Myc}=\left(\left(\left(\mathbf{ERK}\andop\mathbf{YAP}\right)\andop\left(\notop\mathbf{SMAD2\_{}3\_{}4}\right)\right)\orop\left(\left(\mathbf{ERK}\orop\left(\mathbf{YAP}\andop\left(\notop\mathbf{SMAD2\_{}3\_{}4}\right)\right)\right)\andop\left(\mathbf{eIF4E}\andop\left(\notop\mathbf{GSK3}\right)\right)\right)\right)\orop\left(\left(\mathbf{E2F1}\andop\left(\notop\mathbf{pRB}\right)\right)\andop\left(\left(\mathbf{eIF4E}\orop\mathbf{ERK}\right)\orop\left(\notop\mathbf{GSK3}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Myc} activity is turned on by stabilization of the protein via  \textit{ERK} phosphorylation, aided by \textit{YAP}-mediated transcription \cite{chenCMYCBCL2Mediate2018,liInhibitionYAPSuppresses2016} {in the absence of inhibitory \textit{SMADs} \cite{chenDefectiveRepressionCmyc2001}}. To take into account theincrease in translation initiated by \textit{eIF4E} and loss of degradation-promoting phosphorylation when  \textit{GSK3$\beta$} is off \cite{searsMultipleRasdependentPhosphorylation2000}, we assumed that they can compensate for the lack of \textit{ERK}, absence of \textit{YAP}, {or interference from \textit{SMADs}}. Alternatively, increased transcription by  \textit{E2F1} can also promote  \textit{Myc} accumulation in the absence of active  \textit{pRB} \cite{roussel1994dual}, provided that the protein is stabilized by  \textit{ERK},  \textit{eIF4E}, or the absence of  \textit{GSK3$\beta$}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{Ser-62 phosphorylation by \textit{ERK} increases its half life, leading to \textit{Myc} accumulation \cite{searsMultipleRasdependentPhosphorylation2000,lutterbachHierarchicalPhosphorylationNterminal1994}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{eIF4E}&\multicolumn{1}{m{\linkdspec}}{Increased translational initiation in the presence of activated \textit{eIF4E} leads to an increase in \textit{Myc} protein levels \cite{linCMycEIF4FConstitute2009}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{Thr-58 phosphorylation by \textit{GSK-3} promotes \textit{Myc} degradation \cite{searsMultipleRasdependentPhosphorylation2000,welckerFbw7TumorSuppressor2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} is a transcriptional inducer of /textit{c-Myc}  \cite{chenCMYCBCL2Mediate2018,liInhibitionYAPSuppresses2016}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF}$\beta$ stimulation induces \textit{Smad} complex formation that recognizes a \textit{TGF}$\beta$ inhibitory element in the \textit{c-Myc} promoter \cite{chenDefectiveRepressionCmyc2001}. This inhibition may be partially rescued by \textit{YAP}, as \textit{YAP/Smad7} complexes were shown to interfere with \textit{Smad}-dependent gene expterssion control \cite{ferrignoYesassociatedProteinYAP652002}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1}’s ability to induce \textit{Myc} is blocked by active (hypo- phosphorylated \textit{pRB}) \cite{batscheTranscriptionalRepressionActivation1994}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{{\textit{E2F1}} binds and activates the \textit{c-Myc} promoter \cite{Oswald:1994uy,thalmeierNuclearFactorE2F1989}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CyclinD1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CyclinD1}=\left(\left(\notop\mathbf{p15}\right)\andop\left(\notop\mathbf{CHK1}\right)\right)\andop\left(\left(\left(\notop\mathbf{p21}\right)\andop\left(\left(\left(\left(\notop\mathbf{GSK3}\right)\andop\mathbf{YAP}\right)\andop\left(\mathbf{Myc}\orop\mathbf{E2F1}\right)\right)\orop\left(\left(\left(\mathbf{CyclinD1}\andop\mathbf{YAP}\right)\andop\left(\mathbf{Myc}\orop\mathbf{E2F1}\right)\right)\orop\left(\mathbf{Myc}\andop\mathbf{E2F1}\right)\right)\right)\right)\orop\left(\left(\left(\notop\mathbf{pRB}\right)\andop\mathbf{E2F1}\right)\andop\left(\left(\left(\mathbf{Myc}\andop\mathbf{CyclinD1}\right)\orop\left(\mathbf{Myc}\andop\left(\notop\mathbf{GSK3}\right)\right)\right)\orop\left(\left(\mathbf{YAP}\andop\mathbf{CyclinD1}\right)\andop\left(\notop\mathbf{GSK3}\right)\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{Ongoing DNA synthesis keeps the \textit{CHK1} kinase active, which inhibits \textit{Cyclin D1}. {Similarly, the CDKI \textit{p15} also binds and blocks active \textit{Cdk/Cyclin D} complexes.} The precise regulatory logic of \textit{Cyclin D1} as a function of transcriptional control by \textit{Myc} and \textit{E2F1}, combined with the regulation of its protein stability / activity by \textit{GSK3$\beta$} / basal \textit{p21} is not known. Here, we assume that in the absence of \textit{p21} (once \textit{p21} levels drop due to growth factor signals and/or \textit{Cdk2} activation), \textit{Cyclin D1} can be activated by \textit{YAP} and either \textit{Myc} or \textit{E2F1} -- as long as \textit{GSK3$\beta$ is OFF}. In the presence of \textit{GSK3$\beta$}, we assume that \textit{Cyclin D1} can be induced by the combined action of both \textit{Myc} and \textit{E2F1} \cite{leungRoleMycFacilitating2008}, but sustained in an ON state by either. In the presence of basal (normal quiescent) levels of \textit{p21}, we assume that \textit{Cyclin D1} transcription requires \textit{E2F1} unencumbered by \textit{pRB}, as well as any two of the following: \textit{Myc}, already active \textit{Cyclin D1},  sustained by \textit{YAP} and not blocked by \textit{GSK3$\beta$}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{\textit{GSK-3$\beta$} phosphorylates \textit{Cyclin D1} on Thr-286, promoting its ubiquitination and degradation \cite{diehlGlycogenSynthaseKinase3beta1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} is a direct transcriptional inducer of \textit{Cyclin D} \cite{mizunoYAPInducesMalignant2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{p15}&\multicolumn{1}{m{\linkdspec}}{{\textit{p15} is a cyclin-dependent kinase inhibitor of \textit{cdk4 / cdk6}. As it displaces \textit{Cyclin D}, it blocks G1/S progression \cite{hannonP15INK4BPotentialEffector1994}. }}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{p21}&\multicolumn{1}{m{\linkdspec}}{\textit{p21\textsuperscript{Cip1}} is a Cyclin Dependent kinase inhibitor which binds to and blocks the activity of \textit{Cdk2}, \textit{Cdk3}, \textit{Cdk4} and \textit{Cdk6} kinases \cite{harperInhibitionCyclindependentKinases1995} and thus inhibits \textit{CyclinD1/Cdk4,6} \cite{xiongP21UniversalInhibitor1993}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1}’s ability to induce \textit{Cyclin D1} is blocked by active (hypo-phosphorylated) \textit{RB} protein \cite{chellappanE2FTranscriptionFactor1991}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{Extracellular growth signals activate the MAPK pathway, leading to transcriptional activation of \textit{Cyclin D1} by \textit{Myc} \cite{hitomiCyclinD1Production1999,aktasRasLinksGrowth1997}. \textit{Myc} overexpression leads to rapid \textit{Cyclin D1} induction and subsequent cell cycle entry \cite{daksis1994myc}, while its absence halves \textit{Cyclin D1} levels \cite{Mateyak:1999tm}. In addition, \textit{Myc} induces \textit{Cdk4}, aiding the assembly of active \textit{Cyclin D1 / Cdk4,6} complexes \cite{Mateyak:1999tm,matsumuraE2F1CMycCell2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{CyclinD1}&\multicolumn{1}{m{\linkdspec}}{In order to take into account both production and stability of \textit{Cyclin D1}, we assumed that the presence of active \textit{CyclinD/Cdk2,4} complexes renders  transcriptional maintenance of their levels easier.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{The \textit{Cyclin D1} promoter is bound by \textit{E2F} factors including \textit{E2F1} \cite{guoElementsHumanCyclin2011}, and \textit{E2F1} overexpression can increase \textit{Cyclin D1} (though its effects are context-dependent, as \textit{E2F1} overexpression can also lead to apoptosis) \cite{guoElementsHumanCyclin2011}. Dominant negative \textit{E2F1} overexpression results in a 2-3 fold decrease in \textit{Cyclin D} expression and \textit{Cyclin D/Cdk4,6} activity \cite{Fan:1997ij}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CHK1}&\multicolumn{1}{m{\linkdspec}}{During replication, checkpoint kinases such as \textit{CHK1} (active during normal DNA synthesis) suppress \textit{Cyclin D1} \cite{Stacey:2010is}, which has a very short half-life ($\sim$ 24 min) \cite{Diehl:1997te}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{E2F1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{E2F1}=\left(\notop\left(\left(\mathbf{CAD}\orop\mathbf{CyclinA}\right)\orop\mathbf{pRB}\right)\right)\andop\left(\left(\mathbf{YAP}\andop\left(\mathbf{E2F1}\orop\mathbf{Myc}\right)\right)\orop\left(\mathbf{E2F1}\andop\mathbf{Myc}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{In the absence of both \textit{CyclinA} and \textit{pRB}, \textit{E2F1} transcription can be induced by \textit{YAP} and \textit{Myc} or maintained by active \textit{E2F1}. \textit{CAD} deactivates \textit{E2F1} as it destroys the cell’s DNA.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{YAP} is a direct transcriptional inducer of \textit{E2F1} \cite{kimHippoSignalingIntrinsically2019}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{\textit{RB} binds to \textit{E2F/DP1} complexes and switches their DNA binding activity from activation to repression  \cite{chellappanE2FTranscriptionFactor1991,Weintraub:1992gg}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{\textit{Myc} is required for growth-factor mediated induction of \textit{E2F1} \cite{leoneMycRasCollaborate1997,Mateyak:1999tm}. It binds to and remodels the \textit{E2F1} promoter, facilitating \textit{E2F1} transcription \cite{leungRoleMycFacilitating2008}. In addition, \textit{Myc} augments protein expression of \textit{E2F1} \cite{tanakaE2F1CMycPotentiate2002}. Single-cell experiments show that \textit{Myc} is a critical modulator of the amplitude of \textit{E2F} activation \cite{Dong:2014kp}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1} binds to its own promoter and up regulates transcription (as long as \textit{Cyclin D/E} activity blocks \textit{RB}-\textit{E2F1} binding) \cite{johnsonAutoregulatoryControlE2F11994}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{The phosphorylation of the \textit{E2F1}-binding \textit{DP-1} protein by \textit{Cyclin A}, which binds directly to \textit{E2F-1} (as well as \textit{E2F-2,3}) downregulates \textit{E2F1} transcriptional activity in S phase \cite{Krek:1994uc,Xu:1994wi,helinRegulationCellProliferation1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{CAD}&\multicolumn{1}{m{\linkdspec}}{This link from Caspase-activated DNase (\textit{CAD}) to \textit{E2F1} ensures that apoptotic cells settle into an \textit{E2F1}-negative attractor regardless of their initial state. The rationale for this is that \textit{E2F1} cannot maintain its activity if DNA is fragmented.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CyclinE}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CyclinE}=\mathbf{E2F1}\andop\mathbf{Cdc6}\andop\mathbf{Pre\_{}RC}\andop\notop\left(\mathbf{pRB}\orop\left(\mathbf{p27Kip1}\andop\notop\mathbf{CyclinA}\right)\orop\mathbf{CHK1}\orop\mathbf{Casp3}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, the ON state of \textit{Cyclin E} represents active \textit{Cyclin E/Cdk2} complexes. Thus, its full activation requires transcription via \textit{E2F1} not blocked by active \textit{pRB}, binding to \textit{Cdc6} and \textit{pre-RC} complexes, and the absence of its inhibitors \textit{p27\textsuperscript{Kip1}} {\color{NavyBlue} in the absence of \textit{Cyclin A/Cdk2} binding partners}, \textit{CHK1} and \textit{Caspase 3}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin E} transcription by \textit{E2F1} requires the absence of active, un-phosphorylated \textit{RB} \cite{helinRegulationCellProliferation1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{p27Kip1}&\multicolumn{1}{m{\linkdspec}}{\textit{p27Kip1}  binds to and prevents the activation of \textit{Cyclin E/Cdk2} complexes \cite{coqueretNewRolesP212003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1} is a potent transcriptional activator of \textit{Cyclin E} \cite{ohtaniRegulationCyclinGene1995}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdc6}&\multicolumn{1}{m{\linkdspec}}{Chromatin association and full activation of \textit{Cyclin E/Cdk2} requires \textit{Cdc6} \cite{Lunn:2010vg}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Pre\allowbreak\_{}RC}&\multicolumn{1}{m{\linkdspec}}{At the G1/S transition, \textit{Cyclin E} is loaded onto chromatin by \textit{pre-RC} complexes (\textit{Cdc6} and \textit{Cdt1} binding), where it is required for \textit{MCM2} loading, origin firing and the start of DNA synthesis \cite{gengKinaseindependentFunctionCyclin2007}. In addition, activation of its partner \textit{Cdk2} by \textit{Cdc6} is contingent on this localization \cite{Lunn:2010vg}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{Cyclin A/Cdk2} complexes bind and inactivate \textit{p27}$^{Kip1}$ by sequestration, phosphorylate it, and promote its degradation \cite{montagnoliUbiquitinationP27Regulated1999}. Here we assume that full potency of \textit{p27}$^{Kip1}$ requires a lack of \textit{Cyclin A/Cdk} activity.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CHK1}&\multicolumn{1}{m{\linkdspec}}{\textit{Chk1} activation during normal S-phase progression keeps \textit{Cdk2} activity in a physiological range by binding to both \textit{Cdk2} and \textit{Cdc25A}, aiding the loss of \textit{Cyclin E/Cdk1} activity \cite{Sorensen:2003tg}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves and deactivates \textit{Cyclin E}, which is then rapidly degraded \cite{mazumderProteolyticCleavageCyclin2002}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Origin\_{}Licensing module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ORC}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ORC}=\mathbf{E2F1}\orop\left(\left(\mathbf{Pre\_{}RC}\andop\mathbf{Cdt1}\right)\andop\mathbf{Cdc6}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{ORC} proteins can bind at origins of replication when transcribed by \textit{E2F1} or as part of a fully assembled and licensed \textit{Pre-RC} complex (including active \textit{Cdc6} and \textit{Cdt1}).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{Expression of the \textit{ORC1} gene is regulated by \textit{E2F1} \cite{ohtaniExpressionHsOrc1Gene1996}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdc6}&\multicolumn{1}{m{\linkdspec}}{Availability of stable (unphosphorylated) \textit{Cdc6} in the \textit{Pre-RC} is necessary for the maintenance of licensed origins \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdt1}&\multicolumn{1}{m{\linkdspec}}{Active (unphosphorylated and not geminin-bound) \textit{Cdt1} bound to the \textit{Pre-RC} is necessary for the maintenance of licensed origins \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Pre\allowbreak\_{}RC}&\multicolumn{1}{m{\linkdspec}}{Licensed but not yet fired replication complexes (\textit{Pre-RC}s containing \textit{ORC}, \textit{Cdc6}, \textit{Cdt1} and inactive \textit{MCM}s) remain stable at sites of replication origin until fired by the activation of the \textit{MCM} helicase \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdc6}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdc6}=\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\left(\mathbf{f4N\_{}DNA}\andop\mathbf{CyclinA}\right)\right)\right)\andop\left(\left(\left(\mathbf{E2F1}\andop\mathbf{ORC}\right)\andop\left(\notop\mathbf{Plk1}\right)\right)\orop\left(\left(\left(\mathbf{Pre\_{}RC}\andop\mathbf{ORC}\right)\andop\mathbf{Cdc6}\right)\andop\mathbf{Cdt1}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{In our model the \textit{Cdc6} node represents nuclear, chromatin-bound \textit{Cdc6}. Thus, the node is only active during the assembly of pre-replication complexes, or their ongoing presence during DNA replication. \textit{Cdc6} is ON in the absence of \textit{Caspase 3} or \textit{CyclinA / Cdk2} phosphorylation of \textit{Cdc6} in all origins required for the completion of DNA replication (thus, its inhibition by \textit{Cyclin A} also requires 4N DNA). In addition, active \textit{Cdc6} requires either transcription by \textit{E2F1} and recruitment by origin-bound \textit{ORC} proteins in the absence of mitotic \textit{Plk1} or maintenance of \textit{Pre-RCs} by the presence of all of its components.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{Transcription of Cdc6 is directly induced by E2F1 \cite{yanCdc6RegulatedE2F1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{ORC}&\multicolumn{1}{m{\linkdspec}}{\textit{ORC} recruits \textit{Cdc6} to origins of replication \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Cdc6}&\multicolumn{1}{m{\linkdspec}}{Stable (unphosphorylated) \textit{Cdc6} in the \textit{Pre-RC} is necessary for the maintenance of licensed origins \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdt1}&\multicolumn{1}{m{\linkdspec}}{Active (unphosphorylated and not geminin-bound) \textit{Cdt1} bound to the \textit{Pre-RC} is necessary for the maintenance of licensed origins \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Pre\allowbreak\_{}RC}&\multicolumn{1}{m{\linkdspec}}{Licensed but not yet fired replication complexes (\textit{Pre-RC}s containing \textit{ORC}, \textit{Cdc6}, \textit{Cdt1} and inactive \textit{MCM}s) remain stable and \textit{Cdc6}-bound until fired by the activation of the \textit{MCM} helicase \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} binds, phosphorylated and strongly recruits Cdc6 to the spindle pole during metaphase, then to the central spindle in anaphase, leading to its exclusion from chromosomes until telophase, when the majority of \textit{Plk1} is degraded by \textit{APC/C\textsuperscript{Cdh1}} \cite{yimCellDivisionCycle2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{Phosphorylation of \textit{CDC6} by \textit{Cyclin A/Cdk2} during DNA replication leads to its re-localization to the cytoplasm \cite{petersenPhosphorylationMammalianCDC61999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{In our model, full deactivation of Cdc6 represents the firing of all ORCs as DNA replication is completed. Thus \textit{Cyclin A}’s inhibitory action takes full effect once the cell reaches 4N DNA content \cite{petersenPhosphorylationMammalianCDC61999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves and deactivates \textit{Cdc6} \cite{pelizonHumanReplicationProtein2002}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdt1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdt1}=\left(\left(\left(\left(\notop\mathbf{geminin}\right)\andop\mathbf{ORC}\right)\andop\mathbf{Cdc6}\right)\andop\left(\notop\left(\left(\mathbf{CyclinE}\andop\mathbf{CyclinA}\right)\andop\mathbf{Cdc25A}\right)\right)\right)\andop\left(\left(\mathbf{Pre\_{}RC}\andop\left(\mathbf{E2F1}\orop\mathbf{Myc}\right)\right)\orop\left(\mathbf{E2F1}\andop\left(\mathbf{Myc}\orop\left(\notop\mathbf{pRB}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Replication-origin bound \textit{Cdt1} requires the absence of \textit{geminin}, the presence of origin-bound \textit{ORC} and \textit{Cdc6}, and the absence of sustained \textit{Cdk2} activity responsible for the firing of all origins during DNA synthesis (modeled as simultaneous \textit{Cyclin E}, \textit{Cyclin A} and \textit{Cdc25A} activity). Bound into a licensed pre-replication complex (\textit{Pre-RC}), \textit{Cdt1} remains stable as long as it is transcribed by \textit{E2F1} \cite{yoshidaRegulationGemininCdt12004} or \textit{Myc} \cite{valovkaTranscriptionalControlDNA2013} (this guarantees that \textit{Pre-RC} complexes cannot persist indefinitely in the absence of de novo transcription). Alternatively, it can be turned on by \textit{E2F1}, aided by \textit{Myc} or the absence of \textit{RB}, and \textit{FoxO3} in cells with 4N DNA.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1}-mediated transcription of \textit{Cdt1} is blocked by hypo-phosphorylated (active) \textit{pRB} \cite{yoshidaRegulationGemininCdt12004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdt1} is a direct transcriptional target of the \textit{Myc-Max} complex \cite{valovkaTranscriptionalControlDNA2013}, ensuring its availability for \textit{Pre-RC} formation and maintenance.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdt1} is a direct transcriptional target of \textit{E2F1} \cite{yoshidaRegulationGemininCdt12004}, ensuring its availability for \textit{Pre-RC} formation and maintenance.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{Sustained \textit{Cdk2} activity during S-phase (modeled as simultaneous \textit{Cyclin E}, \textit{Cyclin A} and \textit{Cdc25A} activity) is responsible for the firing of all origins required to complete DNA synthesis; it also leads to the phosphorylation and proteasomal degradation of \textit{Cdt1} \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{ORC}&\multicolumn{1}{m{\linkdspec}}{\textit{ORC}-bound origin of replication sites are the point of pre-replication complex assembly, where \textit{Cdt1} is recruited by \textit{ORC}-bound \textit{Cdc6} \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdc6}&\multicolumn{1}{m{\linkdspec}}{\textit{ORC}-bound \textit{Cdc6} recruits \textit{Cdt1} to \textit{pre-RC} complexes \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Pre\allowbreak\_{}RC}&\multicolumn{1}{m{\linkdspec}}{Licensed but not yet fired replication complexes (\textit{Pre-RC}) remain stable until fired during DNA replication \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{geminin}&\multicolumn{1}{m{\linkdspec}}{\textit{Geminin} binds to \textit{Cdt1} at pre-replication complexes, where it blocks \textit{Cdt1} binding to DNA, sequestering it away from \textit{Pre-RC}s \cite{yanagiMouseGemininInhibits2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{Sustained \textit{Cdk2} activity leads to phosphorylation and degradation of \textit{Cdt1} \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{Sustained \textit{Cdk2} activity leads to phosphorylation and degradation of \textit{Cdt1} \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Pre\allowbreak\_{}RC}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Pre\_{}RC}=\left(\left(\mathbf{ORC}\andop\mathbf{Cdc6}\right)\andop\mathbf{Cdt1}\right)\andop\left(\notop\left(\mathbf{Replication}\andop\mathbf{f4N\_{}DNA}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Pre-RC} complexes assemble when \textit{ORC}, \textit{Cdc6}, and \textit{Cdt1} are all bound to sites of replication origin along the DNA. The node denoting their licensing turns OFF at the moment of transition from ongoing \textit{Replication} to \textit{f4N\_DNA} (it is blocked in the one time-point when both of these nodes are ON).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{ORC}&\multicolumn{1}{m{\linkdspec}}{\textit{Pre-RC} complexes assemble when \textit{ORC}, \textit{Cdc6}, and \textit{Cdt1} are all bound to sites of replication origin along the DNA \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdc6}&\multicolumn{1}{m{\linkdspec}}{\textit{Pre-RC} complexes assemble when \textit{ORC}, \textit{Cdc6}, and \textit{Cdt1} are all bound to sites of replication origin along the DNA \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdt1}&\multicolumn{1}{m{\linkdspec}}{\textit{Pre-RC} complexes assemble when \textit{ORC}, \textit{Cdc6}, and \textit{Cdt1} are all bound to sites of replication origin along the DNA, leading to the recruitment of the \textit{MCM} helicase \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Unbind}}}&\multicolumn{1}{m{\linkcspec}}{Replication}&\multicolumn{1}{m{\linkdspec}}{\textit{Pre-RC}s fire and fall apart during DNA replication \cite{depamphilisRegulatingLicensingDNA2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{In our model the \textit{Pre-RC} node turns OFF when \textit{Replication} is completed, marked by the time-point when both \textit{Replication} and \textit{f4N\_DNA} are ON.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{geminin}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{geminin}=\left(\mathbf{E2F1}\andop\left(\notop\mathbf{Cdh1}\right)\right)\andop\left(\notop\left(\mathbf{pAPC}\andop\mathbf{Cdc20}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Geminin} is present when transcribed by \textit{E2F1} and not targeted for degradation by \textit{APC/C\textsuperscript{Cdh1}} or \textit{APC/C\textsuperscript{Cdc20}} \cite{Clijsters:2013do}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{Geminin} is a direct transcriptional target of \textit{E2F1} \cite{yoshidaRegulationGemininCdt12004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ubiq}}}&\multicolumn{1}{m{\linkcspec}}{pAPC}&\multicolumn{1}{m{\linkdspec}}{\textit{Geminin} is a target of \textit{APC/C\textsuperscript{Cdc20}} at the metaphase/ anaphase transition \cite{Clijsters:2013do}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ubiq}}}&\multicolumn{1}{m{\linkcspec}}{Cdc20}&\multicolumn{1}{m{\linkdspec}}{\textit{Geminin} is a target of \textit{APC/C\textsuperscript{Cdc20}} at the metaphase/ anaphase transition \cite{Clijsters:2013do}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ubiq}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{Geminin} is a target of \textit{APC/C\textsuperscript{Cdh1}} ubiquitin ligase \cite{garcia-higueraGenomicStabilityTumour2008}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Phase\_{}SW module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CyclinA\allowbreak\_{}mRNA}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CyclinA\_{}mRNA}=\left(\notop\mathbf{CAD}\right)\andop\left(\left(\mathbf{E2F1}\andop\left(\notop\mathbf{pRB}\right)\right)\orop\mathbf{FoxM1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}mRNA}}&\multicolumn{2}{m{\nodetwocspec}}{In non-apoptotic cells (no \textit{CAD}), \textit{Cyclin A} is transcribed by \textit{E2F1} in the absence of active \textit{RB} or by \textit{FoxM1}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{Active \textit{RB} blocks \textit{E2F1}’s ability to transcribe \textit{Cyclin A} \cite{knudsenCyclinFunctionalTarget1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A} is transcriptionally activated by \textit{E2F} factors \cite{helinRegulationCellProliferation1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxM1}&\multicolumn{1}{m{\linkdspec}}{Depletion of \textit{FoxM1} results in reduced \textit{Cyclin A2} expression (it is not clear whether \textit{FoxM1} is a direct transcriptional inducer of \textit{Cyclin A}) \cite{kimForkheadBoxM12006,kalinIncreasedLevelsFoxM12006,laoukiliFoxM1RequiredExecution2005,alvarez-fernandezRecoveryDNAdamageinducedG22010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CAD}&\multicolumn{1}{m{\linkdspec}}{This link from Caspase-activated DNase (\textit{CAD}) to \textit{Cyclin A} mRNA ensures that apoptotic cells settle into a G0-like attractor regardless of their initial state. The rationale for this is that no mRNA synthesis can be maintained if DNA is fragmented (we only use these links from \textit{CAD} if needed).}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Emi1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Emi1}=\left(\left(\mathbf{E2F1}\orop\left(\notop\mathbf{pRB}\right)\right)\orop\left(\notop\mathbf{p21}\right)\right)\andop\left(\notop\left(\left(\left(\mathbf{Plk1}\andop\mathbf{CyclinB}\right)\andop\mathbf{Cdk1}\right)\andop\left(\mathbf{U\_{}Kinetochores}\orop\mathbf{A\_{}Kinetochores}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Our model allows the sustained presence of \textit{Emi1} protein when it is either actively transcribed by \textit{E2F1} \cite{tateguTranscriptionalRegulationHuman2008,hsuE2FdependentAccumulationHEmi12002} or lacks joint inhibition by \textit{pRB} \cite{hsuE2FdependentAccumulationHEmi12002} and \textit{p21} \cite{leeDNADamageTriggers2009}. Degradation of \textit{Emi1} is mediated by \textit{Plk1} and \textit{CyclinB/Cdk1} complexes; initiation of this degradation requires at least temporary co-localization of \textit{Emi1} with \textit{Plk1} at mitotic spindle poles \cite{hansenPlk1RegulatesActivation2004}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{p21}&\multicolumn{1}{m{\linkdspec}}{\textit{p21} activation during DNA damage lead to a substantial decrease of \textit{Emi1} levels, not observed in \textit{p21}-null cells \cite{leeDNADamageTriggers2009}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{Active retinoblastoma protein can block \textit{Emi1} transcription mediated by \textit{E2F1} \cite{hsuE2FdependentAccumulationHEmi12002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{Emi1} is a direct transactional target of \textit{E2F1} \cite{tateguTranscriptionalRegulationHuman2008,hsuE2FdependentAccumulationHEmi12002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} phosphorylates \textit{Emi1} at mitotic spindle poles, stimulating its \textit{$\beta$TrCP} binding and ubiquitination \cite{hansenPlk1RegulatesActivation2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} enhances the ability of \textit{Plk1} to mediate \textit{Emi1} destruction \cite{hansenPlk1RegulatesActivation2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} enhances the ability of \textit{Plk1} to mediate \textit{Emi1} destruction \cite{hansenPlk1RegulatesActivation2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{As \textit{Plk1}-mediated phosphorylation of \textit{Emi1} occurs at mitotic spindle poles, our model requires ongoing mitosis for this interaction \cite{hansenPlk1RegulatesActivation2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{A\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} phosphorylates \textit{Emi1} at mitotic spindle poles, stimulating its \textit{$\beta$TrCP} binding and ubiquitination \cite{hansenPlk1RegulatesActivation2004}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{FoxM1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{FoxM1}=\left(\left(\left(\mathbf{Myc}\orop\mathbf{YAP}\right)\andop\mathbf{CyclinE}\right)\orop\left(\left(\mathbf{CyclinA}\andop\mathbf{Cdc25A}\right)\andop\mathbf{Cdc25B}\right)\right)\orop\left(\left(\mathbf{Plk1}\andop\mathbf{CyclinB}\right)\andop\mathbf{Cdk1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}TF}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, \textit{FoxM1} activity requires increased expression by \textit{Myc} \cite{panTranscriptionFactorFoxM12018}  or \textit{YAP} \cite{mizunoYAPInducesMalignant2012} and activating phosphorylation by \textit{Cyclin E/Cdk2}. Alternatively, \textit{FoxM1} activity can be sustained by potent \textit{Cdk2 / Cdk1} activity in G2 (supported by \textit{Cdc25A} or \textit{Cdc25B}), or a serial phosphorylation by \textit{Cyclin B/Cdk1} and \textit{Plk1} during mitosis.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{YAP}&\multicolumn{1}{m{\linkdspec}}{{\textit{FoxM1} is a direct transcriptional target of \textit{YAP} \cite{mizunoYAPInducesMalignant2012}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Myc}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxM1} is a direct transcriptional target of \textit{c-Myc} \cite{panTranscriptionFactorFoxM12018}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin E/Cdk2} complexes bind and phosphorylate FoxM1, potently inducing its transcriptional activity, which starts during S-phase \cite{luscher-firzlaffRegulationTranscriptionFactor2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Cdc25A} binds to and enhances the transcriptional activity of \textit{FoxM1}, potentually by bridging FoxM1 and active cyclin-\textit{Cdk2} complexes \cite{sullivanNovelInteractionsFOXM12012}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25B}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdc25B} overexpression can increase \textit{FoxM1}-dependent transcription, likely via aiding \textit{Cdk1} activity \cite{majorForkheadBoxM1B2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} binds and phosphorylates \textit{FoxM1}, which activates \textit{FoxM1}-mediated transcription in early mitosis \cite{fuPlk1dependentPhosphorylationFoxM12008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{In addition to \textit{Cyclin E/Cdk2}, \textit{Cyclin A/Cdk2} complexes can also keep \textit{FoxM1} transcriptionally active by phosphorylating its autoinhibitory N-terminal region \cite{laoukiliActivationFoxM1G22008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxM1} binds \textit{Plk1}, and phosphorylation of two key residues at this binding domain by \textit{Cyclin B/Cdk1} primes it for \textit{Plk1} binding \cite{fuPlk1dependentPhosphorylationFoxM12008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxM1} binds \textit{Plk1}, and phosphorylation of two key residues at this binding domain by \textit{Cyclin B/Cdk1} primes it for \textit{Plk1} binding \cite{fuPlk1dependentPhosphorylationFoxM12008}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdc25A}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdc25A}=\left(\left(\left(\mathbf{FoxM1}\andop\mathbf{E2F1}\right)\andop\left(\notop\mathbf{pRB}\right)\right)\orop\left(\left(\notop\mathbf{Cdh1}\right)\andop\left(\mathbf{FoxM1}\orop\left(\mathbf{E2F1}\andop\left(\notop\mathbf{pRB}\right)\right)\right)\right)\right)\andop\left(\left(\left(\left(\notop\left(\mathbf{GSK3}\orop\mathbf{CHK1}\right)\right)\orop\mathbf{CyclinE}\right)\orop\mathbf{CyclinA}\right)\orop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Ph}}&\multicolumn{2}{m{\nodetwocspec}}{As the precise combinatorial regulation of \textit{Cdc25A} throughput the cell cycle is unknown, our model assumes that accumulation of the \textit{Cdc25A} protein requires transcriptional activation by both \textit{E2F1} in the absence of \textit{pRB}, and \textit{FoxM1} to override destruction by \textit{APC/C\textsuperscript{Cdh1}}. Alternatively, one of the two transcription factors can drive \textit{Cdc25A} accumulation in the absence of \textit{APC/C\textsuperscript{Cdh1}}. In addition, stabilization of \textit{Cdc25A} either requires the absence of \textit{GSK3$\beta$} and \textit{CHK1} (both of which promote its degradation), or stabilization by \textit{Cdk} activity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{\textit{GSK3$\beta$} phosphorylates \textit{Cdc25A}, promoting its proteolysis \cite{kangGSK3BetaTargets2008}. }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{pRB}&\multicolumn{1}{m{\linkdspec}}{Active (hypo-phosphorylated) \textit{pRB} blocks \textit{E2F1}’s ability to drive \textit{Cdc25A} transcription \cite{vigoCDC25APhosphataseTarget1999,wuE2FRbComplexesAssemble2000}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1} is a direct transcriptional inducer of \textit{Cdc25A} \cite{vigoCDC25APhosphataseTarget1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdc25A} protein levels are stabilized during S-phase by \textit{CyclinE/Cdk2}-dependent phosphorylation \cite{Hoffmann:1994wk}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxM1}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxM1} is a direct transcriptional inducer of \textit{Cdc25A} \cite{sullivanNovelInteractionsFOXM12012}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdc25A} protein levels are stabilized during S and G2 by  \textit{Cdk2}-dependent phosphorylation.  \textit{Cdk2} first partners with  \textit{Cyclin E} \cite{Hoffmann:1994wk}, then continues to stabilize \textit{Cdc25A} past the point of \textit{Cyclin E} expression by partnering with \textit{Cyclin A} \cite{mazzoliniPhosphorylationCDC25ASER2832016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{During mitosis, \textit{Cdc25A} is stabilizes by \textit{Cyclin B/Cdk1} phosphorylation, which protects it from the proteasome \cite{mailandRegulationEventsCdc25A2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{During mitosis, \textit{Cdc25A} is stabilizes by \textit{Cyclin B/Cdk1} phosphorylation, which protects it from the proteasome \cite{mailandRegulationEventsCdc25A2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{The \textit{APC/C}$^{Cdh1}$ complex degrades \textit{Cdc25A} at mitotic exit \cite{donzelliDualModeDegradation2002,Manchado:2010j}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CHK1}&\multicolumn{1}{m{\linkdspec}}{\textit{CHK1} phosphorylates \textit{Cdc25A}, promoting its proteolysis and inhibiting its interaction with \textit{Cyclin B/Cdk1} \cite{chenChk1KinaseNegatively2003}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CyclinA}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CyclinA}=\left(\mathbf{CyclinA\_{}mRNA}\andop\left(\notop\mathbf{pAPC}\right)\right)\andop\left(\left(\mathbf{Cdc25A}\andop\left(\left(\notop\mathbf{Cdh1}\right)\orop\mathbf{Emi1}\right)\right)\orop\left(\mathbf{CyclinA}\andop\left(\left(\left(\notop\mathbf{Cdh1}\right)\andop\left(\mathbf{Emi1}\orop\left(\notop\mathbf{UbcH10}\right)\right)\right)\orop\left(\mathbf{Emi1}\andop\left(\notop\mathbf{UbcH10}\right)\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Cyclin A} activity requires transcription (\textit{Cyclin A} mRNA) and the absence of degradation by phosphorylated (mitotic) \textit{pAPC}. In addition, turning ON inactive \textit{Cyclin A} requires activation of \textit{Cdk2} by \textit{Cdc25A} \cite{blombergEctopicExpressionCdc25A1999} and the absence / \textit{Emi1}-mediated inhibition of \textit{APC/C\textsuperscript{Cdh1}}. Once active, \textit{Cyclin A} maintains its activity in the absence of overpowering influences driving its degradation. Namely, \textit{Cyclin A} relies on either \textit{Emi1} or the absence of \textit{UbcH10} for its ability to keep inactive \textit{APC/C\textsuperscript{Cdh1}} in check. To overpower active \textit{APC/C\textsuperscript{Cdh1}}, \textit{Cyclin A} requires both \textit{Emi1} and no \textit{UbcH10}. The precise combinatorial regulation of \textit{Cyclin A} is not known; the above logic is consistent with \textit{Cyclin A} activity pattern during cell cycle progression.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TL}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA\allowbreak\_{}mRNA}&\multicolumn{1}{m{\linkdspec}}{Sustained availability of \textit{Cyclin A} requires ongoing translation from \textit{CyclinA} mRNA.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Emi1}&\multicolumn{1}{m{\linkdspec}}{\textit{Emi1} binding to \textit{Cdh1} is required to stabilize \textit{Cyclin A} levels at the G1/S transition, allowing \textit{Cyclin A/Cdk2} to block \textit{APC/C}$^{Cdh1}$ \cite{machidaAPCInhibitorEmi12007,reimannEmi1MitoticRegulator2001,reimannEmi1RegulatesAnaphasepromoting2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}DP}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdc25A} promotes active  \textit{Cyclin A/Cdk2} complex formation by removing inhibitory phosphorylation of \textit{Cdk2} \cite{blombergEctopicExpressionCdc25A1999,Katsuno:2009di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{We assume that once activated, \textit{Cyclin A/Cdk2,1} complexes can sustain their activity until \textit{Cyclin A} is degraded.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{UbcH10}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A}  degradation by \textit{APC/C}$^{Cdh1}$ requires \textit{UbcH10} \cite{Rape:2004em}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{pAPC}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A} is degraded by the \textit{APC/C}$^{Cdc20}$ in prometaphase (as soon as the APC/C components are phosphorylated by Cdk1) \cite{geleyAnaphasepromotingComplexCyclosomedependent2001, denelzenCyclinDestroyedPrometaphase2001}, before the full activation of the complex at SAC passage \cite{difioreHowCyclinDestruction2010}. In our model, this stage of mitotic \textit{APC/C}$^{Cdc20}$ activation is represented by \textit{Cdk1}-phosphorylated \textit{APC/C} (\textit{pAPC}).}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A} is degraded by \textit{APC/C}$^{Cdh1}$ in the presence of the \textit{UbcH10} protein \cite{Harper:2002ke,Rape:2004em,faureDynamicalAnalysisGeneric2006}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Wee1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Wee1}=\left(\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\mathbf{Replication}\orop\mathbf{CHK1}\right)\right)\andop\left(\notop\left(\mathbf{Cdk1}\andop\mathbf{CyclinB}\right)\right)\right)\andop\left(\mathbf{CHK1}\orop\left(\notop\left(\left(\mathbf{Cdk1}\andop\mathbf{CyclinA}\right)\andop\mathbf{Plk1}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Wee1} is active during \textit{Replication}, unless its activity is blocked by \textit{CyclinA/Cdk1} OR \textit{CyclinB/Cdk1}  \cite{Deibler:2010ds}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} phosphorylation at S53 promotes \textit{Wee1} degradation \cite{watanabeCyclindependentKinaseCDK2005}. This event is primed by \textit{Cdk1} phosphorylation of \textit{Wee1} at S123 \cite{watanabeCyclindependentKinaseCDK2005}. As the main partner of \textit{Cdk1} in mitosis is \textit{Cyclin B}, we assume that assistance from \textit{Plk1} to block \textit{Wee1} is more relevant when paired with \textit{Cyclin A/Cdk1} complexes.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A/Cdk1} is a strong inducer of \textit{Wee1} phosphorylation and deactivation \cite{Deibler:2010ds}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} is a strong inducer of \textit{Wee1} phosphorylation and deactivation \cite{Deibler:2010ds}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{The somatic \textit{Wee1} protein is an order of magnitude more sensitive to \textit{Cdk1} activity than \textit{Cdc25C}. Thus, both \textit{Cyclin A/Cdk1} and \textit{Cyclin B/Cdk1} strongly induce \textit{Wee1} phosphorylation and deactivation \cite{Deibler:2010ds}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Replication}&\multicolumn{1}{m{\linkdspec}}{To model the sensitivity of \textit{Wee1} activation to ongoing DNA synthesis even in the absence of damage, our model turns on \textit{Wee1} immediately upon the start of DNA replication and maintains it until both Replication and the checkpoint kinase \textit{Chk1} is OFF \cite{leePositiveRegulationWee12001}. In addition, \textit{Wee1} activity has been implicated in maintaining normal replication fork procession, linking its activity directly to ongoing replication \cite{dominguez-kellyWee1ControlsGenomic2011}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CHK1}&\multicolumn{1}{m{\linkdspec}}{During DNA replication \textit{Wee1} is activated by the checkpoint kinase \textit{CHK1} \cite{leePositiveRegulationWee12001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves and deactivates \textit{Wee1} \cite{zhouCaspasedependentActivationCyclindependent1998}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{UbcH10}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{UbcH10}=\left(\notop\mathbf{Cdh1}\right)\orop\left(\mathbf{UbcH10}\andop\left(\left(\mathbf{Cdc20}\orop\mathbf{CyclinA}\right)\orop\mathbf{CyclinB}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}UbL}}&\multicolumn{2}{m{\nodetwocspec}}{The ubiquitin-conjugating enzyme (E2) \textit{UbcH10} is active in the absence of \textit{Cdh1}. Alternatively, active \textit{UbcH10}  is maintained in the presence of \textit{Cdh1} when some of its targets are present: \textit{Cdc20} OR  \textit{CyclinA} OR \textit{CyclinB} \cite{Rape:2004em}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PBind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{The presence of \textit{APC/C}$^{Cdh1}$  substrates, including \textit{Cyclin A}, inhibit the autoubiquitination of \textit{UbcH10} but not its function, thus preserving APC activity \cite{Rape:2004em}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PBind}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{The presence of \textit{APC/C}$^{Cdh1}$  substrates, including \textit{CyclinB}, inhibit the autoubiquitination of \textit{UbcH10} but not its function, thus preserving APC activity \cite{Rape:2004em}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{UbcH10}&\multicolumn{1}{m{\linkdspec}}{Active \textit{UbcH10} cannot be autoubiquitinated in the presence of \textit{APC/C}$^{Cdh1}$  substrates and thus remains active \cite{Rape:2004em}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}PBind}}}&\multicolumn{1}{m{\linkcspec}}{Cdc20}&\multicolumn{1}{m{\linkdspec}}{The presence of \textit{APC/C}$^{Cdh1}$  substrates, including \textit{Cdc20}, inhibit the autoubiquitination of \textit{UbcH10} but not its function, thus preserving APC activity \cite{Rape:2004em}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{UbcH10} is degraded by \textit{APC/C}$^{Cdh1}$.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CyclinB}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CyclinB}=\left(\mathbf{FoxM1}\orop\left(\mathbf{FoxO3}\andop\mathbf{CyclinB}\right)\right)\andop\left(\notop\left(\mathbf{Cdh1}\orop\left(\mathbf{pAPC}\andop\mathbf{Cdc20}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Cyclin B} node is ON when the concentration of \textit{Cyclin B} proteins is high (does not represent the activity of \textit{CyclinB/Cdk1} complexes). This occurs when \textit{Cyclin B} is transcribed by \textit{FoxM1}, maintained by \textit{FoxO3} transcription, and not undergoing \textit{APC}-mediated degradation by \textit{APC/C\textsuperscript{Cdc20}} or \textit{APC/C\textsuperscript{Cdh1}}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO3} is a direct transcriptional regulator of \textit{Cyclin B}; its activation in G2 helps increase/maintain \textit{Cyclin B} levels \cite{alvarezForkheadTranscriptionFactors2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxM1}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxM1} is a direct transcriptional regulator of \textit{Cyclin B1} \cite{alvarez-fernandezRecoveryDNAdamageinducedG22010,leungOverexpressionFoxM1Stimulates2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{Here we assume that FoxO3 alone can only maintain, but not independently induce \textit{Cyclin B1} expression.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{pAPC}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B} is degraded by \textit{APC/C}$^{Cdc20}$ \cite{Harper:2002ke}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdc20}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B} is degraded by \textit{APC/C}$^{Cdc20}$ \cite{Harper:2002ke}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B} is degraded by \textit{APC/C}$^{Cdh1}$ \cite{Harper:2002ke}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdc25B}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdc25B}=\mathbf{FoxM1}\andop\mathbf{f4N\_{}DNA}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Ph}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Cdc25B} activation requires transcription by \textit{FoxM1}, centrosomal localization, and activation by \textit{Aurora A} kinase on replicated centrosomes.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxM1}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxM1} is an essential inducer or \textit{Cdc25B} \cite{wangForkheadBoxM12005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdc25B} is localized at centrosomes, where it is activated by \textit{Aurora A} kinase \cite{dutertrePhosphorylationCDC25BAuroraA2004}. As \textit{Aurora A} itself is only recruited to duplicated, centrosomes before their separation \cite{dutertreRoleAuroraACentrosome2002}, \textit{Cdc25B} activation requires duplicated centrosomes. As our model does not directly account for centrosome dynamics, we account for this by requiring the completion of S-phase (4N DNA).}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Plk1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Plk1}=\left(\left(\notop\mathbf{Cdh1}\right)\andop\left(\mathbf{FoxM1}\orop\mathbf{Plk1\_{}H}\right)\right)\andop\left(\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\orop\left(\left(\mathbf{CyclinA}\andop\left(\notop\mathbf{Wee1}\right)\right)\andop\mathbf{Cdc25A}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Plk1} activity requires the absence of \textit{APC/C\textsuperscript{Cdh1}}, transcription by \textit{FoxM1}, or high \textit{Plk1} levels transcribed earlier by both \textit{FoxM1} and \textit{FoxO3} (see \textit{Plk1\_H} below) \cite{alvarezForkheadTranscriptionFactors2001}. In addition, \textit{Plk1} activation requires phosphorylation by either \textit{CyclinB/Cdk1} during mitosis or \textit{Cyclin A/Cdk2} (aided by lack of \textit{Wee1} and \textit{Cdc25A}) at the G2/M boundary \cite{gheghianiPLK1ActivationLate2017}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxM1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} is a direct transcriptional target of \textit{FoxM1} \cite{fuPlk1dependentPhosphorylationFoxM12008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{As we do not include a separate \textit{Cdk2} node in our model, strong \textit{Cyclin A/Cdk2} activity requires ongoing dephosphorylation of \textit{Cdk2} by \textit{Cdc25A} \cite{timofeevHumanCdc25APhosphatase2009}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Wee1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A}-mediated induction of \textit{Plk1} is blocked by \textit{Wee1} kinase, which specifically inhibits \textit{Cdk2} activity \cite{gheghianiPLK1ActivationLate2017}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} activation at the G2/M boundary, before \textit{Cdk1/Cuclin B} complexes are activated, requires active \textit{Cyclin A/Cdk} \cite{gheghianiPLK1ActivationLate2017}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} is activated by \textit{Cyclin B/Cdk1} phosphorylation \cite{kotaniPKAMPFactivatedPololike1998,qianActivatedPololikeKinase1998,thomasCdk1PhosphorylatesSPAT12016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} is activated by \textit{Cyclin B/Cdk1} phosphorylation \cite{kotaniPKAMPFactivatedPololike1998,qianActivatedPololikeKinase1998,thomasCdk1PhosphorylatesSPAT12016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ubiq}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{The majority of \textit{Plk1} is degraded in anaphase by the \textit{APC/C\textsuperscript{Cdh1}} complex \cite{Lindon:2004kl}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Plk1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Our model tracks the accumulauton of high-enough levels of \textit{Plk1} to survive \textit{APC/C\textsuperscript{Cdh1}} mediated destruction into telophase via the \textit{Plk1\_H} node. Its ON state represents strong prior \textit{Plk1} activation. Thus, it sustains the \textit{Plk1} node in the absence of \textit{FoxM1}-mediated transcription until \textit{Plk1\_H} itself is lost as \textit{Plk1} levels fall.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdc25C}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdc25C}=\left(\mathbf{f4N\_{}DNA}\andop\mathbf{Plk1}\right)\andop\left(\left(\mathbf{Cdc25B}\andop\left(\notop\mathbf{CHK1}\right)\right)\orop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Ph}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, \textit{Cdc25C} is active in cells with replicated DNA (see \textit{f4N\_DNA} $\rightarrow$ \textit{Cdc25C} link). Its activation is initiated by a small, initially cytoplasmic pool of \textit{Cyclin B/Cdk1} activated by \textit{Cdc25B} (not directly represented in our model) and further increased by \textit{Cdc25B} itself, which translocates to the nucleus with the aid of \textit{Plk1}. During mitosis, \textit{Plk1} potentiates the ability of \textit{Cyclin B/Cdk1} to maintain \textit{Cdc25C} activity.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25B}&\multicolumn{1}{m{\linkdspec}}{\textit{CDC25B} starts the cascade leading to mitotic entry by activating a small centrosomal pool of \textit{Cyclin B/Cdk1}, leading to their nuclear translocation where they trigger the activation of \textit{Cdc25C} and eventually the larger nuclear \textit{Cyclin B/Cdk1} pool \cite{desouzaCentrosomalCytoplasmicCdc22000,jackmanActiveCyclinB1Cdk12003,lindqvistCdc25BCooperatesCdc25A2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{In addition, \textit{Plk1} induces nuclear transport of \textit{CDC25B}, where it contributes to the initiation of \textit{Cdk1} activity \cite{lobjoisPololikeKinaseRegulates2009}. During mitosis, \textit{Plk1} helps maintain strong \textit{Cdc25C} activation by phosphorylating it on the same site as \textit{Cyclin B/Cdk1} \cite{ouyangPhysicalAssociationPhosphorylation1999}, as indicated by the profound decrease of Cdc25C activity in Plk1-inhibited mitotic cells \cite{qianActivatedPololikeKinase1998,cogswellDominantnegativePololikeKinase2000}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} complexes are potent activators of \textit{Cdc25C},  creating positive feedback that causes switch-like mitotic entry  \cite{hoffmannPhosphorylationActivationHuman1993, Karaiskou:1998kh}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} complexes are potent activators of \textit{Cdc25C},  creating positive feedback that causes switch-like mitotic entry  \cite{hoffmannPhosphorylationActivationHuman1993, Karaiskou:1998kh}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CHK1}&\multicolumn{1}{m{\linkdspec}}{\textit{CHK1} phosphorylates \textit{Cdc25C}, leading to its nuclear exclusion, loss of access to its main target, \textit{Cdk1} \cite{lopez-gironaNuclearLocalizationCdc251999}. In addition, \textit{CHK1} blocks the ability of \textit{Cdc25B} to activate \textit{Cdc25C} at the centrosomes by phosphorylating it and blocking its \textit{Cdk1} activity \cite{schmittCHK1PhosphorylatesCDC25B2006,kramerCentrosomeassociatedChk1Prevents2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{The nature and localization of the signals responsible for the onset and maintenance of \textit{Cdc25C} activity require replicated DNA (\textit{f4N\_DNA}) \cite{lobjoisPololikeKinaseRegulates2009,lindqvistCdc25BCooperatesCdc25A2005}. Namely, \textit{Cdc25C} is initially activated by a small pool of \textit{Cyclin B/Cdk1} (below the ON-threshold of \textit{Cdk1} in our model) which starts our at the replicated centrosome. Moreover, the pool of mitotic \textit{Cdc25C} co-localized with active \textit{Chk1/Cyclin B} is found on condensed chromosomes, again requiring the presence of \textit{f4N\_DNA} \cite{lopez-gironaNuclearLocalizationCdc251999}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdk1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdk1}=\left(\mathbf{CyclinB}\andop\mathbf{Cdc25C}\right)\andop\left(\left(\notop\mathbf{CHK1}\right)\orop\left(\left(\notop\mathbf{Wee1}\right)\andop\mathbf{Cdk1}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{Full \textit{Cdk1} kinase activation requires its binding partner \textit{Cyclin B} and the \textit{Cdc25C} phosphatase, which maintains \textit{Cdk1} in an active dephosphorylated state. \textit{Cdk1} is inhibited by the checkpoint kinase \textit{CHK1}, unless it is already full active and \textit{Wee1} kinase is inhibited.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Wee1}&\multicolumn{1}{m{\linkdspec}}{\textit{Wee1} is a nuclear protein that ensures the completion of DNA replication prior to mitosis by blocking nuclear \textit{Cdk1} activation \cite{Heald:1993wq}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}DP}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25C}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdk1} is subject to inhibitory phosphorylation by \textit{Wee1} or \textit{Myt1}, and its dephosphorylation is carried out by activated \textit{Cdc25C} \cite{hoffmannPhosphorylationActivationHuman1993,jackman1996cyclins,Karaiskou:1998kh}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{Full kinase activation of \textit{Cdk1} in our model requires it to complex with \textit{Cyclin B} \cite{jackman1996cyclins}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{We assume that the presence of fully activated, nuclear \textit{Cdk1} is able to overcome the effect of active \textit{Wee1}, given that  \textit{Wee1} is very sensitive to  \textit{Cdk1}-mediated inhibitory phosphorylation \cite{Deibler:2010ds}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CHK1}&\multicolumn{1}{m{\linkdspec}}{In the absence of \textit{CHK1} kinase, a small cytosolic (centrosomal) pool of \textit{Cyclin B/Cdk1} can be activated by \textit{Cdc25B}, the nuclear translocation of which can trigger a positive feedback loop that activates the full \textit{Cdk1} pool (assuming nuclear \textit{Wee1} is also inactive). Thus, \textit{CHK1} can maintain the OFF state of inactive \textit{Cdk1} \cite{gheghianiPLK1ActivationLate2017}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{pAPC}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{pAPC}=\left(\left(\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\andop\mathbf{Plk1}\right)\orop\left(\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\andop\mathbf{pAPC}\right)\right)\orop\left(\mathbf{pAPC}\andop\mathbf{Cdc20}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{In line with evidence that \textit{Plk1} can aid full activation of \textit{APC/C}, but \textit{Cdk1} appears to be the more potent inducer, our model requires both \textit{Cyclin B/Cdk1} and \textit{Plk1} to activate \textit{APC/C} from an OFF state, but only \textit{Cdk1} activity to maintain it. In addition, ongoing phosphorylation of the functional \textit{APC/C\textsuperscript{Cdc20}} complex is no longer required.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{In addition to \textit{Cyclin B/Cdk1} phosphorylation, full activation of the \textit{APC/C}$^{Cdc20}$ complex also requires the kinase activity of \textit{Plk1} \cite{golanCyclinubiquitinLigaseActivity2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{CyclinB/Cdk1} activation triggers mitotic entry and promotes  \textit{APC/C}$^{Cdc20}$ activity via APC/C subunit  phosphorylation \cite{rudnerPhosphorylationCdc28Activates2000,qiaoAPCCCdh1Cell2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{CyclinB/Cdk1} activation triggers mitotic entry and promotes  \textit{APC/C}$^{Cdc20}$ activity via APC/C subunit  phosphorylation \cite{rudnerPhosphorylationCdc28Activates2000,qiaoAPCCCdh1Cell2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{pAPC}&\multicolumn{1}{m{\linkdspec}}{Activated \textit{APC/C\textsuperscript{Cdc20}} initiates the Metaphase / Anaphase transition by degrading \textit{Cyclin B} and securin \cite{Clijsters:2013do}. Once active, \textit{APC/C\textsuperscript{Cdc20}} no longer requires sustained \textit{CyclinB/Cdk1} or \textit{Plk1} phosphorylation.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cdc20}&\multicolumn{1}{m{\linkdspec}}{Once active, \textit{APC/C}$^{Cdc20}$ no longer requires sustained \textit{CyclinB/Cdk1} or  \textit{Plk1} phosphorylation.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdc20}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdc20}=\left(\left(\mathbf{pAPC}\andop\left(\notop\mathbf{Emi1}\right)\right)\andop\left(\notop\mathbf{Cdh1}\right)\right)\andop\left(\left(\notop\mathbf{Mad2}\right)\orop\left(\left(\notop\mathbf{CyclinA}\right)\andop\left(\notop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{In our model, \textit{APC/C\textsuperscript{Cdc20}} complex formation is represented by the joint activity of \textit{Cdc20} and phosphorylated \textit{APC/C} (\textit{pAPC}). \textit{Cdc20} is thus ON in the presence of \textit{pAPC} when both \textit{Emi1} and \textit{Cdh1} are absent (\textit{APC/C\textsuperscript{Cdh1}} is represented by the \textit{Cdh1} node, see below). In addition, \textit{Cdc20} activity requires either the absence of \textit{Mad2} at unattached kinetochores, or the absence of \textit{Cdc20} phosphorylation by \textit{Cyclin B/Cdk1} or by \textit{Cyclin A/Cdk2} complexes to potentiate the interaction between \textit{Mad2} and \textit{Cdc20}, and \textit{pAPC} is ON (present and phosphorylated) \cite{dangiolellaSpindleCheckpointRequires2003}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{Emi1}&\multicolumn{1}{m{\linkdspec}}{\textit{Emi1} binds \textit{Cdc20} and inhibits the ubiquitin ligase activity of \textit{APC/C}$^{Cdc20}$ \cite{reimannEmi1MitoticRegulator2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A/Cdk2} complexes phosphorylate \textit{Cdc20} and inactivate the \textit{APC/C}$^{Cdc20}$ complex during S and G2 \cite{heinInterphaseAPCCCdc202016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B} partners with \textit{Cdk1} to keep \textit{Cdc20} phosphorylated, increasing its interaction with \textit{Mad2} rather than \textit{APC/C} \cite{avramMitoticCheckpointProteins2014}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdk1}-phosphorylated \textit{Cdc20} interacts with \textit{Mad2} rather than \textit{APC/C}, resulting in a block on \textit{APC/C}$^{Cdc20}$ activation until completion of spindle assembly \cite{dangiolellaSpindleCheckpointRequires2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{pAPC}&\multicolumn{1}{m{\linkdspec}}{\textit{Cdc20} becomes active in early mitosis by binding to \textit{APC/C}, an event that requires \textit{Cyclin B/Cdk1}-mediated phosphorylation of several core \textit{APC/C} subunits \cite{Manchado:2010j,thorntonPreciseDestructionEmerging2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{APC/C}$^{Cdh1}$ complexes degrade \textit{Cdc20}, leading to a complete switch from \textit{APC/C}$^{Cdc20}$ to \textit{APC/C}$^{Cdh1}$  during mitotic exit \cite{Manchado:2010j, petersAnaphasePromotingComplex2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{Mad2}&\multicolumn{1}{m{\linkdspec}}{Eukaryotic cells do not separate their replicated genome until they pass the Spindle Assembly Checkpoint (SAC). Namely, all their chromosomes need to be are aligned with respect to the metaphase plane and the two copies of each chromosome need to be attached to opposite poles of the mitotic spindle \cite{qiaoAPCCCdh1Cell2010}. This physical alignment is monitored via \textit{Mad2}: kinetochores that remain unattached to microtubules catalyze the sequestration of \textit{Cdc20} and thus inhibit \textit{APC/C}$^{Cdc20}$ \cite{Reddy:2007df,Nezi:2009je}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cdh1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cdh1}=\left(\notop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)\andop\left(\notop\left(\mathbf{CyclinA}\andop\left(\mathbf{Emi1}\orop\mathbf{Cdc25A}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PC}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{APC/C\textsuperscript{Cdh1}} activity requires the absence of Cyclin Dependent kinase phosphorylation by  \textit{Cyclin B/Cdk1}, or  \textit{Cyclin A/Cdk2} aided by further inhibition of  \textit{Cdh1} by  \textit{Emi1}, or ongoing  \textit{Cdk2} activation by  \textit{Cdc25A} in the absence of  \textit{Emi1}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{Emi1}&\multicolumn{1}{m{\linkdspec}}{\textit{Emi1} blocks \textit{APC/C\textsuperscript{Cdh1}} binding to its substrates \cite{reimannEmi1RegulatesAnaphasepromoting2001}, as well as its ability to add ubiquitin chains to them \cite{wangEmi1PreferentiallyInhibits2013}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{As we do not include a separate \textit{Cdk2} node in our model, strong \textit{Cyclin A/Cdk2} activity capable of overriding \textit{Cdh1} activity even in the presence of \textit{Emi1} requires ongoing dephosphorylation of \textit{Cdk2} by \textit{Cdc25A} \cite{hoffmannPhosphorylationActivationHuman1993}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Cyclin A/Cdk1,2} complexes  phosphorylate  \textit{Cdh1} during S, G2 and early mitosis, impairing its interaction with \textit{APC/C} until late stages of mitosis when \textit{Cdk1/2} activity falls \cite{Manchado:2010j,Harper:2002ke}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} phosphorylates \textit{Cdh1} during mitosis, impairing its interaction with \textit{APC/C} \cite{Manchado:2010j,Harper:2002ke}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} phosphorylates \textit{Cdh1} during mitosis, impairing its interaction with \textit{APC/C}  \cite{Manchado:2010j,Harper:2002ke}.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Cell\_{}Cycle\_{}Process module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Replication}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Replication}=\left(\left(\notop\mathbf{CAD}\right)\andop\mathbf{Pre\_{}RC}\right)\andop\left(\left(\left(\mathbf{E2F1}\andop\mathbf{CyclinE}\right)\andop\mathbf{Cdc25A}\right)\orop\left(\left(\left(\mathbf{Replication}\andop\mathbf{CyclinA}\right)\andop\mathbf{Cdc25A}\right)\andop\left(\mathbf{E2F1}\orop\left(\notop\mathbf{f4N\_{}DNA}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Proc}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{Replication} node represents ongoing DNA synthesis. This requires a non-apoptotic cell, licensed pre-replication complexes (\textit{Pre\_RC}). The start of DNA synthesis requires \textit{E2F1}-meditated transcription of the genes that help execute it, as well as the firing of the first round of replication origins by \textit{Cyclin E/Cdk2}. Once ongoing, replication is sustained by \textit{Cyclin A/Cdk2} and \textit{Cdc25A}, aided by \textit{E2F1} and terminated by completion of a full round of synthesis (\textit{4N\_DNA}).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{In addition to \textit{E2F1} target genes directly included in our model, \textit{E2F1} transcribes an array of critical S-phase genes responsible for carrying out DNA synthesis (e.g, \textit{POLA1}, \textit{POLA2}, \textit{MCM3}, \textit{MCM5}, \textit{MCM6}, \textit{PCNA}, \textit{TOP2A}, \textit{RFC2}, \textit{TK1}) \cite{renE2FIntegratesCell2002,camEmergingRolesE2F2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CyclinE}&\multicolumn{1}{m{\linkdspec}}{DNA replication is initiated by fully active \textit{Cyclin E/Cdk2} \cite{coverleyDistinctRolesCyclins2002}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Pre\allowbreak\_{}RC}&\multicolumn{1}{m{\linkdspec}}{Ongoing DNA replication requires licensed replication origins, which fire throughout DNA synthesis \cite{Fragkos:2015cw}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Cdc25A}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Cdc25A} is required for onset as well as progression through S-phase \cite{mailandRapidDestructionHuman2000,Donzelli:2003ef}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A/Cdk1} complexes regulate the origin firing program in mammalian cells and are required for the completion of DNA replication \cite{Donzelli:2003ef,Katsuno:2009di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Replication}&\multicolumn{1}{m{\linkdspec}}{Once ongoing, DNA synthesis continues in the presence of active \textit{Cyclin A/Cdk2}, only ending when DNA content is doubled.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{Complete duplication of a cell’s DNA, represented in our model by \textit{f4N\_DNA} = ON, marks the end of active \textit{Replication}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CAD}&\multicolumn{1}{m{\linkdspec}}{Caspase-activated DNase (\textit{CAD}) destroys DNA, preventing ongoing replication.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{ATR}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{ATR}=\mathbf{Replication}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{ATR} accumulates at replication forks during unperturbed DNA synthesis \cite{Petermann:2006fj}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Replication}&\multicolumn{1}{m{\linkdspec}}{\textit{ATR} accumulates at replication forks during unperturbed DNA synthesis \cite{Petermann:2006fj}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CHK1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CHK1}=\mathbf{ATR}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{ATR} kinase activates \textit{CHK1} at replication forks, which not only blocks premature mitosis but also regulates the rate of origin firing by keeping \textit{Cdc25} protein levels from increasing above their physiological range \cite{Petermann:2006fj}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ATR}&\multicolumn{1}{m{\linkdspec}}{\textit{ATR} kinase activates \textit{CHK1} at replication forks (by phosphorylation of serines 317 and 345), which not only blocks premature mitosis but also regulates the rate of origin firing by keeping \textit{Cdc25} protein levels from increasing above their physiological range \cite{Petermann:2006fj}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{f4N\_{}DNA}=\left(\notop\mathbf{CAD}\right)\andop\left(\left(\mathbf{Replication}\andop\left(\left(\mathbf{Pre\_{}RC}\andop\mathbf{CyclinA}\right)\orop\mathbf{f4N\_{}DNA}\right)\right)\orop\left(\mathbf{f4N\_{}DNA}\andop\left(\notop\mathbf{Cytokinesis}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{4N DNA content in our model is reached via the completion of \textit{Replication} (via the firing of the last round of replication origins by \textit{Cyclin A/Cdk} complexes) and maintained in non-apoptotic cells the absence of a contractile ring driving cytokinesis.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Pre\allowbreak\_{}RC}&\multicolumn{1}{m{\linkdspec}}{\textit{Replication} can only complete DNA synthesis and produce double DNA content if the availability of licensed replication origins is not blocked \cite{Fragkos:2015cw}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CyclinA}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin A/Cdk1} complexes regulate the origin firing program in mammalian cells and are required for the completion of DNA replication \cite{coverleyDistinctRolesCyclins2002,Katsuno:2009di}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Replication}&\multicolumn{1}{m{\linkdspec}}{DNA content is doubled by the process of \textit{Replication}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{Once achieved, a cell’s 4N DNA content is sustained up to the point of cytokinesis.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Cytokinesis}&\multicolumn{1}{m{\linkdspec}}{The process of cytokinesis separates the replicated sister chromatids and resets the DNA content of each daughter cell to a diploid 2N.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{CAD}&\multicolumn{1}{m{\linkdspec}}{Caspase-activated DNase (\textit{CAD}) destroys DNA, preventing maintenance of a double DNA content.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{U\_{}Kinetochores}=\left(\left(\mathbf{f4N\_{}DNA}\andop\left(\notop\mathbf{Cdh1}\right)\right)\andop\left(\notop\mathbf{A\_{}Kinetochores}\right)\right)\andop\left(\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\orop\mathbf{U\_{}Kinetochores}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{U\_Kinetochores} node in our model is on from the moment the nuclear envelope is dissolved in prometaphase and the mitotic spindle starts to form, until all kinetochores are properly attached. In addition to the presence of unattached kinetochores, \textit{U\_Kinetochores} = ON requires attached sister chromatics, the absence of \textit{APC/C\textsuperscript{Cdh1}} activity. It is turned on my \textit{Cyclin B/Cdk1} and remains on until the spindle is complete (or it is destroyed by \textit{APC/C\textsuperscript{Cdh1}}).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{The start of mitotic spindle assembly is initiated by active \textit{Cyclin B/Cdk1} \cite{sanchezNewInsightsCyclins2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{The start of mitotic spindle assembly is initiated by active \textit{Cyclin B/Cdk1} \cite{sanchezNewInsightsCyclins2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{Premature activation of \textit{APC/C\textsuperscript{Cdh1}} destroys the incomplete spindle by triggering premature, aberrant anaphase. This occurs due to premature degradation of \textit{APC/C} targets including \textit{Securin} (responsible for keeping sister chromatids attached \cite{uhlmannSisterchromatidSeparationAnaphase1999}), \textit{Cyclin B}, \textit{Cdc20}, and Aurora kinase A (\textit{AURKA}) \cite{listovskySequestrationCDH1MAD2L22013}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{Metaphase requires replicated sister chromatids (\textit{f4N\_DNA}), held together by their kinetochores, face in opposing directions and can be attached to opposite poles of the mitotic spindle.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Once metaphase starts, the mitotic spindle remains incomplete as long as some of the kinetochores remain unattached.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{A\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{In our model, the transition from unattached to all attached kinetochores (\textit{U\_Kinetochores} $\rightarrow$ \textit{A\_Kinetochores}) marks the completion of the mitotic spindle and Spindle Assembly Checkpoint (SAC) passage.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Mad2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Mad2}=\mathbf{U\_{}Kinetochores}\andop\left(\notop\mathbf{A\_{}Kinetochores}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Our model represents the SAC via the \textit{Mad2} kinetochore-binding protein. \textit{Mad2} is active as long as the cell has at least one unattached kinetochore and it is responsible for keeping \textit{Cdc20} sequestered from \textit{APC/C}. By keeping \textit{APC} at bay until the spindle is complete, \textit{Mad2} is required for the proper timing of anaphase \cite{Nezi:2009je}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{The \textit{Mad2} SAC protein is active and potent in the presence of even a single unattached kinetochore \cite{Nezi:2009je}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{A\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{\textit{Mad2} is inhibited by SAC passage, marked by the completion of the spindle and proper attachment of all kinetochore \cite{Nezi:2009je}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{A\allowbreak\_{}Kinetochores}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{A\_{}Kinetochores}=\left(\left(\mathbf{f4N\_{}DNA}\andop\left(\notop\mathbf{Cdh1}\right)\right)\andop\left(\notop\left(\mathbf{pAPC}\andop\mathbf{Cdc20}\right)\right)\right)\andop\left(\mathbf{A\_{}Kinetochores}\orop\left(\left(\left(\left(\mathbf{U\_{}Kinetochores}\andop\mathbf{Src}\right)\andop\mathbf{Plk1}\right)\andop\mathbf{CyclinB}\right)\andop\mathbf{Cdk1}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}MSt}}&\multicolumn{2}{m{\nodetwocspec}}{The completed spindle, represented by the \textit{A\_Kinetochores} node, requires replicated and attached sister chromatids (\textit{f4N\_DNA}) and the absence of \textit{APC/C} activity. It turns on when the process of spindle assembly (\textit{U\_Kinetochores}) is completed by active \textit{Src}, active \textit{Plk1} localized to unattached kinetochores in the presence of ongoing \textit{Cyclin B/Cdk1} activity, and it remains on until anaphase (\textit{APC/C} activation).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Src}&\multicolumn{1}{m{\linkdspec}}{\textit{Src} promotes correct spindle orientation \cite{nakayamaCSrcNotFyn2012}. Moreover, absence of \textit{c-Src} leads to severely reduced astral microtubules \cite{luMolecularPathwaysRegulating2013}. Finally, \textit{Src}-mediated phosphorylation of the \textit{Eg5} motor domain is required for the formation of a bipolar spindle and correct chromosome segregation \cite{bickelSrcFamilyKinase2017}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} activity at unattached kinetochores is required for promoting their attachment \cite{petronczkiPoloRisefromMitotic2008}. In its absence, kinetochores remain unattached and cells eventually undergo mitotic catastrophe and apoptosis \cite{Schmit:2009ki}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{Ongoing \textit{Cyclin B/Cdk1} at unattached kinetochores is necessary to keep \textit{Plk1} active and allow the completion of mitosis \cite{vecchioneFez1Lzts1Absence2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{Ongoing \textit{Cyclin B/Cdk1} at unattached kinetochores is necessary to keep \textit{Plk1} active and allow the completion of mitosis \cite{vecchioneFez1Lzts1Absence2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{pAPC}&\multicolumn{1}{m{\linkdspec}}{During normal mitosis, the completed spindle is pulled apart in response to \textit{APC/C\textsuperscript{Cdc20}}-mediated degradation of \textit{Securin}, which normally blocks \textit{Separate} from severing the \textit{Cohesin} rings keeping sister chromatids attached \cite{uhlmannSisterchromatidSeparationAnaphase1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdc20}&\multicolumn{1}{m{\linkdspec}}{During normal mitosis, the completed spindle is pulled apart in response to \textit{APC/C\textsuperscript{Cdc20}}-mediated degradation of \textit{Securin}, which normally blocks \textit{Separate} from severing the \textit{Cohesin} rings keeping sister chromatids attached \cite{uhlmannSisterchromatidSeparationAnaphase1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Deg}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{APC/C\textsuperscript{Cdh1}} destroys the spindle by triggering anaphase via the degradation of \textit{APC/C} targets, including \textit{Securin} \cite{listovskySequestrationCDH1MAD2L22013}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{Completion of the mitotic spindle requires replicated and attached sister chromatids (\textit{f4N\_DNA}).}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{The mitotic spindle is assembled gradually, as the number of unattached kinetochores gradually decreased by the formation of microtubule attachments.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{A\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Once assembled, separation of the mitotic spindle requires \textit{APC/C} activity to promote the destruction of sister chromatid cohesion  \cite{uhlmannSisterchromatidSeparationAnaphase1999}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Plk1\allowbreak\_{}H}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Plk1\_{}H}=\left(\mathbf{Plk1}\andop\mathbf{FoxM1}\right)\andop\left(\left(\mathbf{Plk1\_{}H}\orop\mathbf{FoxO3}\right)\orop\mathbf{FoxO1}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}K}}&\multicolumn{2}{m{\nodetwocspec}}{The ON state of \textit{Plk1\_H} encodes the short-lived memory of a sufficiently large active \textit{Plk1} pool to temporarily survive \textit{Plk1} destruction by \textit{APC/C\textsuperscript{Cdh1}} \cite{Lindon:2004kl}, recruit \textit{Ect2} to the central spindle, and thus aid the completion of cytokinesis \cite{petronczkiPololikeKinaseTriggers2007}. Thus, \textit{Plk1\_H} requires ongoing \textit{Plk1} activation and transcription by \textit{FoxM1}, and either induction by \textit{FoxO3} or \textit{FoxO1}, or prior accumulation.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} is a direct transcriptional target of \textit{FoxO3}, but \textit{Plk1} appears to be sufficiently induced in the absence of \textit{FoxO} preteens to aid its G2/M and mitotic functions. In contrast, accumulation if a large enough \textit{Plk1} pool to briefly outlast \textit{APC/C\textsuperscript{Cdh1}} activation (modeled by the \textit{Plk1\_H} node), requires \textit{FoxO} activity in G2 \cite{alvarezForkheadTranscriptionFactors2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO1}&\multicolumn{1}{m{\linkdspec}}{In addition to \textit{FoxO3}, \textit{FoxO1} also binds the \textit{Plk1} promoter, potentially aiding its accumulation during G2 \cite{Yuan:2008fu}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxM1}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} is a direct transcriptional target of \textit{FoxM1}; loss of \textit{FoxM1} severely reduces \textit{Plk1} protein levels \cite{laoukiliFoxM1RequiredExecution2005,fuPlk1dependentPhosphorylationFoxM12008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{Active mitotic \textit{Plk1} is a prerequisite for the accumulation of the larger active \textit{Plk1} pool denoted by \textit{Plk1\_H}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Plk1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{Once accumulated, we assume that the \textit{Plk1\_H} pool of active \textit{Plk1} remains stable in the absence of \textit{FoxO}-mediated transcription. This is supported by negative feedback regulation of \textit{FoxO} proteins by \textit{Plk1} \cite{Bucur:2014ix}, indicating that ongoing high FoxO activity is likely not required for the maintenance of \textit{Plk1\_H}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Ect2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Ect2}=\left(\left(\left(\mathbf{f4N\_{}DNA}\andop\mathbf{Plk1\_{}H}\right)\andop\mathbf{Cdh1}\right)\andop\left(\notop\mathbf{U\_{}Kinetochores}\right)\right)\andop\left(\notop\mathbf{A\_{}Kinetochores}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}GEF}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Ect2} activaiton at the spindle midzone represents the step of cytokinesis in our model. Thus, \textit{Ect2} requires \textit{f4N\_DNA}, high \textit{Plk1} activity, as well as \textit{Cdh1} for the assembly of a normal spindle midzone. Finally, \textit{Ect2} cannot be recruited to the mid zone before anaphase is completed.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Cdh1}&\multicolumn{1}{m{\linkdspec}}{\textit{APC/C\textsuperscript{Cdh1}}-mediated destruction of Aurora kinase is required for the assembly of a robust spindle midzone at anaphase and for the normal timing of cytokinesis \cite{floydAPCCdh1Targets2008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{f4N\allowbreak\_{}DNA}&\multicolumn{1}{m{\linkdspec}}{Formation of a spindle midzone, where \textit{Ect2} accumulates in preparation of cytokinesis requires recently separated sister chromatids (4N DNA content).}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Formation of a spindle midzone requires the separation of sister chromatids; thus it cannot occur before anaphase.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{A\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Formation of a spindle midzone requires the separation of sister chromatids; thus it cannot occur before anaphase.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Plk1\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{Plk1} activity in telophase (\textit{Plk1\_H}) is required for the recruitment of \textit{Ect2} to the central spindle \cite{Lindon:2004kl,wolfePololikeKinaseDirects2009}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cytokinesis}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cytokinesis}=\left(\mathbf{Ect2}\andop\mathbf{FAK}\right)\andop\mathbf{Src}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Proc}}&\multicolumn{2}{m{\nodetwocspec}}{In contrast to our previous model in \cite{sizekBooleanModelGrowth2019} where \textit{Ect2} recruitment to the central spindle marked the start of cytokinesis, in \cite{gubermanBooleanModelAnchorage2020} we introduced a separate \textit{Cytokinesis} node to mark cytokinesis and the subsequent resetting of daughter cell DNA content to 2N by a separate node. In addition to \textit{Ect2} recruitment, completion of cytokinesis also requires ECM attachments able to activate \textit{FAK} and \textit{Src} kinases.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{FAK}&\multicolumn{1}{m{\linkdspec}}{{Integrin-activated \textit{FAK} and \textit{Src} control cytokinetic abscission by decelerating \textit{PLK1} degradation at aiding \textit{CEP55} in recruiting abscission process proteins to the midbody \cite{mitraIntegrinregulatedFAKSrcSignaling2006,kasaharaSrcSignalingRegulates2007}.} }\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}ComplProc}}}&\multicolumn{1}{m{\linkcspec}}{Src}&\multicolumn{1}{m{\linkdspec}}{{Integrin-activated \textit{FAK} and \textit{Src} control cytokinetic abscission by decelerating \textit{PLK1} degradation at aiding \textit{CEP55} in recruiting abscission process proteins to the midbody \cite{mitraIntegrinregulatedFAKSrcSignaling2006,kasaharaSrcSignalingRegulates2007}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{Ect2}&\multicolumn{1}{m{\linkdspec}}{{At the start of cytokinesis, the \textit{Ect2} \textit{RhoGEF} is recruited to the central spindle \cite{petronczkiPololikeKinaseTriggers2007}. \textit{Ect2} aids the accumulation of GTP-bound \textit{RhoA} \cite{burkardChemicalGeneticsReveals2007,petronczkiPololikeKinaseTriggers2007} and the formation of the contractile ring.}}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{TRAIL module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Trail}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Trail}=\mathbf{Trail}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Env}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{Trail} node represents environmental availability of the \textit{Trail} protein outside the cell. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Env}}}&\multicolumn{1}{m{\linkcspec}}{Trail}&\multicolumn{1}{m{\linkdspec}}{The \textit{Trail} input node remains on/off if set ON/OFF in the absence of \textit{in silico} perturbation.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{Apoptotic\_{}SW module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{DR4\allowbreak\_{}5}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{DR4\_{}5}=\mathbf{Trail}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Rec}}&\multicolumn{2}{m{\nodetwocspec}}{The \textit{DR4} and  \textit{DR5} death receptors, represented by the  \textit{DR4\_5} node, are activated by extracellular  \textit{Trail} \cite{Guicciardi:2009km}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ligand}}}&\multicolumn{1}{m{\linkcspec}}{Trail}&\multicolumn{1}{m{\linkdspec}}{\textit{DR4} and \textit{DR5} death receptors are activated by extracellular  \textit{Trail} \cite{Guicciardi:2009km}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Casp8}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Casp8}=\mathbf{DR4\_{}5}\orop\mathbf{Casp3}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Pro-Caspase 8} may be cleaved independently by \textit{DISC} (not directly reprsented; an adaptor protein for \textit{DR4\_5}) or \textit{Caspase 3}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{DR4\allowbreak\_{}5}&\multicolumn{1}{m{\linkdspec}}{\textit{Trail}-bound (active) \textit{DR4} and \textit{DR5} receptors trigger the assembly of the pro-apoptotic death-inducing signaling complex (\textit{DISC}), which binds a cluster of \textit{pro-Caspase 8} proteins and initiates their cleavage into active \textit{Caspase 8} \cite{pennarunPlayingDISCTurning2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} indirectly activates \textit{Caspase 8} by cleaving \textit{Caspase 6} \cite{sleeOrderingCytochromeCinitiated1999}, which, in turn, cleaves \textit{Caspase 8} \cite{cowlingCaspase6DirectActivator2002}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Casp2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Casp2}=\mathbf{Casp3}\orop\left(\left(\mathbf{U\_{}Kinetochores}\andop\mathbf{Mad2}\right)\andop\left(\notop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Pro-caspase 2} is cleaved and activated by \textit{Caspase 3}, or by failed cytokinesis marked by the presence of unattached kinetochores, an active SAC, and the absence of active \textit{Cyclin B/Cdk1} complexes to phosphorylate and inhibit \textit{Caspase 2}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B1/Cdk1} phosphorylate \textit{caspase-2} at Ser 340, preventing its activation \cite{Andersen:2009gm}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B1/Cdk1} phosphorylate \textit{caspase-2} at Ser 340, preventing its activation \cite{Andersen:2009gm}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Mad2}&\multicolumn{1}{m{\linkdspec}}{A functional spindle assembly checkpoint is required for mitotic cell death upon prolonged mitotic arrest \cite{vogelMitoticSpindleCheckpoint2005} or spindle damage \cite{masudaAssociationMitoticSpindle2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Although the precise molecular mechanism by which \textit{Caspase 2} is activated during prolonged or stalled mitosis is unclear, its activation platform, the \textit{PIDDosome}, has been localized to unattached kinetochores \cite{shahMitosissensingCaspaseActivation2016}. Even though a checkpoint protein keeps the \textit{PIDDosome} unresponsive to DNA damage signals, the loss of protective \textit{Cyclin B/Cdk1} phosphorylation only leads to \textit{Caspase 2} activation in the presence of a partially assembled mitotic spindle, and requires active SAC.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 2} is a target of  \textit{Caspase 3}, as its inhibition severely limits  \textit{Caspase 2} cleavage during apoptosis \cite{liActivationCaspase2Apoptosis1997,sleeExecutionerCaspase3Perform2001}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{MCL\allowbreak\_{}1}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{MCL\_{}1}=\left(\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\notop\mathbf{Casp2}\right)\right)\andop\left(\left(\notop\mathbf{GSK3}\right)\orop\left(\mathbf{AKT\_{}B}\andop\left(\mathbf{ERK}\orop\left(\notop\mathbf{E2F1}\right)\right)\right)\right)\right)\andop\left(\notop\left(\left(\mathbf{Cdk1}\andop\mathbf{CyclinB}\right)\andop\mathbf{U\_{}Kinetochores}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Caspase 3} or \textit{2}-mediated destruction of \textit{MCL-1} must be absent for \textit{MCL-1} to be ON. Avoiding degradation via textit{GSK3} requires the \textit{GSK3}-weakenjng presence of basal \textit{AKT} activity (\textit{AKT\_B})  \cite{Maurer:2006bo} and either \textit{ERK}-mediated stabilization, or the absence of its repressor \textit{E2F1}. Finally, during mitotic arrest (\textit{U\_Kinetochores}), \textit{MCL-1} is deactivated by \textit{Cyclin B/Cdk1} phosphorylation, which shields it from the \textit{PPA2}-mediated dephosphorylation of its degradation-targeting sites \cite{wertzSensitivityAntitubulinChemotherapeutics2011}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{\textit{ERK} phosphorylates \textit{MCL-1}, promoting its interaction with \textit{Pin1}, which stabilizes it \cite{dingDownregulationMyeloidCell2008,townsendExpressionAntiapoptoticMCL11998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{In order to account for the loss of \textit{MCL-1} in the complete absence of growth factors versus its presence in low growth factor environments, we required basal \textit{AKT} to modulate the strength of \textit{GSK3} inhibition \cite{Maurer:2006bo}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} is phosphorylated by \textit{GSK3}, leading to ubiquitinylation and degradation of Phosphorylation \cite{Maurer:2006bo}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{E2F1}&\multicolumn{1}{m{\linkdspec}}{\textit{E2F1} is a direct transcriptional repressor of \textit{MCL-1} \cite{Croxton:2002dl}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{In cells arrested in mitosis, phosphorylation by \textit{Cyclin B/Cdk1} on T92 initiates \textit{MCL-1} degradation \cite{harleyPhosphorylationMcl1CDK1cyclin2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{Phosphorylation by \textit{Cyclin B/Cdk1} in cells arrested in mitosis initiates \textit{MCL-1} degradation \cite{harleyPhosphorylationMcl1CDK1cyclin2010}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{During prolonged mitotic arrest (\textit{U\_kinetochores}), \textit{MCL-1} levels drop steadily due to phosphorylation by \textit{JNK}, \textit{p38} and/or \textit{CKII} and its subsequent degradation by the E3 ubiquitin ligase \textit{SCF} (\textit{FBW7}) \cite{wertzSensitivityAntitubulinChemotherapeutics2011}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp2}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 2} activation destabilizes the \textit{MCL-1} protein \cite{linGlycogenSynthaseKinase3v2016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves and deactivated \textit{MCL-1} \cite{sungProteasomeInhibitorMG1322012}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BCLXL}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BCLXL}=\left(\left(\notop\mathbf{Casp3}\right)\andop\left(\left(\mathbf{BCL2}\andop\left(\notop\mathbf{SMAD2\_{}3\_{}4}\right)\right)\andop\left(\notop\mathbf{BAD}\right)\right)\right)\andop\left(\left(\left(\notop\mathbf{U\_{}Kinetochores}\right)\orop\left(\mathbf{Plk1}\andop\left(\left(\notop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)\orop\left(\mathbf{BCL2}\andop\mathbf{MCL\_{}1}\right)\right)\right)\right)\orop\left(\left(\mathbf{BCL2}\andop\mathbf{MCL\_{}1}\right)\andop\left(\notop\left(\mathbf{CyclinB}\andop\mathbf{Cdk1}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Bcl-x\textsubscript{L}} activity requires the absence of \textit{Caspase 3}. In addition, \textit{BAD} can block \textit{Bcl-x\textsubscript{L}}, as it preferentially binds to it rather than \textit{BCL2} (meaning in the absence of the latter \textit{Bcl-x\textsubscript{L}} is more likely to be sequestered by basal levels of \textit{BAD}) \cite{yangBadHeterodimericPartner1995}. {Similarly, \textit{TFG}$\beta$ can also repress \textit{Bcl-x\textsubscript{L}} \cite{spenderTGFbetaInducesApoptosis2009}.} Lastly, mitotic \textit{Bcl-x\textsubscript{L}} can be inhibited by \textit{Cdk1} activity if either \textit{BCL2}, \textit{MCL-1}, or \textit{Plk1} are OFF. In the absence of \textit{Plk1}, loss of either \textit{BCL2} or \textit{MCL-1} can result in \textit{Bcl-x\textsubscript{L}} inhibition (even without \textit{Cdk1} phosphorylation), as we assume its targets are no longer competitively bound by its family members. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{\textit{BCL-X\textsubscript{L}} is repressed by \textit{TFG}$\beta$ signaling, an event that mediates \textit{TFG}$\beta$-induced apoptosis \cite{spenderTGFbetaInducesApoptosis2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{In addition to other effects of prolonged mitotic arrest on \textit{BCL-2} proteins, \textit{Plk1} inhibition synergistically enhances the inhibitory phosphorylation of \textit{BCL-2} and \textit{BCL-xL}, as well as downregulation of \textit{MCL-1} \cite{weissSynergisticInductionApoptosis2016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{During normal mitosis, \textit{Cyclin B/Cdk1} only transiently phosphorylates part of the \textit{BCL-xL} pool. Prolonged mitosis, however, results in high levels of \textit{BCL-xL} (and \textit{Bcl-2}) phosphorylation, priming the system for \textit{Caspase 2}-mediated apoptosis \cite{terranoCyclindependentKinase1mediated2010,zhouCDK1SwitchesMitotic2014}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{During normal mitosis, \textit{Cyclin B/Cdk1} only transiently phosphorylates part of the \textit{BCL-xL} pool. Prolonged mitosis, however, results in high levels of \textit{BCL-xL} (and \textit{Bcl-2}) phosphorylation, priming the system for \textit{Caspase 2}-mediated apoptosis \cite{terranoCyclindependentKinase1mediated2010,zhouCDK1SwitchesMitotic2014}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Prolonged mitosis is required for the accumulation of \textit{BCL-xL} phosphorylation, weakening its interaction with \textit{Bax} \cite{bahBclxLControlsSwitch2014}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} competes with \textit{BCL-xL} for \textit{BAK} binding; the presence of \textit{MCL-1} can keep part of the \textit{BCL-xL} pool active \cite{gelinasBH3onlyProteinsControl2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{BCL2}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} competes with \textit{BCL-xL} for \textit{BAD} binding. As \textit{BCL-xL} is a stronger binding partner of \textit{BAD} \cite{yangBadHeterodimericPartner1995}, here we assume that loss of \textit{BCL2} or \textit{BAD} can both result in low \textit{BCL-xL} activity.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BAD}&\multicolumn{1}{m{\linkdspec}}{\textit{Bad} can bind \textit{BCL-xL} and displace it from \textit{BAX}, thus deactivating it \cite{yangBadHeterodimericPartner1995}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} is cleaved and deactivated by \textit{Caspase 3} \cite{kirschCaspase3dependentCleavageBcl21999}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BCL2}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BCL2}=\notop\left(\mathbf{Casp3}\orop\mathbf{BAD}\orop\mathbf{BIM}\orop\mathbf{BIK}\right)\andop\left(\left(\notop\mathbf{U\_{}Kinetochores}\orop\left(\mathbf{MCL\_{}1}\andop\mathbf{BCLXL}\right)\right)\orop\left(\mathbf{Plk1}\andop\left(\mathbf{BCLXL}\orop\mathbf{MCL\_{}1}\orop\notop\left(\mathbf{Cdk1}\andop\mathbf{CyclinB}\right)\right)\right)\orop\mathbf{GLI}\orop\mathbf{GLI\_{}H}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{While the precise combinatorial logic governing \textit{BCL2} activity is not clear from literature, we modeled \textit{BCL2} as ON in the absence of \textit{Caspase 3}, \textit{BAD}, \textit{BIM} or \textit{BIK}. This choice makes \textit{BCL2} the most sensitive of the three family members to activation of its three inhibitors. In addition, mitotic \textit{BCL2} is blocked by \textit{Cdk1} if both \textit{BCL-xL} and \textit{MCL-1} are OFF. In the absence of \textit{Plk1}, loss of either \textit{BCL2} or \textit{MCL-1} can result in \textit{BCL-2} inhibition (even without \textit{Cdk1} phosphorylation), as we assume its targets are no longer competitively bound by its family members.  {\color{NavyBlue} This effect can be counteracted by \textit{GLI2}-mediated \textit{BCL2} induction \cite{reglActivationBCL2Promoter2004}.}}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of \textit{BCL2} \cite{reglActivationBCL2Promoter2004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{GLI\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{GLI2} is a direct transcriptional inducer of \textit{BCL2} \cite{reglActivationBCL2Promoter2004}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Plk1}&\multicolumn{1}{m{\linkdspec}}{In addition to other effects of prolonged mitotic arrest on \textit{BCL2} proteins, \textit{Plk1} inhibition synergistically enhances the inhibitory phosphorylation of \textit{BCL2} and \textit{BCL-xL}, as well as downregulation of \textit{MCL-1} \cite{weissSynergisticInductionApoptosis2016}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{CyclinB}&\multicolumn{1}{m{\linkdspec}}{\textit{Cyclin B/Cdk1} phosphorylates \textit{BCL2} (and \textit{BCL-xL}) during mitosis \cite{terranoCyclindependentKinase1mediated2010,zhouCDK1SwitchesMitotic2014,kirschCaspase3dependentCleavageBcl21999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{Cdk1}&\multicolumn{1}{m{\linkdspec}}{Prolonged mitosis results in high levels of \textit{BCL-xL} and \textit{BCL2} phosphorylation, priming the system for \textit{Caspase 2}-mediated apoptosis \cite{terranoCyclindependentKinase1mediated2010,zhouCDK1SwitchesMitotic2014,kirschCaspase3dependentCleavageBcl21999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{U\allowbreak\_{}Kinetochores}&\multicolumn{1}{m{\linkdspec}}{Prolonged mitosis is required for the accumulation of \textit{BCL2} phosphorylation \cite{terranoCyclindependentKinase1mediated2010,zhouCDK1SwitchesMitotic2014,kirschCaspase3dependentCleavageBcl21999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} competes with \textit{BCL-xL} for binding most of their apoptotic partners, including \textit{BIK}, \textit{BIM}, \textit{BID}, \textit{BAX} and \textit{BAK}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{BCLXL}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} competes with \textit{BCL-xL} for binding most of their apoptotic partners, including \textit{BIK}, \textit{BIM}, \textit{BID}, \textit{BAX} and \textit{BAK}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BAD}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} competes with \textit{BCL-xL} for \textit{BAD} binding. \textit{BAD} displaces \textit{BCL2} from its inhibitory binding of \textit{Bax/Bak}. Although \textit{BCL-xL} is a stronger binding partner, we assume that \textit{BAD} alone cannot fully block \textit{BCL-xL} in the presence of \textit{BCL2} \cite{yangBadHeterodimericPartner1995}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BIK}&\multicolumn{1}{m{\linkdspec}}{\textit{BIK} binds \textit{BCL2} and they mutually inhibit each other’s activity \cite{elangovanFunctionalDissectionProapoptotic1997}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BIM}&\multicolumn{1}{m{\linkdspec}}{\textit{BIM} binds \textit{BCL2} and they mutually inhibit each other’s ability to activate further targets \cite{oconnorBimNovelMember1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} is cleaved and deactivated by \textit{Caspase 3} \cite{kirschCaspase3dependentCleavageBcl21999}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BAD}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BAD}=\left(\mathbf{Casp3}\orop\left(\notop\left(\left(\left(\mathbf{AKT\_{}H}\orop\mathbf{AKT\_{}B}\right)\orop\mathbf{ERK}\right)\orop\mathbf{S6K}\right)\right)\right)\orop\left(\mathbf{Casp8}\andop\left(\left(\notop\left(\left(\mathbf{AKT\_{}B}\andop\mathbf{ERK}\right)\andop\mathbf{S6K}\right)\right)\andop\left(\notop\left(\mathbf{AKT\_{}H}\andop\left(\mathbf{AKT\_{}B}\orop\mathbf{ERK}\right)\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{BAD} in our model is ON when cleaved by \textit{Caspase 3}, or in the complete absence of survival signals (\textit{AKT}, \textit{ERK} or \textit{S6K}). Alternatively, \textit{BAD} can be cleaved and activated by \textit{Caspase 8} in the absence of strong survival signaling. We modeled this inhibitory survival signal as either the combined activity of \textit{ERK}, \textit{S6K} and (at least) basal \textit{AKT}, or high \textit{AKT} in the joint presence of ERK and basal \textit{AKT} (indicating that \textit{AKT\_H} will not drop by the next time-step).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{\textit{ERK} phosphorylates \textit{BAD} at Ser-112, inducing its sequestration away from the mitochondrial membrane where its BCL-2 family targets are located (e.g., \textit{BCL-2}, \textit{BCL-xL})\cite{fangRegulationBADPhosphorylation1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}B}&\multicolumn{1}{m{\linkdspec}}{\textit{Akt} phosphorylates \textit{BAD} at Ser-136, inducing its sequestration away from the mitochondrial membrane where its BCL-2 family targets are located (e.g., \textit{BCL2}, \textit{BCL-xL}) \cite{Datta:1997tf}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{Akt} phosphorylates \textit{BAD} at Ser-136, inducing its sequestration away from the mitochondrial membrane where its BCL-2 family targets are located (e.g., \textit{BCL2}, \textit{BCL-xL}) \cite{Datta:1997tf}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}P}}}&\multicolumn{1}{m{\linkcspec}}{S6K}&\multicolumn{1}{m{\linkdspec}}{\textit{S6K1} phosphorylates \textit{BAD} at Ser-155, directly blocking its binding to \textit{BCL-xL} \cite{tanBADSer155Phosphorylation2000}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp8}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 8} is also able to cleave \textit{BAD}, generating a more potently apoptotic fragment \cite{condorelliCaspaseCleavageEnhances2001}. In addition, \textit{TRAIL}-mediated apoptosis results in \textit{BAD} cleavage by a Caspase upstream of MOMP, creating a potent apoptotic inducer before full \textit{Caspase 3} activation \cite{taghiyevOverexpressionBADPotentiates2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} cleaves \textit{BAD}, generating a more potently apoptotic fragment \cite{condorelliCaspaseCleavageEnhances2001}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BIK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BIK}=\left(\notop\left(\left(\mathbf{MCL\_{}1}\orop\mathbf{BCLXL}\right)\orop\mathbf{BCL2}\right)\right)\andop\left(\mathbf{SMAD2\_{}3\_{}4}\orop\left(\notop\mathbf{AKT\_{}H}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{BIK} is free to activate its target, \textit{BAX}, only when it is not sequestered by any of the three \textit{BCL-2} family proteins \cite{gelinasBH3onlyProteinsControl2005} {and it is either induced by \textit{SMADs} \cite{spenderTGFbetaInducesApoptosis2009} or protected by no more than basal \textit{AKT} activity, which lowers the level of both \textit{BCL2} and \textit{BCL-x\textsubscript{L}} \cite{jacquinGAPDHBindsActive2013}}. }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{{\textit{AKT\_H} elevates the level of both \textit{BCL2} \cite{pugazhenthiAktProteinKinase2000} and \textit{BCL-x\textsubscript{L}} \cite{jacquinGAPDHBindsActive2013}, thus making it easier for these inhibitors to sequester \textit{BIK}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{SMAD2\allowbreak\_{}3\allowbreak\_{}4}&\multicolumn{1}{m{\linkdspec}}{{ \textit{Smad2/3/4} are direct transciprional inducers of \textit{BIK} \cite{spenderTGFbetaInducesApoptosis2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} binds \textit{BIK}; they mutually inhibit each other \cite{gillissenMcl1DeterminesBax2007}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCLXL}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL-xL} binds \textit{BIK}; they mutually inhibit each other \cite{boydBikNovelDeathinducing1995}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCL2}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} binds \textit{BIK}; they mutually inhibit each other \cite{elangovanFunctionalDissectionProapoptotic1997}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BIM}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BIM}=\mathbf{FoxO3}\andop\left(\left(\mathbf{GSK3}\andop\notop\left(\mathbf{ERK}\orop\mathbf{MCL\_{}1}\orop\mathbf{ZEB1}\orop\mathbf{BCLXL}\orop\mathbf{BCL2}\right)\right)\orop\left(\left(\mathbf{Runx1}\andop\notop\mathbf{ERK}\andop\notop\mathbf{NfkB}\right)\andop\left(\notop\left(\left(\mathbf{MCL\_{}1}\andop\mathbf{BCLXL}\right)\andop\mathbf{BCL2}\right)\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{BIM}’s pro-apoptotic activity requires expression driven by \textit{FoxO3} and aided by \textit{GSK3}, as well as the absence of \textit{ERK}, any of the three inhibitory \textit{BCL2} family proteins, {\color{NavyBlue} or \textit{ZEB1}}. {Alternatively, \textit{Runx1} can induce BIM to apoptotic levels in the absence of \textit{ERK} and \textit{NF-$\kappa$B}, assuming the three inhibitory \textit{BCL2} family proteins are not all fully active.} }
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{ERK}&\multicolumn{1}{m{\linkdspec}}{The \textit{MEK/ERK} pathway represses \textit{BIM} protein levels, likely via transcriptional repression \cite{hughesMEKERKPathwayNegatively2011}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{FoxO3}&\multicolumn{1}{m{\linkdspec}}{\textit{FoxO3} is a transcriptional activator of \textit{BIM} \cite{dijkersExpressionProapoptoticBcl22000}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{GSK3}&\multicolumn{1}{m{\linkdspec}}{\textit{GSK3} kinase is likely required for the  \textit{AP1}-dependent expression of  \textit{BIM} \cite{hongistoLithiumBlocksCJun2003}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{NfkB}&\multicolumn{1}{m{\linkdspec}}{{\textit{NF-$\kappa$B} is a transcriptional repressor of \textit{BIM} \cite{potluriTranscriptionalRepressionBim2013,wangConstitutiveProductionNFkappaB22008}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{Runx1}&\multicolumn{1}{m{\linkdspec}}{{\textit{TGF-$\beta$} stimulation leads to increased protein levels of pro-apoptotic \textit{BIM} \cite{wienerOncogenicMutationsIntestinal2014} via \textit{Runx1} induction and subsequent \textit{Foxo3/Runx1} driven transcription \cite{rameshTransformingGrowthFactor2009}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}TR}}}&\multicolumn{1}{m{\linkcspec}}{ZEB1}&\multicolumn{1}{m{\linkdspec}}{{\color{NavyBlue} \textit{ZEB1} is a direct transcriptional repressor of the \textit{BIM} promoter \cite{inoue-yamauchiEMTinducingTranscriptionFactor2020}.}}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} binds \textit{BIM} and inhibits its apoptotic activity \cite{gomez-bougieImbalanceBimMcl12004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCLXL}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL-xL} binds \textit{BIM} and inhibits its apoptotic activity \cite{oconnorBimNovelMember1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCL2}&\multicolumn{1}{m{\linkdspec}}{\textit{BLC2} binds \textit{BIM} and inhibits its apoptotic activity \cite{oconnorBimNovelMember1998}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BID}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BID}=\mathbf{Casp8}\orop\left(\mathbf{Casp2}\andop\left(\notop\left(\left(\mathbf{BCL2}\orop\mathbf{BCLXL}\right)\orop\mathbf{MCL\_{}1}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{BID} is truncated in response to \textit{Caspase 8} activation. In addition, \textit{Caspase 2} can also promote \textit{BID} activation once all three pro-apoptotic \textit{BCL2} family proteins are blocked.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp8}&\multicolumn{1}{m{\linkdspec}}{In response to \textit{TRAIL} (or \textit{FAS} ligand), the initiator \textit{Caspase 8} cleaves \textit{BID} to its active truncated form \cite{yamadaTRAILCausesCleavage1999,huangCleavageCaspaseMitochondrial2016,Li:1998vb}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp2}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 2} cleaves \textit{BID} to its active truncated form \cite{uptonCaspase2CleavageBID2008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{All three anti-apoptotic BCL2 proteins (\textit{BCL2}, \textit{BCL-xL} and \textit{MCL-1}) sequesters \textit{BID} into stable complexes, preventing them from activating \textit{BAX} or \textit{BAK} \cite{kimHierarchicalRegulationMitochondriondependent2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCLXL}&\multicolumn{1}{m{\linkdspec}}{All three anti-apoptotic BCL2 proteins (\textit{BCL2}, \textit{BCL-xL} and \textit{MCL-1}) sequesters \textit{BID} into stable complexes, preventing them from activating \textit{BAX} or \textit{BAK} \cite{kimHierarchicalRegulationMitochondriondependent2006}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCL2}&\multicolumn{1}{m{\linkdspec}}{All three anti-apoptotic BCL2 proteins (\textit{BCL2}, \textit{BCL-xL} and \textit{MCL-1}) sequesters \textit{BID} into stable complexes, preventing them from activating \textit{BAX} or \textit{BAK} \cite{kimHierarchicalRegulationMitochondriondependent2006}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BAK}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BAK}=\left(\mathbf{BID}\andop\left(\left(\mathbf{BIM}\orop\mathbf{BIK}\right)\orop\left(\notop\left(\left(\mathbf{BCL2}\andop\mathbf{BCLXL}\right)\andop\mathbf{MCL\_{}1}\right)\right)\right)\right)\orop\left(\left(\mathbf{BIM}\orop\mathbf{BIK}\right)\andop\left(\notop\left(\mathbf{BCLXL}\orop\mathbf{MCL\_{}1}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Given that \textit{BAK} is preferentially activated by \textit{BID} compared to \textit{BIM} \cite{sarosiekBIDPreferentiallyActivates2013} and that it is less responsive to sequestration by \textit{BCL2} than the other two anti-apoptotic \textit{BCL2} family proteins \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005}, \textit{BAK} in our model turns on when stimulated by \textit{BID} if one or more \textit{BCL2} family proteins are absent, or if \textit{BIM} or \textit{BIK} are also present. In contrast, \textit{BIM} or \textit{BIK} only activate \textit{BAK} if \textit{BCL-xL} and \textit{MCL-1} are absent (\textit{BCL-2} alone cannot block them).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} binds \textit{BAK} and prevent its oligomerization in the mitochondrial membrane \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCLXL}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL-xL} binds \textit{BAK} and prevent its oligomerization in the mitochondrial membrane \cite{leePhysiologicalRestraintBak2016,zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCL2}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} can also bind \textit{BAK} to prevent its oligomerization, but it does so less potently than the other two BCL-2 family members \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005,daiContextdependentBcl2Bak2009}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{BIK}&\multicolumn{1}{m{\linkdspec}}{\textit{BIK} can aid the activation of both \textit{BAK} and \textit{BAX} by triggering \textit{BAK} oligomerization on the ER membrane and promoting a \textit{Ca\textsuperscript{2+}} efflux required for the fragmentation of hyper fused mitochondrial tubules, aiding \textit{BAK} and \textit{BAX} activation \cite{mathaiBH3onlyBIKRegulates2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{BIM}&\multicolumn{1}{m{\linkdspec}}{\textit{BAK} is preferentially activated by \textit{BID} compared to \textit{BIM}, but \textit{BIM} can also promote BAK oligomerization \cite{sarosiekBIDPreferentiallyActivates2013}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{BID}&\multicolumn{1}{m{\linkdspec}}{Activated (truncated) \textit{BID} binds to mitochondrial \textit{BAK}, resulting in its activation and oligomerization in the mitochondrial membrane, followed by \textit{cytochrome c} release \cite{weiTBIDMembranetargetedDeath2000}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{BAX}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{BAX}=\left(\mathbf{BIM}\andop\left(\left(\mathbf{BID}\orop\mathbf{BIK}\right)\orop\left(\notop\left(\left(\mathbf{BCL2}\andop\mathbf{BCLXL}\right)\andop\mathbf{MCL\_{}1}\right)\right)\right)\right)\orop\left(\left(\mathbf{BID}\orop\mathbf{BIK}\right)\andop\left(\notop\left(\mathbf{BCL2}\orop\mathbf{BCLXL}\right)\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{In contrast to \textit{BAK}, \textit{BAX} is preferentially activated by \textit{BIM} compared to \textit{BID} \cite{sarosiekBIDPreferentiallyActivates2013} and it is less responsive to sequestration by \textit{MCL-1} than the other two anti-apoptotic BCL2 family proteins \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005}. \textit{BAX} in our model turns on when stimulated by \textit{BIM} if one or more \textit{BCL2} family proteins are absent, or if \textit{BID} or \textit{BIK} are also present. In contrast, \textit{BID} or \textit{BIK} only activate \textit{BAK} if \textit{BCL2} and \textit{BCL-xL} are both absent (\textit{MCL-1} alone cannot block them).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{MCL\allowbreak\_{}1}&\multicolumn{1}{m{\linkdspec}}{\textit{MCL-1} can also bind \textit{BAK} to prevent its oligomerization, but it does so less potently than the other two BCL-2 family members \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005,germainMCL1InhibitsBAX2008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCLXL}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL-xL} binds \textit{BAX} and prevent its oligomerization in the mitochondrial membrane \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{BCL2}&\multicolumn{1}{m{\linkdspec}}{\textit{BCL2} binds \textit{BAX} and prevent its oligomerization in the mitochondrial membrane \cite{zhaiDifferentialRegulationBax2008,willisProapoptoticBakSequestered2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{BIK}&\multicolumn{1}{m{\linkdspec}}{\textit{BIK} can aid the activation of both \textit{BAK} and \textit{BAX} by triggering \textit{BAK} oligomerization on the ER membrane and promoting a \textit{Ca\textsuperscript{2+}} efflux required for the fragmentation of hyper fused mitochondrial tubules, aiding \textit{BAK} and \textit{BAX} activation \cite{mathaiBH3onlyBIKRegulates2005}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{BIM}&\multicolumn{1}{m{\linkdspec}}{Activated \textit{BIM} binds to mitochondrial \textit{BAX}, resulting in its allosteric activation and oligomerization in the mitochondrial membrane, leading to \textit{cytochrome c} release \cite{sarosiekBIDPreferentiallyActivates2013}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{BID}&\multicolumn{1}{m{\linkdspec}}{\textit{BAX} is preferentially activated by \textit{BIM} compared to \textit{BID}, but \textit{BID} can also promote \textit{BAK} oligomerization \cite{sarosiekBIDPreferentiallyActivates2013}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Cyto\allowbreak\_{}C}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Cyto\_{}C}=\mathbf{BAX}\orop\mathbf{BAK}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Cytochrome C} release from mitochondria requires the oligomerization of either \textit{BAK} or \textit{BAX} \cite{weiProapoptoticBAXBAK2001}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{BAK}&\multicolumn{1}{m{\linkdspec}}{\textit{BAK} oligomerization at the mitochondrial membrane triggers MOMP, which results in the release of \textit{cytochrome C} from mitochondria \cite{weiProapoptoticBAXBAK2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{BAX}&\multicolumn{1}{m{\linkdspec}}{\textit{BAX} oligomerization at the mitochondrial membrane triggers MOMP, which results in the release of \textit{cytochrome C} from mitochondria \cite{oltvaiBcl2HeterodimerizesVivo1993,manonReleaseCytochromeDecrease1997}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{SMAC}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{SMAC}=\mathbf{BAX}\orop\mathbf{BAK}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{SMAC/Diablo} release from mitochondria requires the oligomerization of either \textit{BAK} or \textit{BAX} \cite{weiProapoptoticBAXBAK2001,zhouDynamicsStructureBaxBak2008}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{BAK}&\multicolumn{1}{m{\linkdspec}}{\textit{BAK} oligomerization at the mitochondrial membrane triggers MOMP, which results in the release of \textit{SMAC/Diablo} from mitochondria \cite{weiProapoptoticBAXBAK2001,zhouDynamicsStructureBaxBak2008}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Loc}}}&\multicolumn{1}{m{\linkcspec}}{BAX}&\multicolumn{1}{m{\linkdspec}}{\textit{BAX} oligomerization at the mitochondrial membrane triggers MOMP, which results in the release of \textit{SMAC/Diablo} from mitochondria \cite{weiProapoptoticBAXBAK2001,zhouDynamicsStructureBaxBak2008}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{IAPs}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{IAPs}=\left(\notop\mathbf{SMAC}\right)\orop\mathbf{AKT\_{}H}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}Prot}}&\multicolumn{2}{m{\nodetwocspec}}{Inhibitor of Apoptosis Proteins (\textit{IAP}s) are active in the absence of \textit{SMAC} inhibition, or following \textit{AKT\_H} mediated upregulation (this protection from \textit{SMAC} requires peak or oncogenic \textit{AKT} activity).}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{AKT\allowbreak\_{}H}&\multicolumn{1}{m{\linkdspec}}{\textit{cIAP-2} and \textit{XIAP} are both transcriptionally up-regulated in response to strong \textit{PI3K/AKT1} activation \cite{huCriticalRoleEndogenous2004}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{SMAC}&\multicolumn{1}{m{\linkdspec}}{\textit{SMAC/Diablo} binds tightly to \textit{IAP} proteins and blocks their ability to inhibit \textit{Caspase 3} \cite{duSmacMitochondrialProtein2000}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Casp9}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Casp9}=\mathbf{Casp3}\orop\left(\left(\notop\mathbf{IAPs}\right)\andop\mathbf{Cyto\_{}C}\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{\textit{Procaspase 9} is cleaved into active \textit{Caspase 9} by \textit{Caspase 3}, or by the apoptosome (which relies on \textit{cytochrome C} for its assembly) in the absence of \textit{IAP} proteins.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Compl}}}&\multicolumn{1}{m{\linkcspec}}{Cyto\allowbreak\_{}C}&\multicolumn{1}{m{\linkdspec}}{\textit{Cytochrome c} binds to \textit{APAF-1} proteins, promoting their assembly into the apoptosome, a platform for \textit{procaspase 9} binding and cleavage into its active form \cite{srinivasulaAutoactivationProcaspase9Apaf1mediated1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{IAPs}&\multicolumn{1}{m{\linkdspec}}{\textit{XIAP}, \textit{cIAP1} and \textit{cIAP2} inhibit the \textit{cytochrome c}-induced activation of \textit{procaspase-9} \cite{deverauxIAPsBlockApoptotic1998}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Procaspase 9} is a direct cleavage target of \textit{Caspase 3} \cite{sleeExecutionerCaspase3Perform2001}.}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{Casp3}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{Casp3}=\left(\left(\mathbf{Casp9}\andop\mathbf{Casp8}\right)\orop\left(\mathbf{Casp3}\andop\left(\mathbf{Casp9}\orop\mathbf{Casp8}\right)\right)\right)\orop\left(\left(\notop\mathbf{IAPs}\right)\andop\left(\left(\mathbf{Casp9}\orop\mathbf{Casp8}\right)\orop\mathbf{Casp3}\right)\right)$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}PTase}}&\multicolumn{2}{m{\nodetwocspec}}{Activation of \textit{Caspase 3} requires proteolytic cleavage of \textit{procaspase-3} by initiator caspases such as \textit{Caspase 9} or \textit{Caspase 8}. In our model, cooperation of two of the three caspases (\textit{Casp9}, \textit{Casp8}, \textit{Casp3}) is required in the presence of \textit{IAP}s, which inhibit the proteolytic activity of \textit{Caspase 3} by bind tightly to its active site. In the absence of \textit{IAP}s, either of the three caspases can cleave and activate \textit{Caspase 3}.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp8}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 8} can cleave \textit{Caspase 3} \cite{stennickeProcaspase3MajorPhysiologic1998}, but full \textit{Caspase 3} activation also requires MOMP (potentially due to a need for \textit{IAP} inhibition) \cite{spencerNongeneticOriginsCelltocell2009}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftfootline$\\{\footnotesize{}IBind}}}&\multicolumn{1}{m{\linkcspec}}{IAPs}&\multicolumn{1}{m{\linkdspec}}{\textit{IAP}s bind tightly to the active site of \textit{Caspase 3}, keeping its activity in check \cite{deverauxIAPsBlockApoptotic1998,riedlStructuralBasisInhibition2001}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Lysis}}}&\multicolumn{1}{m{\linkcspec}}{Casp9}&\multicolumn{1}{m{\linkdspec}}{Active \textit{Caspase 9} cleaves \textit{procaspase 3} \cite{liCytochromeDATPdependentFormation1997}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Per}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{Once activated, \textit{Caspase 3} helps sustain its own activation by cleaving \textit{procaspase 8} and \textit{6}. \textit{Caspase 6}, in turn, generates additional active \textit{caspase 8} and \textit{9}. Together they all sustains a continuing active pool of Caspase 3.}\\
\bottomrule
\end{longtable}
\begin{longtable}[c]{@{}llll@{}}
\caption{DNA\_{}Fragmentation module}
\endhead{}\\
\toprule
\multicolumn{1}{p{\nodeoneaspec}}{Target Node}&\multicolumn{3}{m{\nodeonebspec}}{Node Gate}\\
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{Node Type}&\multicolumn{2}{m{\nodetwocspec}}{Node Description}\\
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{Link Type}&\multicolumn{1}{m{\linkcspec}}{Input Node}&\multicolumn{1}{m{\linkdspec}}{Link Description}\\

\midrule
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{p{\nodeoneaspec}}{CAD}&\multicolumn{3}{m{\nodeonebspec}}{$\displaystyle\let\left\relax\let\right\relax \mathbf{CAD}=\mathbf{Casp3}\andop\mathbf{Casp9}$}\\
\addlinespace[\dimexpr1.5\defaultaddspace]
\multicolumn{1}{m{\nodetwoaspec}}{}&\multicolumn{1}{m{\nodetwobspec}}{{\footnotesize{}DNase}}&\multicolumn{2}{m{\nodetwocspec}}{Caspase-activated DNase (\textit{CAD}) is activated when its inhibition is released via the cleavage of \textit{ICAD} (inhibitor of caspase-activated DNase). While \textit{Capsase 3} and \textit{7} (a direct target of \textit{Caspase 9}) can inhibit \textit{ICAD} \cite{wolfCaspase3PrimaryActivator1999}, in our model they are both required, as \textit{CAD} = ON is represents terminal, irreversible apoptotic commitment, which is fully locked in when both \textit{Caspase 3} and \textit{9} are on.}
\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Casp9}&\multicolumn{1}{m{\linkdspec}}{In addition of \textit{Caspase 3}, \textit{CAD} inhibition can also be relieved by \textit{ICAD} cleavage by \textit{Caspase 7}, which is a direct target of \textit{Caspase 9} \cite{wolfCaspase3PrimaryActivator1999}.}\\
\addlinespace[\dimexpr1.0\defaultaddspace]
\multicolumn{1}{m{\linkaspec}}{}&\multicolumn{1}{m{\linkbspec}}{\makecell{$\leftarrow$\\{\footnotesize{}Ind}}}&\multicolumn{1}{m{\linkcspec}}{Casp3}&\multicolumn{1}{m{\linkdspec}}{\textit{Caspase 3} relives \textit{CAD} inhibition by cleaving its inhibitor \textit{ICAD} \cite{wolfCaspase3PrimaryActivator1999}.}\\
\bottomrule
\end{longtable}
\end{subtables}
\newpage{}\begin{subtables}

\begin{longtable}[c]{@{}llp{\dimexpr 0.6\textwidth-2\tabcolsep}@{}}
\caption{Key to Node Type Symbols}
\endhead{}\\
\toprule
Symbol&Node Type&Description\\
\midrule
{\footnotesize{}Cell}&Cell&Node type used when a node's state is used to represent discrete phenotypes of an entire cell; appropriate for multi models.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}DM}&DM\_{}Switch&Multi-stable regulatory module (Dynamically Modular Switch) in a higher-level model layer.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Conn}&Connector&Grouping of nodes that are either mono-stable, or do not form a switch-like circuit that controls discrete phenotype transitions. For example, signaling input layers without feedback, or multi-step links between distinct switches can be represented as connector nodes in coarse-grained (switch-level) models.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Env}&Environment&Nodes or modules that represent the extracellular environment of a single cell or cell collective. These are self-sustaining nodes or node groups that maintain their initial states and receive no feedback from the rest of the network (they act as inputs).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Proc}&Process&Nodes or modules that stand in for complex cellular processes not modeled in detail (e.g., DNA replication or the process of aligning chromosomes at the metaphase plane during mitosis).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}MSt}&Macro\_{}Structure&Nodes or modules that represent the state of large, complex cellular structures such as DNA content, cytoskeletal features, junctions or mitochondria.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Met}&Metabolite&Regulatory node representing a metabolite (not protein, gene product or complex structure).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}mRNA}&MRNA&mRNA.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}miR}&MicroRNA&microRNA.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}PC}&Protein\_{}Complex&Protein complex represented by a single node or via a key member of the complex.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Rec}&Receptor&Cell surface receptor protein or complex.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Adap}&Adaptor\_{}Protein&Protein that helps scaffold a signaling complex or other large assembly of proteins.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Secr}&Secreted\_{}Protein&Protein secreted into the extracellular environment, such that the state of the node tagged with this type represents the availability fo this protein outside the cell.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}TF}&TF\_{}Protein&Transcription factor.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}K}&Kinase&Kinase (enzyme that catalyzes the phosphorylation of its target).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Ph}&Phosphatase&Phosphatase (enzyme that catalyzes the removal of phosphorylation from its target).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}UbL}&Ubiquitin\_{}Ligase&Ubiquitin ligase (protein that recruits an ubiquitin-conjugating enzyme that has been loaded with ubiquitin to a target protein and assists or directly catalyzes the transfer of ubiquitin from the ubiquitin-conjugating enzyme to the target).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}PTase}&Protease&Protease (enzyme that catalyzes the breakdown of proteins into smaller fragments).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}DNase}&DNase&Deoxyribonuclease (DNase, for short); endonuclease that catalyzes the hydrolytic cleavage of the DNA backbone.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}CAM}&CAM&Cell adhesion proteins located on the cell surface.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}CDK}&CDK&Cyclin-dependent kinase.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}CDKI}&CDKI&Cyclin-dependent kinase inhibitor.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}GEF}&GEF&Guanine nucleotide exchange factor.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}GAP}&GAP&GTPase-activating protein (also called GTPase-accelerating protein).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}GTPa}&GTPase&GTPase enzymes that hydrolyze ATP to ADP.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Enz}&Enzyme&Enzyme that does not fit the more specific enzyme categories listed above.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Prot}&Protein&Regulatory protein that does not fit any of the more specific classifications listed above.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}MP}&Membrane\_{}Potential&A relative measure of membrane potential across a biological membrane, generally indicating whether this potential is within the  normal range, or abnormally low / high in a way that affects other regulatory processes.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}lncRNA}&LncRNA&Long intervening noncoding RNA\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}SLig}&Cell\_{}Surgace\_{}Ligand&Membrane-bound signaling molecule that serves as a ligand to receptors on neighboring cells.\\\addlinespace[\dimexpr0.75\defaultaddspace]
\bottomrule
\end{longtable}

\begin{longtable}[c]{@{}llp{\dimexpr 0.6\textwidth-2\tabcolsep}@{}}
\caption{Key to Link Type Symbols}
\endhead{}\\
\toprule
Symbol&Link Type&Description\\
\midrule
{\footnotesize{}Env}&Enforced\_{}Env&This link type represents self-loops on Environment nodes, which guarantee that these nodes maintain their initial state throughout a time-course simulation unless they are explicitly altered by the simulation’s settings.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Ind}&Indirect&Regulatory influence that does not involve direct binding, processing, or enzyme activity.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}ComplProc}&Complex\_{}Process&Regulatory influence that is not modeled in detail, but involves more than one molecule or a macrostructure. For example, the physical need for kinetochores on replicated sister chromatids for the assembly of certain protein complexes can be represented as a link from the node representing kinetochores to the regulatory proteins, with a Complex\_{}Process link type.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Per}&Persistence&This link type represents self-loops that alter the ability of a node to stay in a particular state depending on its own current state. For example, if transcription of a protein is easier to maintain than to induce de novo, this may be encoded by a logic gate that includes the node itself and creates a self-loop. The link type of this loop is “Persistence”.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}TR}&Transcription&Action of a transcription factor to alter the expression of the target node (mRNA or protein). Link type should be used for induction as well as repression (the link effect contains this information).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}TL}&Translation&Regulatory influence that controls the translation of mRNA into protein; should be used for induction as well as repression of translation.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Ligand}&Ligand\_{}Binding&Binding of extracellular ligand to its receptor.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Compl}&Complex\_{}Formation&Binding even that leads to a regulatory protein complex.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}IBind}&Inhibitory\_{}Binding&Binding even that represses the target node’s level or activity.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Loc}&Localization&Regulatory influence that alters the localization of a molecule.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}BLoc}&Binding\_{}Localization&Binding even that alters the localization of a molecule.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}PBind}&Protective\_{}Binding&Binding even that increases / protects the target node’s activity.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Unbind}&Unbinding&A regulatory influence that causes the target node to be released from a protein complex and change its activity (increase or decrease) as a result.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}P}&Phosphorylation&Phosphorylation.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}DP}&Dephosphorylation&Dehosphorylation.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}PLoc}&Phosphorylation\_{}Localization&Phosphorylation resulting in altered protein localization.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Ubiq}&Ubiquitination&Ubiquitination, usually leading to protein degradation.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Deg}&Degradation&Regulatory influence leading to the degradation of the target molecule (more general than Ubiquitination; the latter link type should be used when appropriate).\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}GEF}&GEF\_{}Activity&Action of a Guanine nucleotide exchange factor (GEF) leading to GTP loading onto (and usually the activation of) a GTPase.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}GAP}&GAP\_{}Activity&Actions of a GTPase-activating protein (GAP) leading to the hydrolysis of GTP by (and usually de-activation of) a GTPase.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Lysis}&Proteolysis&Protein cleavage.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Cat}&Catalysis&Increasing the rate of metabolite production by an enzyme.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Epi}&Epigenetic&Process that alters gene expression via modifying chromatin condensation or altering DNA methylation.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}TrConf}&Transcription\_{}Conflict&\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Secr}&Secretion&Secretion or shedding of a protein or other regulatory molecule to the extracellular environment.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}RNAi}&RNAi&This process represents inhibitory binding of cytoplasmic mRNAs by RISC-bound microRNAs that block translation and/or enhance mRNA degradation.\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Acet}&Acetylation&Acetylation\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}Deacet}&Deacetylation&Deacetylation\\\addlinespace[\dimexpr0.75\defaultaddspace]
{\footnotesize{}-OH}&Hydroxylation&Hydroxylation\\\addlinespace[\dimexpr0.75\defaultaddspace]
\bottomrule
\end{longtable}

\begin{longtable}[c]{@{}llp{\dimexpr 0.6\textwidth-2\tabcolsep}@{}}
\caption{Key to Link Effect Symbols}
\endhead{}\\
\toprule
Symbol&Link Effect&Description\\
\midrule
$\leftarrow$&Activation&Link in which the input node aids the expression, activity, persistence or localization of the target such that the target is easier to turn/keep in an ON state. It can be used for multi-level nodes as long as these levels represent increasing intervals of activity.\\\addlinespace[\dimexpr0.75\defaultaddspace]
$\leftfootline$&Repression&Link in which the input node hinders the expression, activity, persistence or localization of the target such that the target is easier to turn/keep in an OFF state. It can be used for multi-level nodes as long as these levels represent increasing intervals of activity.\\\addlinespace[\dimexpr0.75\defaultaddspace]
$\leftblackspoon$&Context\_{}Dependent&Link acting on a node in such a way that it activates it under certain conditions (e.g., when another input is OFF), but represses it when this condition is not met. The XOR gate is a good example of a context-dependent link effect. \\\addlinespace[\dimexpr0.75\defaultaddspace]
$\perp$&Inapt&This link-type refers to connections between complex regulatory switches (rather than molecules), where categorizing the effect of an input as Activation or Repression, or even context-depended activation or repression does not apply. This is usually the case with multi-state switches, where the 3 or more phenotypes represented by the discrete states of these switches do not have a meaningful ordering. Thus, stating that this switch is “activated” by another one is not appropriate.\\\addlinespace[\dimexpr0.75\defaultaddspace]
\bottomrule
\end{longtable}
\end{subtables}\newpage{}\bibliographystyle{unsrt}
\bibliography{references.bib}

\end{document}